<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">

  <meta name="description" content="Wiki TeamFreiburg:chAMBER | iGEM competition 2022 | 11 enthusiastic young scientists creating a toolbox for compartmentalization in bacteria for future synthetic biology applications and beyond.">
  <meta name="author" content="iGEM-Team Freiburg 2022 @JonasWidder @AnneSmedegard">
  <title>Labbook | TeamFreiburg:chAMBER | iGEM2022</title>
  <link rel="icon" type="image/x-icon" href="https://static.igem.wiki/teams/4229/wiki/pictures/general-design/favicon.ico">

  <!-- links -->
  <!-- css external stylesheets -->
  <link rel="stylesheet" href="static/slick.css">
  <link rel="stylesheet" href="static/boxicons.min.css">
  <!-- css internal stylesheets -->
  <link rel="stylesheet" href="static/general.css">
  <link rel="stylesheet" href="static/layout-elements.css">
  <link rel="stylesheet" href="static/specific-styles.css">
</head>

<body>

  <!-- ____________________________ main_navbar ____________________________ -->
  <div class="main_navbar">

    <!-- logo + team name -->
    <a href="index.html" class="main_navbar_logo">
      <img src="https://static.igem.wiki/teams/4229/wiki/pictures/general-design/logo-chamber.png" alt="teamlogo of chAMBER, Freiburgs iGEM team 2022">
      |
      <span class="hidden_navtxt">
         iGEM Freiburg 2022 <span style="font-family:txt_font;font-size:14px;">presents</span>
      </span>
      <span style="font-size:1.3rem;">ch</span>AMBER
    </a>


    <!-- navbar on small screens -->
    <div id="smallNav" class="sn_overlay">

      <!-- small navbar topbar with logo and close-btn -->
      <div class="sn_topbar">
        <!-- logo on sn-nav in top left corner -->
        <a href="index.html">
          <img class="sn_logo" src="https://static.igem.wiki/teams/4229/wiki/pictures/general-design/logo-chamber.png" alt="teamlogo of chAMBER, Freiburgs iGEM team 2022">
          <span class="sn_teamname">
            |
            <span class="hidden_navtxt">
              iGEM Freiburg 2022 <span style="font-family:txt_font;font-size:14px;">presents</span>
            </span>
            <span style="font-size:1.3rem;">ch</span>AMBER
          </span>
        </a>
        <!-- Button to close the overlay navigation -->
        <a href="javascript:void(0)" class="sn_closebtn" onclick="closeNav()"><i class="fa-solid fa-xmark"></i></a>
      </div>

      <!-- small navbar content -->
      <div class="sn_overlay_content">
        <a href="index.html">Home</a>
        <a onclick="sn_dropdownFnc1()" href="#1">
          Project <i id="sn_arrow1" class="fa fa-solid fa-angle-down fa-sm"></i>
        </a>
        <ul id="sn_dropdown1">
          <li><a href="description.html">Description</a></li>
          <li><a href="design.html">Design</a></li>
          <li><a href="results.html">Results</a></li>
          <li><a href="model.html">Modelling</a></li>
          <li><a href="engineering.html">Engineering Success</a></li>
          <li><a href="proof-of-concept.html">Proof of concept</a></li>
          <li><a href="implementation.html">Proposed Implementation</a></li>
          <li><a href="contribution.html">Contribution</a></li>
        </ul>
        <a class="active" onclick="sn_dropdownFnc2()" href="#1">
          Experiments <i id="sn_arrow2" class="fa fa-solid fa-angle-down fa-sm"></i>
        </a>
        <ul id="sn_dropdown2">
          <li><a href="protocols.html">Protocols</a></li>
          <li><a class="active" href="labbook.html">Labbook</a></li>
          <li><a href="safety.html">Safety</a></li>
        </ul>
        <a href="parts.html">Parts</a>
        <a onclick="sn_dropdownFnc4()" href="#1">
          Human Practices <i id="sn_arrow4" class="fa fa-solid fa-angle-down fa-sm"></i>
        </a>
        <ul id="sn_dropdown4">
          <li><a href="human-practices.html">Integrated Human Practices</a></li>
          <li><a href="communication.html">Education & Communication</a></li>
        </ul>
        <a onclick="sn_dropdownFnc5()" href="#1">
          Team <i id="sn_arrow5" class="fa fa-solid fa-angle-down fa-sm"></i>
        </a>
        <ul id="sn_dropdown5">
          <li><a href="team.html">Team Members</a></li>
          <li><a href="collaborations.html">Collaborations</a></li>
          <li><a href="attributions.html">Attributions</a></li>
        </ul>
        <a href="awards.html">Medals/Awards</a>
      </div>
    </div>

    <!-- menu-hamburger for open small navbar -->
    <div id="sn_menu_bar">
      <span onclick="openNav()"><i class="fa-solid fa-bars fa-2x"></i></span>
    </div>


    <!-- nav (larger screens) -->
    <nav class="main_nav">
      <ul>
        <li><a href="index.html">Home</a></li>
        <li><a href="#">Project</a>
        <ul>
          <li><a href="description.html">Description</a></li>
          <li><a href="design.html">Design</a></li>
          <li><a href="results.html">Results</a></li>
          <li><a href="model.html">Modelling</a></li>
          <li><a href="engineering.html">Engineering Success</a></li>
          <li><a href="proof-of-concept.html">Proof of concept</a></li>
          <li><a href="implementation.html">Proposed Implementation</a></li>
          <li><a href="contribution.html">Contribution</a></li>
        </ul>
      </li>
        <li><a class="active" href="#">Experiments</a>
          <ul>
            <li><a href="protocols.html">Protocols</a></li>
            <li><a class="active" href="labbook.html">Labbook</a></li>
            <li><a href="safety.html">Safety</a></li>
          </ul>
        </li>
        <li><a href="parts.html">Parts</a></li>
        <li><a href="#">Human Practices</a>
          <ul>
            <li><a href="human-practices.html">Integrated Human Practices</a></li>
            <li><a href="communication.html">Education & Communication</a></li>
          </ul>
        </li>
        <li><a href="#">Team</a>
          <ul>
            <li><a href="team.html">Team Members</a></li>
            <li><a href="collaborations.html">Collaborations</a></li>
            <li><a href="attributions.html">Attributions</a></li>
          </ul>
        </li>
        <li><a href="awards.html">Medals/Awards</a></li>
      </ul>
    </nav>
  </div>

  <!-- _______________________________ header_______________________________ -->
  <header class="sp_header">
    <div class="sp_header_txt">
      <h1>Labbook</h1>
    </div>
  </header>


  <!-- ______________________________ main ________________________________ -->
  <main>
    <div class="container grey">
      <div class="flex_flexstart py-2">

              <!-- subordinated navigation -->
              <div class="sub_nav_wrap">


              <section class="sub_nav sn_col_1">
                <ul>
                  <li><a href="#link_labbook_compartments"><strong>Compartments</strong></a>
                    <ul>
                      <li><a href="#link_labbook_encapsulin"> > Encapsulins</a></li>
                      <li><a href="#link_labbook_phase"> > Phase Seperation</a></li>
                      <li><a href="#link_labbook_microcopartments"> > Microcompartments</a></li>
                    </ul>
                  </li>
                <br>
                  <li><a ><strong>Noncanonical Amino Acids </strong></a>
                    <ul>
                      <li><a href="#link_labbook_ncAA"> > ncAAs</a></li>
                    </ul>
                  </li>
                <br>
                  <li><a> <strong>Pathways</strong> </a>
                    <ul>
                      <li><a href="#link_labbook_indigo"> > Indigo</a></li>
                      <li><a href="#link_labbook_trehalose"> > Trehalose</a></li>
                    </ul>
                  </li> <br>
                </ul>
              </section>
              </div>

  <!-- _______________________________ Compartments ______________________________ -->
  <section class="text_block">
    <h2 id='link_labbook_microcompartments'><strong> Microcompartments</strong></h3>
                Starting from thinking we would like to work with microcompartments, our project expanded into a comparison of different compartments. We wanted to explore how we can use these different compartments to incoorporate pathways and thereby expand the metabolic capacities of bacteria. Our project ended up involving RNA scaffolds, Encapsulins, Microcompartments and phaseseperation.
              <div class="line-1"> </div>
              <b>08/05-22</b><br>
              <div class="line-2"> </div>

              LB agar plates + Chloramphenicol 25 μg/ml<br>
              LB agar plates + Kanamycin 50 μg/ml
              See protocol #004

              <br><br>
              <b>09/05-22</b><br>
              <div class="line-2"> </div>
              LB medium (3 liters). See protocol #001<br><br>

              <b>11/05-22</b><br>
              <div class="line-2"> </div>
              For this week we wanted to test the competent cells in the -80 °C freezer from last year. For this
              LB+Chloramphinicol plates as well as SOC medium was prepared. For this see protocol #001
              and #002.<br>
              <strong>Heatshock</strong> of competent bacteria:<br>
              There are 4 boxes with competent cells in the freezer, and we labeled the boxes and our
              samples 1-4. Heat shock was performed with pBAD33 (10 ng) in ~25 μl competent bacteria and
              975 μl SOC medium was added. See protocol #003.<br>
              Bacteria were plated after incubation at 37°C at LB+resistence plates in two different
              concentrations: 50 μl and 200 μl and placed in the incubator O/N.<br>
              <br>
              <strong>LB agar plates</strong> + Chloramphenicol 25 μg/mL<br>
              See protocol #004<br><br>

              <b>12/05-22</b><br>
              <div class="line-2"> </div>
              For the plates corresponding to box 1-3 there was about 100-200 colonies. Box 4 had few
              and we decided not to make a miniprep for this sample (should be thrown out).<br>
              The plates were kept too long in the incubator and had started to become large and have
              some satellite colonies, which should be avoided.<br>
              For the box 1-3 samples <strong>startercultures</strong> of 5ml was prepared and a single colony was
              picked and added to LB + Chloramphenicol medium. The cultures were placed at 37 °C
              200 rpm.<br><br>
              <strong>LB agar plates</strong> + Chloramphenicol 25 μg/ml. See protocol #004<br>

                <br><b>13/05-22</b><br>
                <div class="line-2"> </div>
                A <strong>miniprep</strong> of the cultures from yesterday was prepared according to the protocol #005.
                2 ml from each starter culture was used. The absorbances in the Nanodrop looked nice and clean. The
                concentrations were all ~50 ng/ul and the elution was done with 30 µl dH2O. The eluted
                plasmids were stored at -20 °C in the plasmid box. <br><br>
                <strong>LB agar plates</strong> + Chloramphenicol 25 µg/ml + Kanamycin 50 µg/ml <br>
                See protocol #004<br>

                <br><b>15/05-22</b><br>
                <div class="line-2"> </div>
                Inoculate <strong>overnight culture</strong> of plasmid #1882 and #2333 in LB medium with Streptomycin and Kanamycin respectively <br><br>
                As a controll competent E.coli from 2019 where also grown over night in LB medium: <br>
                - w/o any antibiotic <br>
                - with Sm <br>
                - with Cm<br><br>
                We also made 10 <strong>LB-agar plates</strong> with SM and Cm

                <br><b>30/05-22</b><br>
                <div class="line-2"> </div>
                <p><strong>Starterculture of BL21: </strong>Pick 1 colony from plate from Saturday and incubate 10ml LB no resistance</p><p>&nbsp;</p><p><strong>CaCl2 Solution: </strong>Prepared 1 M CaCl2 an 100mM Cacl2. Autoclave didn&rsquo;t finish and a salt was preticipated after 3+ hours.</p>

                <br><b>31/05-22</b><br>
                <div class="line-2"> </div>
                <p>&nbsp;</p><p><strong>CaCl2 1M&nbsp; and 100mM preparation:</strong></p><table style="width: 259px;" border="black"><tbody><tr><td style="width: 103px;"><p>&nbsp;</p></td><td style="width: 75px;"><p>100 mM</p></td><td style="width: 59px;"><p>1 M</p></td></tr><tr><td style="width: 103px;"><p>CaCl2 (111g/mol)</p><p>&nbsp;</p></td><td style="width: 75px;"><p>2.2 g</p></td><td style="width: 59px;"><p>5.5 g</p></td></tr><tr><td style="width: 103px;"><p>dH2O</p></td><td style="width: 75px;"><p>200ml</p></td><td style="width: 59px;"><p>50 ml</p></td></tr></tbody></table><p>&nbsp;--&gt;Autoclaved</p><p><strong>Making competent BL21 cells</strong></p><p>300ml LB no res was inoculated with ~3ml starterculture from yesterday and placed at 37C 200rpm until OD600 = 0.5. Cells (2x50ml) were spun down 4 &deg;C 4000 rpm for 10 min. Resuspended in 20 ml CaCl2 (100 mM) per pellet. Kept on ice 30 min. Centrifuge 4 &deg;C 4000 rpm 10 min. Resuspend in 5 ml 100 mM CaCl2 15 % glycerol. Aliquited 50 &micro;l/eppi total 84 eppis --&gt; See protocol #17</p>

                <br><b>01/06-22</b><br>
                <div class="line-2"> </div>
                <p><strong>Heat shock trafo</strong> of pBAD33 into newly prepared competent BL21: 50&micro;l Bacteria, 10ng pBAD33. See protocol #003. The cells were plated on 2 LB+amp plates (20 &micro;l and 200 &micro;l) and left in incubator 37 &deg;C.</p>

                <br><b>02/06-22</b><br>
                <div class="line-2"> </div>
                <p><strong>Heat shock trafo</strong> of pBAD33 into newly prepared competent BL21: Yesterday we took the wrong plates (with ampicillin res.). However, pBAD33 has an Chloramphinicol resistance. This was also the case for the Gibson assembly. We used 50 &micro;l BL21 and 10ng pBAD33. See protocol #003. The cells were plated on 2 LB+Chl plates (20 &micro;l and 200 &micro;l) and left at incubator 37 &deg;C.</p>


                <br><b>03/06-22</b><br>
                <div class="line-2"> </div>
                <p><strong>Overnight Cultures: </strong>There were colonies on all plates for the pBAD33 into the Bl-21 competent cells. They were almost completely full, so we thought maybe the resistence on the plate is not working. To verify that the cells actually had the pBAD33, we prepared an overnight culture of 2 colonies in LB-Chloramphinicol medium at 30 &deg;C 200 rpm overnight. </p>
                <br><p>There were also colonies on the trafold with the pBAD_SPD5_sfGFP and from these we picked 12 colonies and prepared overnight culture in 5ml LB-Chl medium. These were grown at 30 &deg;C 200 rpm overnight.</p>
                <br>

  <!-- _______________________________ Encapsulins ______________________________ -->
  <h2 id='link_labbook_encapsulin'>Encapsulins</h2>
    <div class="line-1"> </div>


      <b>29/06-22</b><br>
      <div class="line-2"> </div>
      <p><span data-contrast="none"><strong>PCR of EncA fragment from IDT</strong>: in order to amplify via T7 primers, which are incoorporated in the sequence. We make 2 x 25 &micro;l total volume</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p>
      <p><span data-contrast="none">Primers: T7 standard primers</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p>
      <p><span data-contrast="none">Expected length 900 bp&nbsp;</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p>
      <p><span data-contrast="none">Annealing temperature 56&deg;C&nbsp;</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p>
      <p><span data-contrast="none">Extension time: 30 sec</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p>
      <p><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p>
      <p><span data-contrast="none">Gel showed bands at the right size, which were cut out and purified via gel extraction. Eluted in 20 &micro;l water.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p>
      <p><span data-contrast="none">Conc.: 26.9 ng/&micro;l</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}"> <br /></span></p>
      <br><p><span data-contrast="none">As the concentration of the EncA is not so high we do another <strong>PCR</strong> with the same conditions but 2 x 50 &micro;l total volume.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p>

      <br><b>30/06-22</b><br>
      <div class="line-2"> </div>
      <p><span data-contrast="none"><strong>Gelextraction</strong>: Gel from yesterdays PCR of EncA was made and bands at the right size (900 bp) were cut out and cleaned:&nbsp;</span></p>
      <p><span data-contrast="none">Conc.: 55ng/ &micro;l and 39,1 ng/&micro;l</span></p>

      <br><b>01/07-22</b><br>
      <div class="line-2"> </div>
      <p>For the EncA the second step is to <strong>digest the fragment</strong> with restriction enzymes. This we do so we can ligate it into the backbone. This way we can create the same sticky overhangs on the fragment and the backbone.</p><p>&nbsp;</p><p><strong>Digest of pBAD33</strong></p><table><tbody><tr><td width="165"><p>DNA</p></td><td width="90"><p>1 &micro;g</p></td></tr><tr><td width="165"><p>10X NEB CutSmart</p></td><td width="90"><p>5 &micro;l</p></td></tr><tr><td width="165"><p>XmaI</p></td><td width="90"><p>1 &micro;l</p></td></tr><tr><td width="165"><p>SphI</p></td><td width="90"><p>1&micro;l</p></td></tr><tr><td width="165"><p>Water</p></td><td width="90"><p>Up to 50 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>&nbsp;</p><p><strong>Digest of EncA</strong></p><table><tbody><tr><td width="165"><p>DNA</p></td><td width="90"><p>1 &micro;g</p></td></tr><tr><td width="165"><p>10X NEB CutSmart</p></td><td width="90"><p>5 &micro;l</p></td></tr><tr><td width="165"><p>XmaI</p></td><td width="90"><p>1 &micro;l</p></td></tr><tr><td width="165"><p>SphI</p></td><td width="90"><p>1&micro;l</p></td></tr><tr><td width="165"><p>Water</p></td><td width="90"><p>Up to 50 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><ol><li>Calculate volume of DNA and water</li><li>Pipet water, CutSmart buffer and DNA into eppi on ice</li><li>Incubate at 37&deg;C for 60 min</li><li>Cool down to room temperature before using, spin down any possible condensed water</li></ol><p>&nbsp;</p><p><strong>Run on agarose gel</strong>, we expect a band at 900 kb and one at 5300 bp</p><p>&nbsp;</p><p><strong>Cut out</strong> <strong>and purify</strong> via Monarch gel extraction kit</p><p>EncA digest (conc.: 15 ng/&micro;l)</p><p>PBAD digest (conc.: 8 ng/&micro;l)</p><p>&nbsp;</p><p>These digests were used to make a <strong>ligation</strong> with the T4 DNA ligase and T4 ligation buffer 10x. According to protocol #028.</p><p>Vector: 0.02 pmol (25.5 ng)</p><p>Insert: 0.06 pmol (50 ng)</p><p>&nbsp;</p><p>This was used for a <strong>transformation</strong> of 1 &micro;l into NEB5-alpha (50 &micro;l), everything was plated on Chloramphinicol plates.</p><p>&nbsp;</p>

      <br><b>03/07-22</b><br>
      <div class="line-2"> </div>
      As we used most of the EncA PCR fragment for the digest we do another <strong>PCR</strong> with the same conditions but 2 x 50 µl total volume

      <br><b>10/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Ligation of EncA and pBAD</strong> (ratio 1:3), calculated with NEB calculator.</p><p>&nbsp;</p><table border="black"><tbody><tr><td width="101"><p>pBAD: 50 ng</p></td><td width="98"><p>pBAD digest</p></td><td width="85"><p>1.6 &micro;l</p></td></tr><tr><td width="101"><p>EncA: 25 ng</p></td><td width="98"><p>EncA digest</p></td><td width="85"><p>6.25 &micro;l</p></td></tr><tr><td width="101"><p>Water</p></td><td width="98"><p>&nbsp;</p></td><td width="85"><p>9.15 &micro;l</p></td></tr><tr><td width="101"><p>T4 buffer</p></td><td width="98"><p>&nbsp;</p></td><td width="85"><p>2 &micro;l</p></td></tr><tr><td width="101"><p>T4 Ligase</p></td><td width="98"><p>&nbsp;</p></td><td width="85"><p>1 &micro;l</p></td></tr></tbody></table><p>&nbsp;<strong>Transformation</strong> of ligation into top10, 2,5 &micro;l of ligation (EncA)</p><p>&nbsp;all was plated on LB+Cm plates</p>

      <br><b>11/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Plates from yesterday:</strong></p>
      <p>Encapsulin: 20 colonies</p>



      <br><b>18/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Miniprep</strong> of EncA colonies C1,&nbsp; C3,&nbsp; C4,&nbsp; C7.</p>
      <p>Send C1 for <strong>sequencing</strong> using eurofins and with 2 standard primers</p>
      <p>-&gt;result: it did not have the insert</p>
      <p>&nbsp;</p>
      <p>Pellets of 3ml from other overnight cultures of different colonies, froze at -20&deg;C</p>

      <br><b>19/07-22</b><br>
      <div class="line-2"> </div>
      <p>As I saw from the <strong>sequencin</strong>g that the EncA was not inserted I tried to send another 3 colonies to sequencing. This time it was C3, C4, C7. They were sent using Genewiz and standard primers.</p>
      <p>-&gt;Result: none of them have the insert, at this point I didn&rsquo;t know why since on the gel it looked correct</p>
      <p>&nbsp;</p>


      <br><b>20/07-22</b><br>
      <div class="line-2"> </div>
      <p>To test which bands are produced from the colony PCR of just pBAD I run 9 <strong>PCR</strong>s with different temperatures to also test if the annealing temperature is ok. Besides I ran a PCR with just the top10 cells to see if the primers bind somewhere in the genome.</p>
      <p>&nbsp;</p>
      <p>Dreamtaq Mastermix</p>
      <p>Annealing temp: 55&deg;C and testing between 52&deg;C &ndash; 58&deg;C, extension time 2 min, 30 cycles</p>

      <br><b>22/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight cultures </strong>of colonies from the pBAD_EncA ligation (C6 + C11)</p>
      <p>Overnight culture of #1924 which is pET302_sfGFP plasmid. It contains already sfGFP in a backbone compatible with the pBAD and I will use this for further cloning.</p>

      <br><b>23/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight cultures</strong> of pBAD_SPD5 C1, C2, C3: even though they showed no bands at the colony PCR, I will just try to make a miniprep as the primers I used might have not bound properly.</p>

      <br><b>25/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Sequencing </strong>using Genewiz and standard primers for pBAD. I send two colonies from the EncA C6 and C11.</p>
      <p>&nbsp;Result: The encapsulins had a region missing at the C-terminal.</p>

      <br><b>27/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>PCR</strong> for cloning of TP onto sfGFP using #1924 as a template. Amplify sGFP with sfGFP_fwd and sfGFP_TP_rev_1 (primer 31 + 29)</p>
      <p>Anneal: 63-65 &deg;C, Extension: 30 sec (600bp), 30 cycles, total 8 samples 2 at every temperature (63, 64,64.5, 65&deg;C).</p>
      <p>&nbsp;</p>
      <p><strong>Gel</strong> of the PCR with sfGFP_TP: expected band size: 600bp. 1 kb ladder from Jena bioscience</p>
      <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Result: bands were present for all conditions</p>
      <p>&nbsp;</p>
      <p>Band from the gel with sfGFP were cut out and purified using <strong>gel extraction</strong>.</p>
      <p>&nbsp;</p>
      <p>Since the sequencing from yesterday showed a problem with the IDT fragment, I become the pJET_encA from our supervisor who cloned it in the meantime. An overnight culture is prepared.</p>
      <p>&nbsp;</p>
      <p><strong>PCR</strong> for cloning of TP onto sfGFP using PCR above as a template. Amplify sGFP with sfGFP_fwd and sfGFP_TP_rev_2 (primer 30 + 31)</p>
      <p>Anneal: 65 &deg;C, Extension: 30 sec (600bp), 30 cycles, total 5 samples.</p>
      <p>&nbsp;</p>

      <br><b>28/07-22</b><br>
      <div class="line-2"> </div>
      <p>Digest of pBAD</p><table><tbody><tr><td width="141"><p>DNA pBAD33 (1 &micro;g)</p></td><td width="76"><p>22.22 &micro;l</p></td></tr><tr><td width="141"><p>Xma</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>SphI</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>cutsmart</p></td><td width="76"><p>5 &micro;l</p></td></tr><tr><td width="141"><p>water</p></td><td width="76"><p>20.7 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>Digest of EncA using pJET_EncA</p><table><tbody><tr><td width="141"><p>DNA pJET_EncA (1 &micro;g)</p></td><td width="76"><p>15.15 &micro;l</p></td></tr><tr><td width="141"><p>Xma</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>SphI</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>cutsmart</p></td><td width="76"><p>5 &micro;l</p></td></tr><tr><td width="141"><p>water</p></td><td width="76"><p>27.8 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>All digests were incubated at 37&deg;C for 1 h.</p><p>After 1 h digests were run on gels:</p><p>Gel 1:</p><table width="627"><tbody><tr><td width="63"><p>Marker:</p><p>ZR 1kb ladder</p></td><td width="82"><p>pBAD undigested</p></td><td width="67"><p>pBAD digested</p></td><td width="67"><p>pBAD digested</p></td><td width="82"><p>1924</p><p>undigested</p></td><td width="67"><p>1924</p><p>digested</p></td><td width="67"><p>1924</p><p>digested</p></td><td width="67"><p>sfGFP</p><p>digested</p></td><td width="67"><p>sfGFP digested</p></td></tr></tbody></table><p>&nbsp;</p><p>Gel 2:</p><table width="627"><tbody><tr><td width="142"><p>Marker:</p><p>ZR 1kb ladder</p></td><td width="185"><p>Other probes x 3</p></td><td width="150"><p>EncA digested</p></td><td width="150"><p>EncA digested</p></td></tr></tbody></table><p>&nbsp;</p><p>All the digested backbones and the inserts were cut out and purified using gel-extraction.</p><p>&nbsp;</p><table><tbody><tr><td width="107"><p>Digest</p></td><td width="107"><p>pBAD</p></td><td width="107"><p>EncA</p></td></tr><tr><td width="107"><p>Conc.</p></td><td width="107"><p>20 ng/&micro;l</p></td><td width="107"><p>5,6 ng/&micro;l</p></td></tr></tbody></table>

      <br><b>29/07-22</b><br>
      <div class="line-2"> </div>
      <p>From the digests from yesterday I prepare ligation reactions for the final plasmids: pBAD_EncA and pET_sfGFP_TP respectively</p><p>&nbsp;</p><p><strong>pBAD_encA</strong></p><table><tbody><tr><td width="151"><p>pBAD digest (50 ng)</p></td><td width="66"><p>2.5 &micro;l</p></td></tr><tr><td width="151"><p>EncA digest (26 ng)</p></td><td width="66"><p>1 &micro;l</p></td></tr><tr><td width="151"><p>T4 buffer</p></td><td width="66"><p>2 &micro;l</p></td></tr><tr><td width="151"><p>T4 ligase</p></td><td width="66"><p>1 &micro;l</p></td></tr><tr><td width="151"><p>water</p></td><td width="66"><p>9.9 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p><strong>pET_sfGFP</strong></p><table><tbody><tr><td width="151"><p>1924 digest (50 ng)</p></td><td width="66"><p>3.8 &micro;l</p></td></tr><tr><td width="151"><p>EncA digest (18.75 ng)</p></td><td width="66"><p>0.65 &micro;l</p></td></tr><tr><td width="151"><p>T4 buffer</p></td><td width="66"><p>2 &micro;l</p></td></tr><tr><td width="151"><p>T4 ligase</p></td><td width="66"><p>1 &micro;l</p></td></tr><tr><td width="151"><p>water</p></td><td width="66"><p>12.55 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>*for each digest a relegation test was also made, with the respective backbone only.</p><p>&nbsp;</p><p>Both digests were <strong>ligated </strong>at RT for 10 min and heat-inactivated 65 &deg;C for 10 min. Then 5 &micro;l from each ligation mix was used for a heat shock trafo into top10 competent cells and the total volume was plated on LB + CmR/AmpR plates and incubated overnight.</p><p><strong>&nbsp;</strong></p><p>From this the TA of our supervisor have cloned it into pJET and I get the plate to do the<strong> colony PCR</strong> of 20 colonies. This was done using the Dreamtaq polymerase green MM and the standard forward and reverse primers for pJET. Anneal at 52&deg;C, Extention for 4 min, 30 cycles.</p><p>&nbsp;</p><p>The colony PCR of pJET_SPD5_1 was run on a<strong> gel </strong>(OBS C1 is missing) with the ladder from Zymo ZR 1 kb.</p><p>This gel showed that all colonies had an insert at the right size about 2000 bp.</p>

      <br><b>30/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Plates from yesterday:</strong> There were no colonies on the plates with sfGFP_TP. However, there were nice colonies on the plates with pBAD_EncA and fewer for the relegation with pBAD.</p><p>&nbsp;</p><p><strong>Colony PCR</strong> of 20 colonies from pBAD_EncA ligation. This was perfomed using Dreamtaq green MM and &ldquo;T7 promoter&rdquo; and &ldquo;T7 terminal primers&rdquo;. Annealing at 58&deg;C and extension of 2 min, 30 cycles.</p><p>&nbsp;</p><p>Following a <strong>gel was run </strong>using the ladder: Jena bioscience 1kb plus. OBS: C8 is missing. There were several colonies with an insert: C1, 3, 4, 5, 6, 9, 10, 11, 13, 15, 16, 17, 18, 20.</p><p>&nbsp;</p><p>Following <strong>overnight cultures</strong> were prepared: 10 ml of 1924 (AmR), 10 ml of pBAD33 (CmR).</p><p>&nbsp;</p><p>Another <strong>heat shock trafo</strong> of the ligation with sfGFP_TP and the religation was perfomed using 5 &micro;l of the ligation and 10 &micro;l bought DH5alpha competent cells. The total volume was plated on two LB +AmR plates and incubated overnight.</p><p>&nbsp;</p>

      <br><b>31/07-22</b><br>
      <div class="line-2"> </div>
      <p>From the overnight cultures from yesterday <strong>miniprep</strong> was performed using the Sigma Aldrich kit.</p><table><tbody><tr><td width="107"><p>&nbsp;</p></td><td width="107"><p>pBAD33</p></td><td width="107"><p>1924</p></td><td width="107"><p>1924</p></td><td width="107"><p>1924</p></td></tr><tr><td width="107"><p>Conc.</p></td><td width="107"><p>91 ng/&micro;l</p></td><td width="107"><p>137 ng/&micro;l</p></td><td width="107"><p>197 ng/&micro;l</p></td><td width="107"><p>138 ng/&micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p><strong>Plates from yesterday:</strong> on the pET_sfGFP_TP plate there were 4 colonies and on the relegation 0.</p><p>I use these 4 colonies for a colony PCR using the Dreamtaq green MM and the primers: sfGFP_fwd and sfGFP_TP_rev with 64&deg;C as annealing and 1 min as extension for 30 cycles.</p><p>&nbsp;</p><p>Gel of <strong>colony PCR</strong> with pETsfGFP there were bands at the correct size for all colonies, but because of how the primers were made this could still just be a carryover of undigested whole 1924 without the target peptide. An overnight culture was prepeared from all 4 colonies. Ladder ZR 1kb from Zymo.</p><p>&nbsp;</p><p>Digest of #1924 with BlpI and KplnI:</p><table><tbody><tr><td width="141"><p>DNA #1924 (1 &micro;g)</p></td><td width="76"><p>5 &micro;l</p></td></tr><tr><td width="141"><p>KpnI-HF</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>BlpI</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>Cutsmart</p></td><td width="76"><p>5 &micro;l</p></td></tr><tr><td width="141"><p>Water</p></td><td width="76"><p>38 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>This was repeated for 5 samples and incubated 2-3 h at 37 &deg;C</p><p>&nbsp;</p><p>Afterwards, the 5 digests were run on an <strong>agarose gel</strong> and the band from the digested backbone was cut out and purified using gel-extraction. Ladder ZR 1kb from Zymo.</p><p>&nbsp;Concentration after clean-up: 8,9 ng/&micro;l.</p><p>&nbsp;</p><p>From the new digest and the already digested insert (sfGFP_TP) a new ligation was prepared. This time I diluted the sfGFP 2x to avoid the pipetting of small volumes:</p><p>&nbsp;</p><table><tbody><tr><td width="151"><p>1924 digest (50 ng)</p></td><td width="66"><p>&nbsp;&micro;l</p></td></tr><tr><td width="151"><p>sfGFP digest (18.75 ng)</p></td><td width="66"><p>1.3 &micro;l</p></td></tr><tr><td width="151"><p>T4 buffer</p></td><td width="66"><p>2 &micro;l</p></td></tr><tr><td width="151"><p>T4 ligase</p></td><td width="66"><p>1 &micro;l</p></td></tr><tr><td width="151"><p>water</p></td><td width="66"><p>&nbsp;&micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>As previously, a relegation test without the insert was prepared. The samples were ligated for 10 min at RT and afterwards heat-inactivated at 65&deg;C for 10 min. Following ligation 5 &micro;l of each ligation mix was transformed into 10 &micro;l DH5alpa competent cells and the final volume was plated on two LB + Amp plates and incubated O/N.</p><p>&nbsp;</p>
      <p><strong>Overnight cultures</strong> were prepared 4 colonies from the pBAD_encA which all seemed to have an insert at the right size from the colony PCR.</p>


      <br><b>01/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Miniprep</strong> of the overnight cultures from yesterday was prepared 4 from pBAD_EncA </p>
      <p>--&gt; The colonies from the minipreps were all send for <strong>sequencing</strong> using eurofins and standard primers.</p>
      <p>&nbsp;</p>
      <p><strong>Colony PCR</strong> of the pET_sfGFP_TP plates from yesterdays ligation. Using Thermo fischer Green Taq pol. Master mix.</p>
      <p>&nbsp;The colony PCR was <strong>run on a gel</strong> with the RZ 1kb ladder.</p>
      <p>--&gt;The gel showed that there was a band for multiple colonies, but one colony had a stronger band at the right size.</p>
      <p>&nbsp;</p>
      <p><strong>Overnight cultures</strong> were prepared for the colonies of pET_sfGFP_TP which showed bands. (c3,c4,c5,c6).</p>

      <br><b>02/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Sequencing results</strong>:</p>
      <p>For pBAD_EncA there is also one colony with the right sequence, no mutations, c3.</p>
      <p>For pET_sfGFP_TP there is also the correct sfGFP, but it looks like it has only been amplified with the first PCR, so it is missing some part of the target peptide.</p>
      <p><strong>Colony PCR</strong> of the pET_sfGFP_TP plates from yesterdays ligation. Using Thermo fischer Green Taq pol. Master mix.</p>
      <p>&nbsp;</p>
      <p>The colony PCR was <strong>run on a gel</strong> with the RZ 1kb ladder.</p>
      <p>The gel showed that there was a band for multiple colonies, but one colony had a stronger band at the right size.</p>
      <p>&nbsp;</p>
      <p><strong>Overnight cultures</strong> were prepared for the colonies of pET_sfGFP_TP which showed bands. (c3,c4,c5,c6).</p>

      <br><b>03/08-22</b><br>
      <div class="line-2"> </div>
      <p>From the pBAD_encA c3 a <strong>transformation</strong> is made to have a clean colony to pick for a glycerol stock.</p><p><strong>PCR</strong> was prepared using pET_sfGFP_TP c1 as a template to amplify the sfGFP_TP, this time with the full target peptide. Annealing temperature: 65&deg;C, extension time: 30 sec, 30 cycles. Prepared with Q5 polymerase.</p><p>An <strong>agarose gel </strong>was prepared of the above PCR, RZ 1kb ladder.</p><table width="100%"><tbody><tr><td width="8%"><p>Marker</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="2%"><p>&nbsp;</p></td><td width="8%"><p>SPD5_1</p></td><td width="8%"><p>SPD5_1</p></td><td width="8%"><p>SPD5_1</p></td><td width="8%"><p>SPD5_1</p></td><td width="8%"><p>SPD5_1</p></td></tr></tbody></table><p>&nbsp;</p><p>The bands for the respective fragments were cut out and purified with gel extraction.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; sfGFP_TP fragment: 97 ng/&micro;l and 207 ng/&micro;l</p><p>&nbsp;</p><p><strong>Miniprep</strong> of overnight cultures of pET_sfGFP_TP from the 01/08-22 c3-c6.</p><table><tbody><tr><td width="75"><p>C3</p></td><td width="95"><p>C4</p></td><td width="113"><p>C5</p></td><td width="104"><p>C6</p></td></tr><tr><td width="75"><p>71 ng/&micro;l</p></td><td width="95"><p>133 ng/&micro;l</p></td><td width="113"><p>141 ng/&micro;l</p></td><td width="104"><p>97 ng/&micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>Those sequences were send for sequencing using eurofins and standard primers

      <br><b>04/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Sequencing results</strong> from yesterday of pET_sfGFP_TP showed that c3 has the entire target peptide incorporated. Therefor I don&rsquo;t need the extra fragments which I made fresh.</p>
      <p>&nbsp;<strong>Overnight cultures</strong> of the pBAD_EncA c3 for making glycerol stocks.</p>

      <br><b>05/08-22</b><br>
      <div class="line-2"> </div>
      <p>A <strong>double trafo</strong> was made in order to create BL21 (DE3) cells with both the pBAD_EncA and the pET_sfGFP (#1924) and another strain with pBAD_EncA and the pET_sfGFP_TP. Those I plan to use for fluorescent microscopy. Both were plated on LB-plates + Amp + Chl.</p>
      <p><strong>Glycerol stocks</strong> were prepared for the pBAD_EncA</p>

      <br><b>06/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight cultures</strong> were prepared from one colony from the double trafo with pBAD_EncA and pET_sfGFP.</p>

      <br><b>07/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Glycerol stocks</strong> were prepared of the two strains of bacteria containing pBAD_encA and either pET_sfGFP with or without target peptide.</p><p>&nbsp;</p><p>&nbsp;</p>

      <strong>Fluorescence microscopy of EncA #01:</strong> <br>
      <div class="line-2"> </div>
      <p>&nbsp;</p><p><strong>Aim:</strong> observe wether the GFP with target peptide and the encapsulins come together to form fluorescent &ldquo;dots&rdquo; as has been observed for the microcompartments.</p><p>&nbsp;</p><p>To do this I have prepared two strains of BL21 (DE3):</p><ul><li>Carry pBAD_encA and pET_sfGFP</li><li>Carry pBAD_encA and pET_sfGFP_TP</li></ul><p>Strain 1 in this experiment is the control as the GFP has no targeting peptide. Strain 2 should show dots.</p><p>&nbsp;</p><p>From these two strains overnight cultures are prepared and later a larger culture which grows until OD around 0.6. Then 3 ml for every condition will be induced or not induced. The plasmid pBAD_EncA is induced by arabinose and pET_sfGFP is induced with IPTG.</p><p>&nbsp;</p><p>After induction the cultures they produce the proteins for 24 h at 18 &deg;C. Then they are subjected to fluorescence microscopy. 1 ml will be spun down and is lysed and both pellet and supernatant will be used for westernblot for detection of a his-tag present on the encapsulins.</p>
      <div class="line-2"> </div>

      <br><b>09/08-22</b><br>
      <div class="line-2"> </div>
      <p>For the experiment 2 <strong>overnight cultures</strong> were prepared from the glycerol stocks containing 1. pBAD_EncA and pET_sfGFP and 2. pBAD_EncA and pET_sfGFP_TP. Both were grown in LB+amp+CmR</p>
      <p>&nbsp;</p>
      <p><strong>LB for tomorrow</strong> 200 ml total was prepared with both resistences and left at RT.</p>

      <br><b>10/08-22</b><br>
      <div class="line-2"> </div>
      <p>Today <strong>2 larger cultures</strong>, 100 ml each, were prepared from the overnight cultures and they were grown until OD around 0.6 and then they were induced according to the scheme:</p><p>&nbsp;</p><table border="black"><tbody><tr><td width="80"><p>IPTG (&micro;M)</p></td><td width="36"><p>0</p></td><td width="61"><p>0</p></td><td width="55"><p>0</p></td><td width="47"><p>100</p></td><td width="61"><p>100</p></td><td width="55"><p>100</p></td><td width="36"><p>0</p></td><td width="61"><p>0</p></td><td width="55"><p>0</p></td><td width="32"><p>100</p></td><td width="32"><p>100</p></td><td width="32"><p>100</p></td></tr><tr><td width="80"><p>Arabinose (%)</p></td><td width="36"><p>0</p></td><td width="61"><p>0,0025</p></td><td width="55"><p>0,005</p></td><td width="47"><p>0</p></td><td width="61"><p>0,0025</p></td><td width="55"><p>0,005</p></td><td width="36"><p>0</p></td><td width="61"><p>0,0025</p></td><td width="55"><p>0,005</p></td><td width="32"><p>0</p></td><td width="32"><p>0,0025</p></td><td width="32"><p>0,005</p></td></tr></tbody></table><p>&nbsp;</p><p>For pipetting a stock solution of IPTG of 100&micro;M was used and for arabinose a 5% solution was used.</p><p>&nbsp;</p><p>The cultures were shaking at 18&deg;C for 24 h</p>

      <br><b>11/08-22</b><br>
      <div class="line-2"> </div>
      <p>After 24 h has passed 1 ml of each sample is centrifuged at 3000 x g for 10 min and thrown supernatent away and the pellet is frozen at -20&deg;C. The rest of the sample is used for <strong>fluorescence microscopy</strong> and agarose pads protocol #025 are prepared with each sample for imaging. 3 pictures pr. Slide are taken.</p>

      <br><b>12/08-22</b><br>
      <div class="line-2"> </div>
      <p>The spun-down samples from 11/08-22 are thawed and resuspeded in 1 ml lysis buffer (20 mM Tris + 50 mM NaCl). Sonication of all the samples. Centrifuge all the samples at 3000 rpm for 10 min</p>
      <p><strong>Prepare samples for SDS</strong>: Pipette 10 &micro;l supernatent of each sample together with 10 &micro;l loading buffer (Laemmli buffer) and a sample from the pellet is made.</p>
      <p>&nbsp;</p>
      <p>SDS-gels are prepared according to protocol #031. And 1 gel is run pr 12 samples.</p>
      <p>&nbsp;</p>
      <p>Gel 1: samples 1-6 (supernatant) and samples 1-6 (pellet)</p>
      <p>Gel 2: samples 7-12 (supernatant) and samples 7-12 (pellet)</p>
      <p>&nbsp;</p>
      <p>Both SDS-gels are then used for transfer for westernblot. For this a semi-dry system is used.</p>
      <p>Protocol #030.</p>
      <p>Membranes were incubated overnight with the primary antibody (anti-his).</p>

      <br><b>13/08-22</b><br>
      <div class="line-2"> </div>
      Membranes were washed and incubated with the secondary antibody (anti-mouse-HRP) for 2 h at RT. After the membranes were washed the signal was detected using the imager, protocol #030.

      <br><b>19/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Agarose gel</strong> of yesterday&rsquo;s PCR of different PCR amplified fragments of EncA.</p><p>&nbsp;</p><table><tbody><tr><td width="128"><p>Marker</p></td><td width="128"><p>EncA_micro</p></td><td width="36"><p>X4</p></td><td width="94"><p>SPD5_micro</p></td><td width="28"><p>X4</p></td></tr><tr><td width="128"><p>GeneRuler 1kb</p></td><td width="128"><p>Bands</p></td><td width="36"><p>&nbsp;</p></td><td width="94"><p>Bands</p></td><td width="28"><p>&nbsp;</p></td></tr></tbody></table><p>&nbsp;</p><p>All bands were cut out and purified using <strong>gel-extraction</strong></p><p>EncA_micro: 96 ng/&micro;l</p><p>SPD5_micro: 148 ng/&micro;l</p><p>&nbsp;</p><p><strong>Overnight culture</strong> of pBAD_encA + pET_sfGFP_TP and pBAD_EncA + pET_sfGFP for the EncA #02 experiment. This was done by starting from the glycerol stocks, which was prepared last time.</p><p>&nbsp;<strong>Digest of EncA</strong></p><table><tbody><tr><td width="113"><p>SPD5</p></td><td width="104"><p>1000ng (10 &micro;l)</p></td></tr><tr><td width="113"><p>NdeI</p></td><td width="104"><p>1 &micro;l</p></td></tr><tr><td width="113"><p>XhoI</p></td><td width="104"><p>1 &micro;l</p></td></tr><tr><td width="113"><p>Cutsmart buffer</p></td><td width="104"><p>5 &micro;l</p></td></tr><tr><td width="113"><p>Water</p></td><td width="104"><p>33 &micro;l</p></td></tr></tbody></table>

      <br><b>20/08-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;<strong>Induction of EncA and sfGFP_TP</strong> according to scheme. This is a part of the experiment EncA #02</p><p>&nbsp;</p><p><strong>Gel with digest</strong> from yesterday&rsquo;s digest of the backbone and the inserts for pMIC_SPD5 and pMIC_EncA</p><table width="100%"><tbody><tr><td width="13%"><p><strong>Marker</strong></p></td><td width="20%"><p><strong>pHT1spy_snoopT2T3 undigested</strong></p></td><td width="20%"><p><strong>pHT1spy_snoopT2T3 digest</strong></p></td><td width="4%"><p><strong>X3</strong></p></td><td width="9%"><p><strong>SPD5 digested</strong></p></td><td width="9%"><p><strong>SPD5 digested</strong></p></td><td width="10%"><p><strong>EncA digested</strong></p></td><td width="10%"><p><strong>EncA digested</strong></p></td></tr><tr><td width="13%"><p>GeneRuler 1kb</p></td><td width="20%"><p>band</p></td><td width="20%"><p>band</p></td><td width="4%"><p>X3</p></td><td width="9%"><p>band</p></td><td width="9%"><p>band</p></td><td width="10%"><p>band</p></td><td width="10%"><p>Band</p></td></tr><tr><td width="13%"><p><strong>Marker</strong></p></td><td width="20%"><p><strong>SPD5_snoop 2&deg;</strong></p></td><td width="20%"><p><strong>SPD5_spy 2&deg;</strong></p></td><td width="4%"><p>&nbsp;</p></td><td width="9%"><p>&nbsp;</p></td><td width="9%"><p>&nbsp;</p></td><td width="10%"><p>&nbsp;</p></td><td width="10%"><p>&nbsp;</p></td></tr><tr><td width="13%"><p>GeneRuler 1kb</p></td><td width="20%"><p>Band x4</p></td><td width="20%"><p>Band x 4</p></td><td width="4%"><p>&nbsp;</p></td><td width="9%"><p>&nbsp;</p></td><td width="9%"><p>&nbsp;</p></td><td width="10%"><p>&nbsp;</p></td><td width="10%"><p>&nbsp;</p></td></tr></tbody></table><p>&nbsp;</p><p>Clean up of digest from gel was performed for all bands using <strong>gel-extraction</strong></p><p>EncA_micro: 24.9 ng/ &micro;l</p>

      <br><b>21/08-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;</p><p><strong>Ligation of pMIC_encA and pMIC_SPD5: </strong>Ratio was 1:2, 3 ligations were performed as described:</p><table><tbody><tr><td width="149"><p>&nbsp;</p></td><td width="105"><p>pMIC_EncA</p></td><td width="103"><p>Religation</p></td></tr><tr><td width="149"><p>pHT1spy/snoopT2T3 digest (50ng)</p></td><td width="105"><p>1.1 &micro;l</p></td><td width="103"><p>1.1 &micro;l</p></td></tr><tr><td width="149"><p>Insert</p></td><td width="105"><p>1 &micro;l</p></td><td width="103"><p>&nbsp;</p></td></tr><tr><td width="149"><p>water</p></td><td width="105"><p>14.9 &micro;l</p></td><td width="103"><p>15.9 &micro;l</p></td></tr><tr><td width="149"><p>T4 buffer</p></td><td width="105"><p>2 &micro;l</p></td><td width="103"><p>2 &micro;l</p></td></tr><tr><td width="149"><p>T4 ligase</p></td><td width="105"><p>1 &micro;l</p></td><td width="103"><p>1 &micro;l</p></td></tr></tbody></table><p>All ligations were incubated 10 min at RT at heat inactivated for 10 min at 65&deg;C.</p><p>&nbsp;</p><p><strong>Fluorescence microscopy</strong> of the samples prepared for EncA #02 was performed.</p><p>&nbsp;</p>

      <br><b>22/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Colony PCR</strong> of the constructs; pMIC_SPD5 and pMIC_EncA using the Green Dreamtaq polymerase Mastermix. Total samples 8.</p>
      <p>&nbsp;</p>
      <p><strong>Gel of colony PCR</strong> of the 8 colonies showed that 1 of each construct had a band at the right size.</p>
      <p>&nbsp;</p>
      <p><strong>Overnight cultures</strong> were prepared of the pMIC_SPD5 c1 and pMIC_EncA c1, which showed bands at the agarose gel.</p>
      <p>&nbsp;</p>

      <br><b>23/08-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;</p>
      <p>I <strong>send the pMIC_EncA c3 a for sequencing</strong> using standard primers for the encapsulins. Result: The pMIC_EncA looks good and has no mutations. </p>
      <p>&nbsp;</p>
      <p><strong>Miniprep</strong> of the overnight cultures from yesterday of pMIC_encA, eluted in 50 &micro;l using Sigma Aldrich kit.</p>
      <p>&nbsp;</p>

      <br><b>25/08-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;</p>
      <p><strong>Overnight cultures</strong> were prepared of pMIC_EncA c3 in 5 ml LB + Amp. They were incubated 25&deg;C for 24 h.</p>

      <br><b>26/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Miniprep </strong>of overnight cultures of&nbsp; pMIC_EncA was prepared and they were subsequently sent for sequencing.</p>

      <div class="line-2"> </div>
      <p><strong>Encapsulins #03</strong></p><p><strong>Aim:</strong> To test whether a difference in the media composition can influence the induction of encapsulins via arabinose. The two mediums which are compared are TB and LB. Besides, a higher concentration of arabinose is tested, 0.01%.</p><p>&nbsp;</p><p>2 strains in BL21:</p><p>Strain 1: (pBAB_EncA + pET_sfGFP)</p><p>Strain 2: (pBAD_EncA + pETsfGFP_TP)</p><p>&nbsp;</p><p><strong>Concentrations tested in LB</strong></p><table><tbody><tr><td width="64"><p>Sample</p></td><td width="64"><p>1/10</p></td><td width="64"><p>2/11</p></td><td width="64"><p>3/12</p></td><td width="64"><p>4/13</p></td><td width="64"><p>5/14</p></td><td width="64"><p>6/15</p></td><td width="64"><p>7/16</p></td><td width="64"><p>8/17</p></td><td width="64"><p>9/18</p></td></tr><tr><td width="64"><p>Arabinose %</p></td><td width="64"><p>0</p></td><td width="64"><p>0.005</p></td><td width="64"><p>0.01</p></td><td width="64"><p>0</p></td><td width="64"><p>0</p></td><td width="64"><p>0.005</p></td><td width="64"><p>0.005</p></td><td width="64"><p>0.01</p></td><td width="64"><p>0.01</p></td></tr><tr><td width="64"><p>IPTG &micro;M</p></td><td width="64"><p>0</p></td><td width="64"><p>0</p></td><td width="64"><p>0</p></td><td width="64"><p>25</p></td><td width="64"><p>100</p></td><td width="64"><p>25</p></td><td width="64"><p>100</p></td><td width="64"><p>25</p></td><td width="64"><p>100</p></td></tr></tbody></table><p>&nbsp;</p><p>Strain 1: samples 1-9</p><p>Strain 2: samples 10-18</p><p>&nbsp;</p><p><strong>Concentrations tested in TB</strong></p><table><tbody><tr><td width="80"><p>Sample</p></td><td width="62"><p>1/5</p></td><td width="63"><p>2/6</p></td><td width="62"><p>3/7</p></td><td width="63"><p>4/8</p></td></tr><tr><td width="80"><p>Arabinose %</p></td><td width="62"><p>0</p></td><td width="63"><p>0.005</p></td><td width="62"><p>0</p></td><td width="63"><p>0.005</p></td></tr><tr><td width="80"><p>IPTG &micro;M</p></td><td width="62"><p>0</p></td><td width="63"><p>0</p></td><td width="62"><p>100</p></td><td width="63"><p>100</p></td></tr></tbody></table><p>&nbsp;</p><p>Strain 1: samples 1-4</p><p>Strain 2: samples 5-8</p><p>&nbsp;</p><p>&nbsp;</p>
      <div class="line-2"> </div>

      <br><b>28/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight </strong>cultures from glycerol stock were prepared for the experiments Encapsulins #03. Cultures are grown in LB or TB as described.</p>

      <br><b>29/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>pBAD_EncA #03:</strong></p><p><strong>Diluted overnight cultures</strong> for the different strains 1:50 in 100 ml and grew the larger cultures at 30&deg;C until OD 0.4 (TB medium samples) or OD 0.6 (LB medium samples). This was done with a 1 h delay between the first samples (pBAD_EncA in LB) and last samples (pBAD_encA in TB). <strong>Induction</strong> was performed at the given OD with the given concentrations stated earlier.</p><p>&nbsp;</p><p><strong>PCRs for cloning</strong> of the sfGFP_TP constructs into the mVENUS backbone to have it in a backbone without the T7 promoter. Four x 25 &micro;l was prepared for each fragment.</p><p>PCR for fragment sfGFP_TP for the mVEN</p><p>Primer 38 + 40, annealing: 63&deg;C, extension time: 30 sec, template pET_sfGFP</p><p>PCR for the fragment sfGFP for the mVEN</p><p>Primer 38 + 39, annealing: 63&deg;C, extension time: 30 sec, template pET_sfGFP</p><p>&nbsp;</p><p>The PCR samples were run on <strong>an agarose gel </strong>with the GeneRuler 1 kb ladder:</p><table><tbody><tr><td width="75"><p>Marker</p></td><td width="151"><p>sfGFP_TP fragment</p></td><td width="123"><p>sfGFP fragment</p></td></tr><tr><td width="75"><p>&nbsp;</p></td><td width="151"><p>121 ng/&micro;l</p></td><td width="123"><p>113 ng/&micro;l</p></td></tr></tbody></table><p>All PCRs showed bands at the correct size and were cut out for purification using gel-extraction.</p>

      <br><b>30/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Digests of multiple fragments:</strong></p><table><tbody><tr><td><p>Fragment/plasmid</p></td><td><p>DNA</p></td><td><p>Enzymes</p></td><td><p>Water</p></td><td><p>CutSmart</p></td></tr><tr><td><p>sfGFP (38 +39)</p></td><td><p>8.8 &micro;l</p></td><td><p>BamHI, EcoRI</p></td><td><p>34.2 &micro;l</p></td><td><p>5 &micro;l</p></td></tr><tr><td><p>sfGFP_TP (38 +40)</p></td><td><p>8.2 &micro;l</p></td><td><p>BamHI, EcoRI</p></td><td><p>34.8 &micro;l</p></td><td><p>5 &micro;l</p></td></tr><tr><td><p>mVENUS backbone</p></td><td><p>21 &micro;l</p></td><td><p>BamHI, EcoRI</p></td><td><p>22 &micro;l</p></td><td><p>5 &micro;l</p></td></tr></tbody></table><ul><li>Incubate all at 37&deg;C for 1 h</li></ul><p>&nbsp;</p><p>All Digests were loaded on a gel and subsequently cut out an purified using gel-extraction:</p><table style="width: 393px;"><tbody><tr><td style="width: 81px;"><p>Marker</p></td><td style="width: 57px;"><p>sfGFP</p></td><td style="width: 51px;"><p>sfGFP_TP</p></td><td style="width: 49.6px;"><p>mVENUS</p></td></tr><tr><td style="width: 81px;"><p>GeneRuler 1kb</p></td><td style="width: 57px;"><p>13.6 ng/&micro;l</p></td><td style="width: 51px;"><p>18 ng/&micro;l</p></td><td style="width: 49.6px;"><p>6.8 ng/&micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p><strong>Microscopy</strong> of the Encapsulins #03&nbsp; was performed.</p>

      <br><b>31/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight ligation</strong> of pVEN_sfGFP(_TP) ratio 1:3</p><table border="BLACK"><tbody><tr><td><p>&nbsp;</p></td><td><p>pVENUS backbone (50 ng)</p></td><td><p>Insert (93 fmol)</p></td><td><p>T4 buffer</p></td><td><p>T4 ligase</p></td><td><p>Water</p></td></tr><tr><td><p>pVEN_sfGFP</p></td><td><p>7.35 &micro;l</p></td><td><p>3.2 &micro;l</p></td><td><p>2 &micro;l</p></td><td><p>1 &micro;l</p></td><td><p>6.45 &micro;l</p></td></tr><tr><td><p>pVEN_sfGFP_TP</p></td><td><p>7.35 &micro;l</p></td><td><p>2.44 &micro;l</p></td><td><p>2 &micro;l</p></td><td><p>1 &micro;l</p></td><td><p>7.21 &micro;l</p></td></tr><tr><td><p>Religation</p></td><td><p>7.35 &micro;l</p></td><td><p>&nbsp;</p></td><td><p>2 &micro;l</p></td><td><p>1 &micro;l</p></td><td><p>9.65 &micro;l</p></td></tr></tbody></table><ul><li>Leave at 4&deg;C overnight</li></ul><p>&nbsp;</p><p>&nbsp;</p>

      <br><b>01/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Heat shoch transformation</strong> of the pVEN_sfGFP(_TP) into DH5-alpha using 2 &micro;l of the overnight ligation. The bacteria were plated on LB + CmR plates overnight.</p><p>&nbsp;</p><p><strong>Preparation of samples from the pBAD_EncA for westernblot</strong>. Pellets were stored at -20&deg;C and were first resuspended in Lysis-buffer and sonicated for 30 sec. The samples were centrifuged for 10 min at 4000xg and 15 &micro;l was mixed with 15 &micro;l SDS sample buffer. Then samples were stored at -20&deg;C until SDS-PAGE was prepared.</p><p>&nbsp;</p><p>&nbsp;</p>

      <div class="line-2"> </div>
      <h2><strong>EncA_#03</strong></h2><p><strong>Aim:</strong> To test whether a difference in the media composition can influence the induction of encapsulins via arabinose. The two mediums which are compared are TB and LB. Besides, a higher concentration of arabinose is tested, 0.01%.</p><p>&nbsp;</p><p>Concentrations tested in LB</p><table border="black"><tbody><tr><td width="64"><p>Sample</p></td><td width="64"><p>1/10</p></td><td width="64"><p>2/11</p></td><td width="64"><p>3/12</p></td><td width="64"><p>4/13</p></td><td width="64"><p>5/14</p></td><td width="64"><p>6/15</p></td><td width="64"><p>7/16</p></td><td width="64"><p>8/17</p></td><td width="64"><p>9/18</p></td></tr><tr><td width="64"><p>Arabinose %</p></td><td width="64"><p>0</p></td><td width="64"><p>0.005</p></td><td width="64"><p>0.01</p></td><td width="64"><p>0</p></td><td width="64"><p>0</p></td><td width="64"><p>0.005</p></td><td width="64"><p>0.005</p></td><td width="64"><p>0.01</p></td><td width="64"><p>0.01</p></td></tr><tr><td width="64"><p>IPTG &micro;M</p></td><td width="64"><p>0</p></td><td width="64"><p>0</p></td><td width="64"><p>0</p></td><td width="64"><p>25</p></td><td width="64"><p>100</p></td><td width="64"><p>25</p></td><td width="64"><p>100</p></td><td width="64"><p>25</p></td><td width="64"><p>100</p></td></tr></tbody></table><p>&nbsp;</p><p>Strain 1: samples 1-9</p><p>Strain 2: samples 10-18</p><p>&nbsp;</p><p>Concentrations tested in TB</p><table border="black"><tbody><tr><td width="80"><p>Sample</p></td><td width="62"><p>1/5</p></td><td width="63"><p>2/6</p></td><td width="62"><p>3/7</p></td><td width="63"><p>4/8</p></td></tr><tr><td width="80"><p>Arabinose %</p></td><td width="62"><p>0</p></td><td width="63"><p>0.005</p></td><td width="62"><p>0</p></td><td width="63"><p>0.005</p></td></tr><tr><td width="80"><p>IPTG &micro;M</p></td><td width="62"><p>0</p></td><td width="63"><p>0</p></td><td width="62"><p>100</p></td><td width="63"><p>100</p></td></tr></tbody></table><p>&nbsp;</p><p>Strain 1: samples 1-4</p><p>Strain 2: samples 5-8</p><p>&nbsp;</p><p><strong></strong></p><p>&nbsp;</p>
      <div class="line-2"> </div>

      <br><b>02/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Transformations</strong> via heat-shock:</p><p>pBAD33 (empty) + pET_sfGFP into BL21</p><p>pBAD33 (empty) + pET_sfGFP_TP into BL21</p><p>&nbsp;</p><p><strong>Colony PCR</strong> of pVEN_sfGFP and pVEN_sfGFP_TP was prepared, screening 8 colonies for each construct. This was done using Green Dream-tag polymerase MM using primers 38 + 39 to amplify sfGFP and 38 + 40 to amplify sfGFP_TP.</p><p>&nbsp;</p><p><strong>Agarose gel of colony PCRs:</strong></p><p>Gel 1: marker, sfGFP_TP (primer 38 + 40) (x8)</p><p>Gel 2: marker sfGFP (primer 38 + 39) (x8)</p><p>&nbsp;</p><p><strong>Miniprep</strong> of mVENUS in TOP10, 3 ml were used for the miniprep with the sigma Aldrich kit.</p><p>&nbsp;</p><p><strong>Overnight cultures</strong> were prepared of the colonies which seemed to have a band in the colony PCR of the pVEN_sfGFP_(TP) construct. 3 colonies were picked and cultured overnight.</p>

      <br><b>03/09-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;</p><p><strong>Miniprep of 3 colonies</strong> from each construct of pVEN_sfGFP_(TP) was performed, for sequencing on Monday.</p><p>&nbsp;</p>

      <br><b>04/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Glycerol stock</strong> was prepared from the overnight cultures of pBAD33 + pETsfGFP_TP, pBAD33 + pET_sfGFP.</p>
      <p>&nbsp;</p>

      <br><b>05/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Heat shock transformation</strong> of pBAD33+pET_sfGFP/pBAD33 + pET_sfGFP_TP all in BL21.</p>

      <br><b>06/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Result of sequencing</strong> showed that the pVEN_sfGFP_TP_C1 had worked out, for pET_sfGFP there were unfortunately no match.</p>
      <p><strong>&nbsp;</strong></p>
      <p><strong>Overnight cultures</strong> from yesterday of pBAD33+pET_sfGFP/pBAD33 + pET_sfGFP_TP were dilutes 1:50 in larger cultures and grown at 30&deg;C until they reach OD: 0.6</p>
      <p>&nbsp;</p>
      <p><strong>Colony PCR of pVEN_sfGFP</strong> of the same plate, but with other colonies.Using Dreamtaq Green MasterMix. Primer 38 + 39. Annealing 55&deg;C, Extension 4 min.</p>

      <div class="line-2"> </div>
      <p><strong>Control experiment for the Encapsulins #01</strong></p><p><strong>&nbsp;</strong></p><p><strong>Aim:</strong> The aim is to observe whether the observed fluorescent dots for the experiments with Encapsulins are due to the Encapsulins being expressed or if they form even in the absence of the plasmids.</p><p>&nbsp;</p><p>Strain 1: pBAD33 + pET_sfGFP</p><p>Strain 2: pBAD33 + pET_sfGFP_TP</p><p>&nbsp;</p><p><strong>Experimental</strong>: BL21 was transformed with pBAD33, same backbone as for the Encapsulins, and a plasmid containing either sfGFP or sfGFP with the encapsulation tag C-terminally.&nbsp; The induction and incubation follow the same procedure as for the previous experiment.</p><p>&nbsp;</p><table><tbody><tr><td width="128"><p>Sample</p></td><td width="128"><p>1 / 5</p></td><td width="128"><p>2 / 6</p></td><td width="128"><p>3 / 7</p></td><td width="128"><p>4 / 8</p></td></tr><tr><td width="128"><p>Arabinose (%)</p></td><td width="128"><p>0</p></td><td width="128"><p>0</p></td><td width="128"><p>0.005</p></td><td width="128"><p>0.005</p></td></tr><tr><td width="128"><p>IPTG (&micro;M)</p></td><td width="128"><p>0</p></td><td width="128"><p>100</p></td><td width="128"><p>0</p></td><td width="128"><p>100</p></td></tr></tbody></table><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p>
      <div class="line-2"> </div>

      <br><b>07/09-22</b><br>
      <div class="line-2"> </div>
      Microscopy of Control experiment for Encapsulins

      <br><b>13/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Colony PCR of pVEN_sfGFP</strong> using Green Mastermix taq Polymerase and primers 38 +39 at 55&deg;C and extension time 1 min.</p><p>&nbsp;</p><p><strong>Colony PCR</strong> was run on an agarose gel with the GeneRuler 1 kb ladder. However, no samples showed an amplified insert.</p><p>&nbsp;</p><p><strong>Digest</strong> of pVEN backbone with EcoRI and BamHI.</p><p>&nbsp;</p><table><tbody><tr><td><p>pVEN (1000 ng)</p></td><td><p>23 &micro;l</p></td></tr><tr><td><p>EcoRI</p></td><td><p>1 &micro;l</p></td></tr><tr><td><p>BamHI</p></td><td><p>1 &micro;l</p></td></tr><tr><td><p>CutSmart</p></td><td><p>5 &micro;l</p></td></tr><tr><td><p>Water</p></td><td><p>20 &micro;l</p></td></tr></tbody></table><p>Incubate 30 min at 37&deg;C and then add 1 &micro;l Fast Alkaline Phosphatase and continue incubation 30 min.</p><p>&nbsp;</p><p>Digest was run on an <strong>agarose gel</strong> with the GeneRuler 1 kb ladder, there was a nice band for the cut-out and the backbone was cut out and cleaned up using gel extraction according to the kit.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Conc.: 4.7 ng/&micro;l</p><p>&nbsp;</p><p><strong>Heat shock transformation</strong> was performed with the MG1655 and two double trafos with either: pVEN_sfGFP_TP + 1939 or pVEN_sfGFP + pMIC_EncA. After the 1 h incubation everything was plated on 2 LB + CmR +AmR plates.</p><p>&nbsp;</p>

      <br><b>14/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight culture</strong> of MG1655 with either 1939 + pVEN_sfGFP_TP or pMIC_EncA + pVEN_sfGFP_TP in CmR and AmR.</p>
      <p>&nbsp;</p>
      <p><strong>Colony PCR of MG1655</strong> with either 1939 + pVEN_sfGFP_TP or pMIC_EncA + pVEN_sfGFP_TP checking 2 colonies from both strains for both plasmids. This is done using Green Master mix Taq polymerase and primer 38 + 40 for sfGFP_TP and 32 +33 at annealing temp: 55 or 60&deg;C respectively.</p>
      <p>&nbsp;</p>
      <p><strong>Trafo of overnight ligation</strong> from yesterday of pVEN_sfGFP into DH5-alpha.</p>
      <p>&nbsp;</p>

      <br><b>15/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Plate with trafo</strong> of overnight ligation from yesterday of pVEN_sfGFP showed no colonies.</p><p>&nbsp;</p><p><strong>Trafo heat shock</strong> of either (pMIC_EncA + pVEN_sfGFP_TP) or (1939 + pVEN_sfGFP_TP) in BL21 and plated on LB + AmR + CmR plates</p><p>&nbsp;</p><p><strong>Glycerol stock</strong> of the two MG1655 strains: pMIC_EncA + pVEN_sfGFP_TP or pMIC_EncA + 1939 was prepared</p><p>&nbsp;</p><p>Larger culture and <strong>induction</strong> of pMIC_EncA/pVEN_sfGFP in MG1655 was induced according to Emcapsulin #04. The cultures were induced 2 hours earlier than planned.</p><p>&nbsp;</p>

      <br><b>16/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Heat-shock transformation</strong> of</p><p>pMIC_EncA + pVEN:sfGFP_TP in ME5119&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (AmR + CmR)</p><p>1939 + pVEN_sfGFP_TP in ME5119&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (AmR + CmR)</p><p>&nbsp;</p><p><strong>Microscopy of Encapsulin #04</strong></p>
      
     
      <br><b>17/09-22</b><br>
      <div class="line-2"> </div>
      Overnight cultures from glycerol stocks of pMIC_EncA + pVEN_sfGFP_TP or pMIC_EncA + pBAD33
    
      <div class="line-2"> </div>
      <p><strong>Encapsulins #05</strong></p><p>Aim: To observe if the results produced in BL21 can be reproduced in MG1655 and if a strain not devoted to protein production might be better suited for forming the Encapsulins. Both strains are in MG1655. Since last time the expression of sfGFP was too low it had to be repeated.</p><p>Strain A: in MG1655</p><p>Strain B: in BL21</p><p>&nbsp;</p><p>Strain A1: 1934 + pVEN_sfGFP_TP</p><p>Strain A2: pMIC_EncA + pVEN_sfGFP_TP</p><p>Strain B1: 1934 + pVEN_sfGFP_TP</p><p>Strain B2: pMIC_EncA + pVEN_sfGFP_TP</p><p>&nbsp;</p><p>&nbsp;</p><table style="height: 137px; width: 550px;" border="b"><tbody><tr><td style="width: 108px;"><p>Sample</p></td><td style="width: 41px;"><p>1</p></td><td style="width: 33px;"><p>2</p></td><td style="width: 36.35px;"><p>3</p></td><td style="width: 29.65px;"><p>4</p></td><td style="width: 23px;"><p>5</p></td><td style="width: 23px;"><p>6</p></td><td style="width: 23px;"><p>7</p></td><td style="width: 23px;"><p>8</p></td><td style="width: 23px;"><p>9</p></td><td style="width: 33px;"><p>10</p></td><td style="width: 33px;"><p>11</p></td><td style="width: 33px;"><p>12</p></td></tr><tr><td style="width: 108px;"><p>Doxycyclin (ng/ml)</p></td><td style="width: 41px;"><p>0</p></td><td style="width: 33px;"><p>0</p></td><td style="width: 36.35px;"><p>0</p></td><td style="width: 29.65px;"><p>10</p></td><td style="width: 23px;"><p>25</p></td><td style="width: 23px;"><p>50</p></td><td style="width: 23px;"><p>50</p></td><td style="width: 23px;"><p>50</p></td><td style="width: 23px;"><p>50</p></td><td style="width: 33px;"><p>100</p></td><td style="width: 33px;"><p>100</p></td><td style="width: 33px;"><p>100</p></td></tr><tr><td style="width: 108px;"><p>IPTG (&micro;M)</p></td><td style="width: 41px;"><p>0</p></td><td style="width: 33px;"><p>50</p></td><td style="width: 36.35px;"><p>100</p></td><td style="width: 29.65px;"><p>0</p></td><td style="width: 23px;"><p>0</p></td><td style="width: 23px;"><p>0</p></td><td style="width: 23px;"><p>10</p></td><td style="width: 23px;"><p>25</p></td><td style="width: 23px;"><p>50</p></td><td style="width: 33px;"><p>10</p></td><td style="width: 33px;"><p>25</p></td><td style="width: 33px;"><p>50</p></td></tr></tbody></table><p>&nbsp;</p><p>Pipetting was done using a 100mM stock of IPTG and 50ng/&micro;l Doxycyclin stock.</p><p>&nbsp;</p><p>The samples of which I did microscopy were:</p><p>Strain A: none showed the results seen on Friday so I didn&rsquo;t spend more time on them</p><p>Strain B: sample 1, 2, 4, 8</p>
      <div class="line-2"> </div>

      <br><b>18/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Induction</strong> of Encapsulin #05 according to previously described</p>

      <br><b>19/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Microscopy</strong> of Encapsulins #05</p>


  <!-- _______________________________ SPD5 ______________________________ -->
  <h2 id='link_labbook_phase'>Phase seperation</h2>
      <div class="line-1"> </div><br>
      <br><br><b>30/05-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;</p><p><strong>PCRs of pBAD33 backbone, SPD5_1, SPD5_2 and sfGFP_linker using Q5 polymerase 2X Mastermix.</strong></p><table style="height: 120px; width: 770px;" border="Black"><tbody><tr><td style="width: 88px;">Fragments</td><td style="width: 112px;">Backbone pBAD33 (5.3kb)</td><td style="width: 155px;">SPD5_1 (2kb)</td><td style="width: 184.517px;">SPD5_2 (1.6kb)</td><td style="width: 190.483px;">sfGFP_linker</td></tr><tr><td style="width: 88px;">&nbsp;</td><td style="width: 112px;"><p>&nbsp;pBAD33_fwd (7)</p><p>pBAD33_rv (8)</p></td><td style="width: 155px;"><p>SPD5_pBAD33_gibson_fw (9)</p><p>SPD5_1_gibson_rev (10)</p></td><td style="width: 184.517px;"><p>SPD5_2_gibson_fw (11)</p><p>SPD5_2_sfGFP_gibson_rev (12)</p></td><td style="width: 190.483px;"><p>sfGFP_SPD5_2_gibson_fw (13)</p><p>spGFP_pBAD33_gibson_rev (14)</p></td></tr></tbody></table><p>&nbsp;</p><p><strong>Program for pBAD33 Backbone</strong></p><table style="width: 324.1px; height: 291px;" border="Black"><tbody><tr style="height: 36px;"><td style="width: 86px; height: 36px;"><p>&nbsp;</p></td><td style="width: 74px; height: 36px;"><p>Temperature</p></td><td style="width: 74px; height: 36px;"><p>Duration [s]</p></td><td style="width: 59.1px; height: 36px;"><p>Cycles</p></td></tr><tr style="height: 36px;"><td style="width: 86px; height: 36px;"><p>Initial</p></td><td style="width: 74px; height: 36px;"><p>98&deg;C</p></td><td style="width: 74px; height: 36px;"><p>30</p></td><td style="width: 59.1px; height: 36px;"><p>1</p></td></tr><tr style="height: 36px;"><td style="width: 86px; height: 36px;"><p>Denaturation</p></td><td style="width: 74px; height: 36px;"><p>98&deg;C</p></td><td style="width: 74px; height: 36px;"><p>15</p></td><td style="width: 59.1px; height: 107px;" rowspan="3"><p>&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 30</p></td></tr><tr style="height: 35px;"><td style="width: 86px; height: 35px;"><p>Annealing</p></td><td style="width: 74px; height: 35px;"><p>56.1&deg;C</p></td><td style="width: 74px; height: 35px;"><p>30</p></td></tr><tr style="height: 36px;"><td style="width: 86px; height: 36px;"><p>Extension</p></td><td style="width: 74px; height: 36px;"><p>72&deg;C</p></td><td style="width: 74px; height: 36px;"><p>160</p></td></tr><tr style="height: 36px;"><td style="width: 86px; height: 36px;"><p>Final extension</p></td><td style="width: 74px; height: 36px;"><p>72&deg;C</p></td><td style="width: 74px; height: 36px;"><p>600 (10 min)</p></td><td style="width: 59.1px; height: 36px;"><p>1</p></td></tr><tr style="height: 36px;"><td style="width: 86px; height: 36px;"><p>Storage</p></td><td style="width: 74px; height: 36px;"><p>4&deg;C</p></td><td style="width: 74px; height: 36px;"><p>infinite</p></td><td style="width: 59.1px; height: 36px;"><p>1</p></td></tr></tbody></table><p>&nbsp;</p><p>&nbsp;<strong>Program for SPD5_1, SPD5_2, sfGFP_linker</strong></p><table style="width: 315px;" border="Black"><tbody><tr><td style="width: 80px;"><p>&nbsp;</p></td><td style="width: 77px;"><p>Temperature</p></td><td style="width: 75px;"><p>Duration [s]</p></td><td style="width: 55px;"><p>Cycles</p></td></tr><tr><td style="width: 80px;"><p>Initial</p></td><td style="width: 77px;"><p>98&deg;C</p></td><td style="width: 75px;"><p>30</p></td><td style="width: 55px;"><p>1</p></td></tr><tr><td style="width: 80px;"><p>Denaturation</p></td><td style="width: 77px;"><p>98&deg;C</p></td><td style="width: 75px;"><p>15</p></td><td style="width: 55px;" rowspan="3"><p>&nbsp;</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 30</p></td></tr><tr><td style="width: 80px;"><p>Annealing</p></td><td style="width: 77px;"><p>Gradient &deg;C</p></td><td style="width: 75px;"><p>30</p></td></tr><tr><td style="width: 80px;"><p>Extension</p></td><td style="width: 77px;"><p>72&deg;C</p></td><td style="width: 75px;"><p>70</p></td></tr><tr><td style="width: 80px;"><p>Final extension</p></td><td style="width: 77px;"><p>72&deg;C</p></td><td style="width: 75px;"><p>300</p></td><td style="width: 55px;"><p>1</p></td></tr><tr><td style="width: 80px;"><p>Storage</p></td><td style="width: 77px;"><p>4&deg;C</p></td><td style="width: 75px;"><p>infinite</p></td><td style="width: 55px;"><p>1</p></td></tr></tbody></table><p>&nbsp;</p><p><strong>Pippeting scheme:</strong></p><p>For the fragments 1&micro;l was used as the concentration was unknown, and 10ng for the Backbone.</p><table style="width: 308px;" border="Black"><tbody><tr><td style="width: 85px;"><p>&nbsp;</p></td><td style="width: 74px;"><p>Volume [&micro;l]</p></td><td style="width: 60px;"><p>stock</p></td><td style="width: 61px;"><p>final</p></td></tr><tr><td style="width: 85px;"><p>Q5 mastermix</p></td><td style="width: 74px;"><p>12,5</p></td><td style="width: 60px;"><p>2x</p></td><td style="width: 61px;"><p>1x</p></td></tr><tr><td style="width: 85px;"><p>Fw primer</p></td><td style="width: 74px;"><p>1,25</p></td><td style="width: 60px;"><p>10&micro;M</p></td><td style="width: 61px;"><p>0,5&micro;M</p></td></tr><tr><td style="width: 85px;"><p>Rv primer</p></td><td style="width: 74px;"><p>1,25</p></td><td style="width: 60px;"><p>10&micro;M</p></td><td style="width: 61px;"><p>0,5&micro;M</p></td></tr><tr><td style="width: 85px;"><p>DNA template</p></td><td style="width: 74px;"><p>10ng</p></td><td style="width: 60px;"><p>&nbsp;</p></td><td style="width: 61px;"><p>&nbsp;</p></td></tr><tr><td style="width: 85px;"><p>Water</p></td><td style="width: 74px;"><p>&nbsp;rest</p></td><td style="width: 60px;"><p>&nbsp;</p></td><td style="width: 61px;"><p>&nbsp;</p></td></tr><tr><td style="width: 85px;"><p>Total</p></td><td style="width: 74px;"><p>25</p></td><td style="width: 60px;"><p>&nbsp;</p></td><td style="width: 61px;"><p>&nbsp;</p></td></tr></tbody></table></p>

      <br><b>31/05-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Gel for PCR fragments SPD5_1, SPD5_2, linkeR_sfGFP </strong>(1% agarose gel, 1kb plus quick load ladder)</p><table style="width: 340px;" border="BLACK"><tbody><tr style="height: 36px;"><td style="height: 36px; width: 46px;"><p>Marker</p></td><td style="height: 36px; width: 73px;"><p>SPD5_1</p></td><td style="height: 36px; width: 102px;"><p>SPD5_2</p></td><td style="height: 36px; width: 91px;"><p>Linker_sfGFP</p></td></tr></tbody></table><p>&nbsp;--&gt; All fragments were amplified!</p>

      <br><b>01/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Gel for PCR fragment pBAD33 backbone </strong>(0,7% agarose gel, 1kb plus quick load ladder)</p><table style="width: 183px;" border="black"><tbody><tr><td style="width: 61px;"><p>Marker</p></td><td style="width: 106px;"><p>PBAD33 backbone</p></td></tr></tbody></table><p>--&gt; All fragments were amplified!</p><p>&nbsp;</p><p><strong>HiFi Gibson Assembly of SPD5_sfGFP (pIG22_007)</strong></p><p>&nbsp;See protocol #020. Because the concentrations from the cleanup of the PCR fragments were too low to do an assembly with 0.2 pmol we did:</p><p>Total amount of DNA= 0.05 pmol</p><p>Ratio of vector: insert 1:2</p><p>Backbone: 0.017pmol</p><p>Per fragment: 0.017pmol/3 = 0.00567 pmol</p><p>&nbsp;</p><p><strong>Example of calculations:</strong></p><p>Pmol*(basepairs x 650 daltons) / 1000 = __ ng</p><table border="black"><tbody><tr><td width="131"><p>&nbsp;</p></td><td width="83"><p>Amount</p></td><td width="71"><p>Volume</p></td></tr><tr><td width="131"><p>SPD5_1</p></td><td width="83"><p>7.48 ng</p></td><td width="71"><p>1.41 &micro;l</p></td></tr><tr><td width="131"><p>SPD5_2</p></td><td width="83"><p>5.9 ng</p></td><td width="71"><p>1.18 &micro;l</p></td></tr><tr><td width="131"><p>sfGFP</p></td><td width="83"><p>2.76 ng</p></td><td width="71"><p>0.31 &micro;l</p></td></tr><tr><td width="131"><p>pBAD33</p></td><td width="83"><p>59.16 ng</p></td><td width="71"><p>1.22 &micro;l</p></td></tr><tr><td width="131"><p>Water</p></td><td width="83"><p>&nbsp;</p></td><td width="71"><p>5.88 &micro;l</p></td></tr><tr><td width="131"><p>HiFi 2X assembly</p></td><td width="83"><p>&nbsp;</p></td><td width="71"><p>10 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>&nbsp;</p><p><strong>Heat shock trafo</strong> of SPD5_sfGFP_pBAD33 in competent Top10: 50&micro;l Bacteria, 1.5&micro;l Plasmid. See protocol #003. Cells were plated on 2 LB + amp plates (200 &micro;l and 800 &micro;l) + left at incubator&nbsp; 37 &deg;C.</p>

      <br><b>02/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Clean-up of PCR fragments from gel</strong></p>
      <p>Fragments that were cut out and purified: SPD5_1, SPD5_2, sfGFP, pBAD33. Protocol #021</p><br>
      <br><p><strong>Heat shock trafo</strong> of SPD5_sfGFP_pBAD33 in competent Top10: We used 50 &micro;l Bacteria, 1.5 &micro;l assembled pBAD33_SPD5_sfGFP. See protocol #003. Cells were plated on 2 LB + Chl plates (200&micro;l and 800&micro;l) + left at incubator 37 &deg;C.</p>
      <p>&nbsp;</p>

      <br><b>04/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Colony PCR of the pIG22_007 </strong>(pBAD_SPD5_sfGFP) (colony 1- 12)</p><p>In order to check if the overnight cultures worked we prepared a colony PCR of all 12 colonies by amplifying the insert between primer (9) and (14). This should be in total 4,3kb.</p><table style="width: 265px;" border="black"><tbody><tr><td style="width: 111px;"><p>&nbsp;</p></td><td style="width: 82px;"><p>Temperature</p></td><td style="width: 50px;"><p>rounds</p></td></tr><tr><td style="width: 111px;"><p>Initial denaturation</p></td><td style="width: 82px;"><p>98&deg;C</p></td><td style="width: 50px;"><p>1</p></td></tr><tr><td style="width: 111px;"><p>Denaturation</p></td><td style="width: 82px;"><p>98&deg;C</p></td><td style="width: 50px;" rowspan="3"><p>&nbsp;</p><p>30</p></td></tr><tr><td style="width: 111px;"><p>Anealing</p></td><td style="width: 82px;"><p>64,5&deg;C</p></td></tr><tr><td style="width: 111px;"><p>Extension</p></td><td style="width: 82px;"><p>72&deg;C</p></td></tr><tr><td style="width: 111px;"><p>Final extension</p></td><td style="width: 82px;"><p>72&deg;C</p></td><td style="width: 50px;"><p>1</p></td></tr><tr><td style="width: 111px;"><p>Storage</p></td><td style="width: 82px;"><p>4&deg;C</p></td><td style="width: 50px;"><p>1</p></td></tr></tbody></table><p>&nbsp;</p><p>Here we use the Q5 polymerase as this was what we had in the lab</p><table style="width: 365px;" border="black"><tbody><tr><td style="width: 197px;"><p>&nbsp;</p></td><td style="width: 75px;"><p>1 rxn</p></td><td style="width: 71px;"><p>15 rxn</p></td></tr><tr><td style="width: 197px;"><p>5X Q5 Reaction Buffer</p></td><td style="width: 75px;"><p>5 &micro;l</p></td><td style="width: 71px;"><p>75&micro;l</p></td></tr><tr><td style="width: 197px;"><p>10 mM dNTPs</p></td><td style="width: 75px;"><p>0.5 &micro;l</p></td><td style="width: 71px;"><p>7.5 &micro;l</p></td></tr><tr><td style="width: 197px;"><p>10 &micro;M Forward primer</p></td><td style="width: 75px;"><p>1.25 &micro;l</p></td><td style="width: 71px;"><p>18.75 &micro;l</p></td></tr><tr><td style="width: 197px;"><p>10 &micro;M Reverse primer</p></td><td style="width: 75px;"><p>1.25 &micro;l</p></td><td style="width: 71px;"><p>18.75 &micro;l</p></td></tr><tr><td style="width: 197px;"><p>Template</p></td><td style="width: 75px;"><p>1 colony</p></td><td style="width: 71px;"><p>-</p></td></tr><tr><td style="width: 197px;"><p>Q5 polymerase HiFi</p></td><td style="width: 75px;"><p>0.25 &micro;l</p></td><td style="width: 71px;"><p>3.75 &micro;l</p></td></tr><tr><td style="width: 197px;"><p>Q5 high GC enhancer</p></td><td style="width: 75px;"><p>5 &micro;l</p></td><td style="width: 71px;"><p>75 &micro;l</p></td></tr><tr><td style="width: 197px;"><p>Nuclease free water</p></td><td style="width: 75px;"><p>10.75 &micro;l</p></td><td style="width: 71px;"><p>161.25 &micro;l</p></td></tr></tbody></table><p>We added 5 ml to the overnight cultures and left them in the fridge as we won&rsquo;t do the miniprep today.</p>

      <br><br><b>05/06-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;Gel of the <strong>colony PCRs</strong> on 0,7% agarose, 1kb plus Quick load ladder</p><table border="black"><tbody><tr><td width="59"><p>Marker</p></td><td width="44"><p>c1</p></td><td width="44"><p>c2</p></td><td width="44"><p>c3</p></td><td width="44"><p>c4</p></td><td width="44"><p>c5</p></td><td width="44"><p>c6</p></td><td width="44"><p>c7</p></td><td width="44"><p>c8</p></td><td width="44"><p>c9</p></td><td width="44"><p>c10</p></td><td width="44"><p>c11</p></td><td width="44"><p>c12</p></td></tr></tbody></table><p>None of the colonies had a band at 4,3 kb instead they all had a band at 0,5 kb and 1 lower.</p><p>&nbsp;</p><p><strong>Colony PCR of the pIG22_007</strong> (c13-c24): We decide to do another colony PCR from 12 other colonies. Similar to the one from yesterday. However this time we ran out of Q5 polymerase, so it wasn't enough. We ran the PCR anyway.</p><p>&nbsp;</p><p><strong>HiFi Gibson Assembly of SPD5_sfGFP</strong> (pIG22_007):&nbsp; We also decided to repeat the assembly of the plasmid with extractions of the fragments with higher concentrations. This time we did following:</p><p>&nbsp;</p><p>Total amount of DNA= 0.2pmol</p><p>Ratio of vector: insert 1:1</p><p>0.2pmol / 2 parts = 0.1 pmol</p><p>Backbone: 0.1pmol</p><p>Per fragment: 0.1pmol/ 3 = 0.033 pmol</p><table border="black"><tbody><tr><td width="132"><p>SPD5_1</p></td><td width="76"><p>8.22 &micro;l</p></td></tr><tr><td width="132"><p>SPD5_2</p></td><td width="76"><p>0.3 &micro;l</p></td></tr><tr><td width="132"><p>sfGFP</p></td><td width="76"><p>0.6 &micro;l</p></td></tr><tr><td width="132"><p>pBAD33</p></td><td width="76"><p>2.36 &micro;l</p></td></tr><tr><td width="132"><p>HiFi 2X assembly</p></td><td width="76"><p>10 &micro;l</p></td></tr></tbody></table><p>Note: The total volume of DNA was a bit more than 10 &micro;l.</p><p>&nbsp;</p><p><strong>Heat shock trafo</strong> of pIG22_007 in competent Top10. We used 50 &micro;l bacteria and 1.5 &micro;l of assembled plasmid. See protocol #003</p><p>Cells were plated on 3 LB + Chl plates (20 &micro;l, 200 &micro;l and 780 &micro;l) + left at incubator 37 &deg;C</p><p>&nbsp;</p>

      <br><b>06/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong><span data-contrast="auto">Colony PCR of the pIG22_007 (c1-c12)</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></strong></p><p><span data-contrast="auto">Same protocol as last week. But this time we test the new Gibson assembly (0,2pmol 1:1 ratio).</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><p><span data-contrast="auto">Gel of colony PCRs: </span><span data-contrast="auto">The bands are again at 0,5 kb and not at 4,3 kb.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><p><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><p><strong><span data-contrast="auto">Miniprep of c1 of pIG22_007</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></strong></p><p><span data-contrast="auto">We spin down 5 ml. The concentration is very low. Conc: 29,7 ng/&micro;l</span><span data-contrast="auto">. See protocol #008</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><p>&nbsp;</p><p><strong><span data-contrast="auto">Restriction digest of pIG22_007</span> : </strong><span data-contrast="auto">In order to check if the reason we get wierd bands is because something with the PCR/primers we make a digestion of the plasmid to compare with the undigested. P</span><span data-contrast="auto">rotocol #022</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><table data-tablestyle="MsoTableGrid" data-tablelook="1696"><tbody><tr><td data-celllook="0"><p><span data-contrast="auto">DNA (c1 of pIG22_007)</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td><td data-celllook="0"><p><span data-contrast="auto">33 &micro;l</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td></tr><tr><td data-celllook="0"><p><span data-contrast="auto">10X NEB CutSmart</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td><td data-celllook="0"><p><span data-contrast="auto">5 &micro;l</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td></tr><tr><td data-celllook="0"><p><span data-contrast="auto">XbaI</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td><td data-celllook="0"><p><span data-contrast="auto">1 &micro;l</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td></tr><tr><td data-celllook="0"><p><span data-contrast="auto">water</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td><td data-celllook="0"><p><span data-contrast="auto">Up to 50 &micro;l</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}"> <br /></span></p></td></tr></tbody></table><p><span data-contrast="auto">This was incubated 15 min at 37C.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><p><span data-contrast="auto">&nbsp;</span></p><p><strong><span data-contrast="auto">Gel for comparison of digested and undigested pIG22_007</span>: </strong><span data-contrast="auto">We prepared a 0,7% agarose gel and used the 1kb+ Quick load ladder.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><table data-tablestyle="MsoTableGrid" data-tablelook="1696"><tbody><tr><td data-celllook="0"><p><span data-contrast="auto">Marker</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td><td data-celllook="0"><p><span data-contrast="auto">Undigested</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><p><span data-contrast="auto">10&micro;l miniprep</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td><td data-celllook="0"><p><span data-contrast="auto">Digested</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p><p><span data-contrast="auto">25 &micro;l of digest</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}">&nbsp;</span></p></td></tr></tbody></table><p><span data-contrast="auto">&nbsp;</span><span data-contrast="auto">The band on the gel looked like it was about 5 kb, which would fit with just the backbone.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559740&quot;:259}"> We decide to stop working with the old primers from the 2019 Freiburg team and instead design our own new primers.<br /></span></p>


      <br><b>29/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did another <strong>apmlification of the SPD5_1 fragment</strong> with the new 2022 primers. This is done to test different annealing temperatures of the new primers as we have had difficulties to amplify via the new primers. 12 PCRs were prepared (25 &micro;l) and temperatures between 60 - 66&deg;C were tested.</p>
      <p>Result: Annealing temperature at 64&deg;C gives bands and we continue with this temperature for future PCRs. Bands at the right size (2000bp), were cut out and purified via gel extraction.</p>
      <p>Conc.: 8 ng/&micro;l</p>
      <p>&nbsp;</p>

      <b>30/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>PCR amplification of SPD5_1</strong>: As we used all the PCR amplified SPD5_1, we first need to amplify it with the 2019 primers, before we can amplify it with optimized conditions and new 2022 primers.</p>
      <p>&nbsp;50 &micro;l HIFI master mix (Primers: 9 and 10)</p>
      <p>Annealing: 64,5 &deg;C, Extension time 1 min 10 sec</p>
      <p><strong>Gel extraction of agarose bands:</strong></p>
      <p>SPD5_1: 29 ng/&micro;l</p>

      <br><b>01/07-22</b><br>
      <div class="line-2"> </div>
      <p>For the SPD5_1 we did another <strong>PCR</strong> with the 2022 primers to get the right overlaps for the Gibson assembly.</p>
      <p>-&gt; Same conditions as for the previous PCRs</p>
      <p>&nbsp;</p>
      <p><strong>Gel extraction</strong>: Bands with the right size for SPD5_1 (2000 bp) were cut out and purified:</p>
      <p>SPD5_1: 30 ng/&micro;l and 20.4 ng/&micro;l</p>

      <br><b>02/07-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;</p><p><strong>Gibson of pBAD_sfGFP, SPD5_1 and SPD5_2</strong></p><p>According to protocol 020#. Total amount of DNA: 0,2 pmol (Ratio 1:1)</p><p>&nbsp;</p><p>Vector: 0,067 pmol</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; PBAD33_sfGFP: 269,5 ng</p><p>Inserts : 0,067 pmol</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SPD5_1: 87,5 ng</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SPD5_2: 69,4 ng</p><p>Add water to 10 &micro;l and 10 &micro;l HIFI assembly mix.</p><p>--&gt; This was to low a ratio and too much DNA as 50-100 ng of the backbone is recommended.</p><p>&nbsp;</p><p>A <strong>heat shock transformation</strong> of the Gibson was performed with 1&micro;l into 50 &micro;l Top10 cells</p><p>&nbsp;</p>

      <br><b>03/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Plates from yesterday:</strong> the Gibson of SPD5 did not seem to work.</p><ul><li>Next time higher ratio, lower amount of vector</li></ul>

      <br><b>10/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Plates from trafo from yesterday</strong>: 5 colonies</p><p>&nbsp;</p><p><strong>PCR of SPD5_2</strong> to amplify it as I am running out of the fragment</p><p>2x 25&micro;l using 2 X Mastermix, annealing at 64&deg;C extend for 1min 10sec</p><p>--&gt;Gel showed a small band which is cut out and stored at 4 &deg;C overnight.</p><p>&nbsp;</p><p><strong>Gibson of SPD5 and pBAD</strong> with fresh assemblymix (ratio1:3)</p><table border="black"><tbody><tr style="height: 36px;"><td style="height: 36px;" width="170"><p>SPD5_1</p></td><td style="height: 36px;" width="76"><p>2.28 &micro;l</p></td></tr><tr style="height: 36px;"><td style="height: 36px;" width="170"><p>SPD5_2</p></td><td style="height: 36px;" width="76"><p>1.38 &micro;l</p></td></tr><tr style="height: 36px;"><td style="height: 36px;" width="170"><p>pBAD33_PCR fragment</p></td><td style="height: 36px;" width="76"><p>0.38 &micro;l</p></td></tr><tr style="height: 36px;"><td style="height: 36px;" width="170"><p>water</p></td><td style="height: 36px;" width="76"><p>6 &micro;l</p></td></tr><tr style="height: 36px;"><td style="height: 36px;" width="170"><p>Hifi Assembly mix (NEB)</p></td><td style="height: 36px;" width="76"><p>10 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>incubate 1 h at 50 &deg;C</p><p>&nbsp;</p><p><strong>Transformation</strong> of 2.5 &micro;l SPD5 gibson into top10</p><p>&nbsp;all was plated on LB+Cm plates</p><p>&nbsp;</p><p>&nbsp;</p>

      <br><b>11/07-22</b><br>
      <div class="line-2"> </div>
      <strong>Plates from yesterday:</strong></p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SPD5 gibson: nothing</p><p>&nbsp;</p><p><strong>Clean up from gel piece of SPD5_2 from yesterday:</strong></p><p>Kit: protocol #. Eluted in water.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Conc: 37 ng/&micro;l</p><p>&nbsp;</p><p><strong>PCR of SPD5_whole (both SPD5_1 and SPD5_2) </strong></p><p>Using primer 17 and 18, and both fragments as templates.</p><p>&nbsp;</p><p>Expected size: 3.6 kb, annealing temperature 62.6 &deg;C, 63.9&deg;C, 64.9 &deg;C. Extension time 2 min.</p><p>Using Q5 HIFI mastermix (NEB) 25&micro;l/rxn, 15 cycles.</p><p>&nbsp;</p><p>Gel showed only smeared band around 1600 bp. Probably, the conditions were not optimal for amplifying SPD5_1.</p><p>&nbsp;</p><p><strong>PCR of SPD5_whole (both SPD5_1 and SPD5_2) </strong></p><p>Using primer 17 and 18, and both fragments as templates.</p><p>&nbsp;</p><p>Expected size: 3.6 kb, annealing temperature 64.5 &deg;C. Extension time 2 min.</p><p>Using Q5 with GC enhancer (NEB) 25&micro;l/rxn, 15 cycles.</p><p>&nbsp;</p><p>Gel showed a bigger band, which could be that SPD5_1 was also amplified, but maybe the number of cycles was not high enough.</p>


      <br><b>12/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Electoporation</strong> of SPD5 gibson 1:3 ratio from previous was via electroporation</p>
      <p>1&micro;l gibson in 50 &micro;l bacteria.</p>


      <br><b>13/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Plate with electroporation of SPD5:</strong> no colonies</p>
      <p>&nbsp;</p>
      <p><strong>PCR of both SPD5_1 and SPD5_2</strong></p>
      <p>Using primer:</p>
      <p>SPD5_1: 10 and 17</p>
      <p>SPD5_2: 11 and 18</p>
      <p><strong>Expected size</strong>: 2 kb and 1.6 kb respectively, annealing temperature 64 &deg;C. Extension time 1min 10sec.</p>
      <p>Using Q5 with GC enhancer (NEB) 50&micro;l/rxn, 30 cycles. For each 2 x 50 &micro;l was prepared.</p>

      <br><b>18/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Miniprep</strong> of SPD5_2 in pJET colony 18 + colony 20</p>
      <p>Send both colonies for sequencing with eurofins and primer 11 + 18</p>
      <p>-&gt;result: C18 had the SPD5 and did not have any problems with the sequence</p>

      <br><b>20/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Gel of PCR</strong> of 3 colonies of pBAD_SPD5 using primer 21 + 22 (binding at the promoter an the terminal)</p>
      <ul>
      <li>Gel showed band for all samples with backbone but no band for samples using top10 cells.</li>
      </ul>
      <p>I realized that there is a blurry band underneath the marker and that what I thought was 2000 bp was really the 1000 bp mark.</p>
      <p> I continue with C 6 and C11 as they have a band at actually 1000bp, which should be correct.</p>
      <p>&nbsp;</p>
      <p><strong>Electoporation</strong> of SPD5_gibson ratio 1:3</p>
      <p>I try again an old Gibson to see if another electroporation would give any results.</p>
      <p>1 &micro;l of Gibson + 50 &micro;l electrocompetent cells + 950 &micro;l after 1 h at 37&deg;C it was plated on a LB + CmR plate.</p>

      <br><b>21/07-22</b><br>
      <div class="line-2"> </div>
      The plate from yesterday had 3 colonies.

      <br><b>22/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight culture</strong> of pJET_SPD5_2 from plate C18</p><p>&nbsp;</p><p><strong>Colony PCR of the 3 colonies pBAD_SPD5 with primer 17 + 18 </strong></p><p>Dreamtaq Mastermix, Annealing temp: 65 &deg;C, extension time 4 min, 30 cycles</p><p>&nbsp;</p><table><tbody><tr><td width="85"><p>Marker</p><p>1 kb</p><p>Jena Bioscience</p></td><td width="62"><p>Other samples</p></td><td width="62"><p>C1</p></td><td width="63"><p>C2</p></td><td width="63"><p>C3</p></td></tr></tbody></table><p>&nbsp;</p>

      <br><b>23/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>PCR of SPD5_2 using pJET_SPD5_2 as template. </strong>I made 5 x 25 &micro;l amplifying it with primer 11+18, annealing temp: 64&deg;C, extension time: 1 min 10 sec</p><p>&nbsp;</p><p><strong>PCR of SPD5_1</strong> using different gel extractions from different days to test if any of them works for amplifying the fragment. I made 4 x 25 &micro;l and amplified using primer 9 + 10. Annealing: 64,5&deg;C, extension time 1min 10 sec.</p><p>&nbsp;</p><p>Gel of PCRs</p><table><tbody><tr><td width="83"><p>Marker</p></td><td width="69"><p>SPD5_2</p></td><td width="69"><p>SPD5_2</p></td><td width="69"><p>SPD5_2</p></td><td width="69"><p>SPD5_2</p></td><td width="69"><p>SPD5_2</p></td><td width="69"><p>SPD5_2</p></td><td width="36"><p>SPD5_1</p></td><td width="36"><p>SPD5_1</p></td><td width="36"><p>SPD5_1</p></td><td width="36"><p>SPD5_1</p></td></tr><tr><td width="83"><p>Jena Bioscience 1 kb</p></td><td width="69"><p>Band</p></td><td width="69"><p>Band</p></td><td width="69"><p>Band</p></td><td width="69"><p>Band</p></td><td width="69"><p>Band</p></td><td width="69"><p>Band</p></td><td width="36"><p>IDT, no band</p></td><td width="36"><p>No band</p></td><td width="36"><p>Band</p></td><td width="36"><p>No band</p></td></tr></tbody></table><p>&nbsp;</p><p>The SPD5_2 bands were cutout for purification, 3 together on one column.</p><p>The SPD5_1 band was cut out and extracted using gel extraction kit.</p>

      <br><b>24/07-22</b><br>
      <div class="line-2"> </div>
      <p>Another <strong>PCR</strong> with SPD5_1 using the template which gave bands on the previous PCR. I made 1 x 25 &micro;l. Annealing: 64,5 &deg;C and extension: 1 min 10 sec, 30 cycles.</p><p>&nbsp;</p><table border="BLACK"><tbody><tr><td width="73"><p>Marker</p></td><td width="56"><p>SPD5_1</p></td><td width="56"><p>pBAD_SPD5_c1</p></td><td width="56"><p>pBAD_SPD5_c2</p></td><td width="56"><p>pBAD_SPD5_c3</p></td></tr><tr><td width="73"><p>Jena Bioscience 1 kb</p></td><td width="56"><p>Band</p></td><td width="56"><p>No band</p></td><td width="56"><p>Band</p></td><td width="56"><p>No band</p></td></tr></tbody></table><p>&nbsp;</p><p>The SPD5_1 band was cut out and cleaned up with the piece from yesterday using <strong>gel extraction</strong>.</p><p>Conc: 24 ng/ul eluted in 10 &micro;l</p><p>&nbsp;</p><p>This was enough to make a <strong>cloning into pJET</strong> using #__ and a 1:3 ratio</p><table border="BLACK"><tbody><tr><td width="141"><p>Reaction buffer</p></td><td width="113"><p>10 &micro;l</p></td></tr><tr><td width="141"><p>pJET blunt end</p></td><td width="113"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>Insert SPD5_1</p></td><td width="113"><p>7,7 &micro;l (185 ng)</p></td></tr><tr><td width="141"><p>water</p></td><td width="113"><p>0,3 &micro;l</p></td></tr><tr><td width="141"><p>T4-ligase</p></td><td width="113"><p>1 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>This was ligated at RT 5min and 5 &micro;l was directly used for <strong>heat shock trafo</strong> into Top10 cells. The complete volume was plated on a LB + Amp plate O/N.</p><p>&nbsp;</p><p>Since there was a band for the c2 from the pBAD_SPD5 at 10 kb I also make an <strong>overnight culture</strong> of this to check the plasmid with sequencing. 10 ml LB +CmR.</p>

      <br><b>25/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Miniprep</strong> of c2 from the pBAD_SPD5 overnight culture: Conc: ~60 ng/&micro;l.</p>
      <p><br />&nbsp;</p>
      <p><strong>Colony PCR</strong> of 20 colonies from pJET_SPD5_1. Using standard primers for pJET and the Green Dreamtag Mastermix. Annealing at 52&deg;C and extension was 2 min, 30 cycles.</p>
      <p>&nbsp;</p>
      <p><strong>Gel 1% agarose</strong>, Jena Bioscience 1 kb ladder. The gel showed multiple colonies with a band at 2000 bp.</p>
      <p>Three were chosen for following <strong>overnight culture</strong>: C2,5,9 in 10 ml LB + Amp.</p>
      <p><strong>Sequencing </strong>using Genewiz and standard primers for pBAD I send the C2 from pBAD_SPD5 using standard primers.</p>
      <p>Result: The pBAD_SPD5 only had the SPD5_2 inserted</p>

      <br><b>27/07-22</b><br>
      <div class="line-2"> </div>
      Miniprep of pJET_SPD5_1 colonies and the colonies c2, 5, 9 were send for sequencing using standard primers.
      <p>Results from <strong>sequencing</strong> from the pJET_SPD5_1 colonies: all had some kind of mutation.</p>
      <p>As we ordered the SPD5_1 again from fresh (the other was from 2019) we will continue with the new fragment.</p>

      <br><b>28/07-22</b><br>
      <div class="line-2"> </div>
      <p>Gel of sfGFP_TP from 2<sup>nd</sup> PCR, 100 bp plus thermos fischer.</p><table><tbody><tr><td width="90"><p>Ladder</p></td><td width="96"><p>sfGFP_TP</p></td><td width="96"><p>sfGFP_TP</p></td><td width="96"><p>sfGFP_TP</p></td><td width="96"><p>sfGFP_TP</p></td><td width="96"><p>sfGFP_TP</p></td><td width="74"><p>sfGFP_TP</p></td></tr></tbody></table><p>&nbsp;</p><p>Bands were all at the right size and were cutout out and purified using gel-extraction.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Conc: 260 ng/&micro;l</p><p>&nbsp;</p><p>Digest of #1924 with BlpI and KplnI:</p><table><tbody><tr><td width="141"><p>DNA #1924 (1 &micro;g)</p></td><td width="76"><p>17.45 &micro;l</p></td></tr><tr><td width="141"><p>KpnI-HF</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>BlpI</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>cutsmart</p></td><td width="76"><p>5 &micro;l</p></td></tr><tr><td width="141"><p>water</p></td><td width="76"><p>25.55 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>Digest of sfGFP_TP from PCR 2</p><table><tbody><tr><td width="141"><p>DNA sfGFP_TP (1 &micro;g)</p></td><td width="76"><p>3.86 &micro;l</p></td></tr><tr><td width="141"><p>KpnI-HF</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>BlpI</p></td><td width="76"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>cutsmart</p></td><td width="76"><p>5 &micro;l</p></td></tr><tr><td width="141"><p>water</p></td><td width="76"><p>39.3 &micro;l</p></td></tr></tbody></table><p><strong>&nbsp;</strong></p><p>All digests were incubated at 37&deg;C for 1 h.</p><p>&nbsp;</p><p>After 1 h digests were run on gels:</p><p>Gel 1:</p><table width="627"><tbody><tr><td width="63"><p>Marker:</p><p>ZR 1kb ladder</p></td><td width="82"><p>pBAD undigested</p></td><td width="67"><p>pBAD digested</p></td><td width="67"><p>pBAD digested</p></td><td width="82"><p>1924</p><p>undigested</p></td><td width="67"><p>1924</p><p>digested</p></td><td width="67"><p>1924</p><p>digested</p></td><td width="67"><p>sfGFP</p><p>digested</p></td><td width="67"><p>sfGFP digested</p></td></tr></tbody></table><p>All the digested backbones and the inserts were cut out and purified using gel-extraction.</p><p>&nbsp;</p><table style="height: 120px;" width="576"><tbody><tr><td style="width: 108.4px;"><p>Digest</p></td><td style="width: 108.4px;"><table><tbody><tr><td width="107"><p>sfGFP</p></td><td width="107"><p>1294</p></td></tr></tbody></table></td></tr><tr><td style="width: 108.4px;"><p>Conc.</p></td><td style="width: 108.4px;"><table><tbody><tr><td width="107"><p>29 ng/&micro;l</p></td><td width="107"><p>13 ng/&micro;l</p></td></tr></tbody></table></td><td style="width: 108.4px;">&nbsp;</td><td style="width: 108.4px;">&nbsp;</td><td style="width: 108.4px;">&nbsp;</td></tr></tbody></table><p>&nbsp;</p><p>&nbsp;</p>

      <br><b>31/07-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight cultures</strong> were prepared of 4 of the colonies from the pJET_SPD5_1 (new fragment) </p>

      <br><b>01/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Miniprep</strong> of the overnight cultures from yesterday was prepared 4 from pJET_SPD5_1.</p>
      <p>--&gt;The colonies from the minipreps were all send for <strong>sequencing</strong> using eurofins and standard primers.</p>

      <br><b>02/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Sequencing results from pJET_SPD51</strong>: There are an insert for all colonies and colony nr. 4 also has the right sequence with no mutations.</p>

      <br><b>03/08-22</b><br>
      <div class="line-2"> </div>
      <p>.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>&nbsp;</p><p>For pJET_SPD5_1 c4 a <strong>transformation</strong> is made to have a clean colony to pick for a glycerol stock.</p><p>&nbsp;</p><p><strong>PCR</strong> was prepared using the pJET_SPD5_1 c4 as a template for amplifying the SPD5_1 using the primers 10 + 17. Annealing temperature: 64,5&deg;C, extension time 1 min 10 sec, 30 cycles. Prepared with Q5 polymerase. This is done to achieve the correct overhangs for the following Gibson.</p><p>&nbsp;</p><p>An <strong>agarose gel </strong>was prepared of the two above PCR, RZ 1kb ladder.</p><table width="100%"><tbody><tr><td width="8%"><p>Marker</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="9%"><p>sfGFP_TP</p></td><td width="2%"><p>&nbsp;</p></td><td width="8%"><p>SPD5_1</p></td><td width="8%"><p>SPD5_1</p></td><td width="8%"><p>SPD5_1</p></td><td width="8%"><p>SPD5_1</p></td><td width="8%"><p>SPD5_1</p></td></tr></tbody></table><p>&nbsp;</p><p>The bands for the respective fragments were cut out and purified with gel extraction.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SPD5_1: 139 ng/&micro;l</p><p><strong>&nbsp;</strong></p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p>

      <br><b>04/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Gibson was prepared</strong> for inserting the SPD5_1 and SPD5_2 fragments into pBAD33.</p><p>Ratio 1:2</p><p>I will use 100ng of the backbone, which is pBAD33 amplified with two primers. As the backbone is 5,3 kb this is 30.5 fmol. As the ratio is 1:2 the amount of insert will be 61 fmol. From the concentrations and lengths of my fragments I get:</p><p>&nbsp;</p><table><tbody><tr><td width="141"><p>SPD5_1 (2000 bp)</p></td><td width="85"><p>75.4 ng</p></td><td width="76"><p>0.54 &micro;l</p></td></tr><tr><td width="141"><p>SPD5_2 (1600bp)</p></td><td width="85"><p>60 ng</p></td><td width="76"><p>1.25 &micro;l</p></td></tr><tr><td width="141"><p>pBAD33 (5.3 kb)</p></td><td width="85"><p>100 ng</p></td><td width="76"><p>0.5 &micro;l</p></td></tr><tr><td width="141"><p>Water</p></td><td width="85"><p>&nbsp;</p></td><td width="76"><p>7.7 &micro;l</p></td></tr><tr><td width="141"><p>HiFi Assembly mix</p></td><td width="85"><p>&nbsp;</p></td><td width="76"><p>10 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>This incubate 1 h at 50 &deg;C and then 1.5 &micro;l of the Gibson mix was used for a trafo with 10 &micro;l competent bought Dh5-alpha cells. And plated on Chloramphenicol plates.</p><p>&nbsp;<strong>Overnight cultures</strong> of the&nbsp; pJET_SPD5_1 for making glycerol stocks.</p><p><strong>&nbsp;</strong></p><p><strong>&nbsp;</strong></p>

      <br><b>05/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Colony PCR</strong> was made of colonies from the 6 colonies I had an the pBAD_SPD5 plate, using Thermo fischer green mastermix. The primers were the T7-primers. Annealing was at 52&deg;C and extension tim was 5 min. 30 cycles.</p>
      <p>The colony PCR was run on<strong> an agarose gel </strong>with the RZ 1 kb ladder. The colonies c1 and c2 showed a band at the expected size ~4,3 kb.</p>
      <p><strong>Glycerol stocks</strong> were prepared for &nbsp;pJET_SPD5_1</p>

      <br><b>06/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight cultures</strong> were prepared of the two colonies C1 and C2 of pBAD_SPD5 which showed to have an insert in the colony PCR.</p>

      <br><b>07/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Minipreps</strong> were prepare of the SPD5 c1 and c2, eluted in 30 &micro;l:</p><table><tbody><tr><td width="75"><p>Conc.</p></td><td width="132"><p>C1: 183 ng/&micro;l</p></td><td width="113"><p>C2: 33 ng/&micro;l</p></td></tr></tbody></table><p>&nbsp;</p>

      <br><b>18/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>PCR of SPD5</strong> to produce different fragments with and without the Spy-tag and the Snoop-tag.</p>
      <p>First PCRs:</p>
      <p>SPD5_snoop: 34 + 26 anneal at 59&deg;C</p>
      <p>SPD5_spy: 35 + 24 anneal at 59&deg;C</p>
      <p>SPD5_both: 26 + 24 anneal at 59&deg;C</p>

      <br><b>19/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Agarose gel</strong> of yesterday&rsquo;s PCR of different PCR amplified fragments of SPD5.</p><p>&nbsp;</p><table><tbody><tr><td width="128"><p>Marker</p></td><td width="128"><p>EncA_micro</p></td><td width="36"><p>X4</p></td><td width="94"><p>SPD5_micro</p></td><td width="28"><p>X4</p></td></tr><tr><td width="128"><p>GeneRuler 1kb</p></td><td width="128"><p>Bands</p></td><td width="36"><p>&nbsp;</p></td><td width="94"><p>Bands</p></td><td width="28"><p>&nbsp;</p></td></tr></tbody></table><p>&nbsp;</p><table><tbody><tr><td width="128"><p>Marker</p></td><td width="128"><p>SPD5_spy 1&deg;</p></td><td width="36"><p>X4</p></td><td width="94"><p>SPD5_snoop 1&deg;</p></td><td width="28"><p>X4</p></td></tr><tr><td width="128"><p>GeneRuler 1kb</p></td><td width="128"><p>Bands</p></td><td width="36"><p>&nbsp;</p></td><td width="94"><p>Bands</p></td><td width="28"><p>&nbsp;</p></td></tr></tbody></table><p>All bands were cut out and purified using <strong>gel-extraction</strong></p><p>SPD5_micro: 148 ng/&micro;l</p><p>&nbsp;</p><p><strong>Trafo </strong>of BAD_SPD5 into Bl21 and 2332 into Bl21 using heat-shock. 30 &micro;l was plated on LB + CmR plates.</p><p>&nbsp;</p><p><strong>Digest of pHT1_spy_snoopT2T3</strong></p><table><tbody><tr><td width="141"><p>pHT1spy_snoopT2T3</p></td><td width="94"><p>1000ng (5 &micro;l)</p></td></tr><tr><td width="141"><p>NdeI</p></td><td width="94"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>XhoI</p></td><td width="94"><p>1 &micro;l</p></td></tr><tr><td width="141"><p>Cutsmart buffer</p></td><td width="94"><p>5 &micro;l</p></td></tr><tr><td width="141"><p>Water</p></td><td width="94"><p>38 &micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p><strong>Digest of SPD5</strong></p><table><tbody><tr><td width="113"><p>SPD5</p></td><td width="104"><p>1000ng (6.7 &micro;l)</p></td></tr><tr><td width="113"><p>NdeI</p></td><td width="104"><p>1 &micro;l</p></td></tr><tr><td width="113"><p>XhoI</p></td><td width="104"><p>1 &micro;l</p></td></tr><tr><td width="113"><p>Cutsmart buffer</p></td><td width="104"><p>5 &micro;l</p></td></tr><tr><td width="113"><p>Water</p></td><td width="104"><p>36.3 &micro;l</p></td></tr></tbody></table>

      <br><b>20/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight culture</strong> of BL21 with either pBAD_SPD5_sfGFP or pBAD_sfGFP were prepared.</p><p>&nbsp;</p><p><strong>Miniprep</strong> pMIC_SPD5_c1 and pHT1_spy/snoop_T2T3</p><p>&nbsp;</p><p>I ran the second <strong>PCR for cloning of the pMIC_SPD5_both</strong>, which contains both the spy and the snoop catcher. This was done using Q5 polymerase and the pimers 36 + 37 annealing at 64 &deg;C.</p><p>&nbsp;</p><p>An <strong>agarose gel</strong> was made to confirm the PCR fragments of SPD5_both. The band was at the expected size and was cut out and purified using <strong>gel-extraction</strong>.</p><p>&nbsp;</p><table><tbody><tr><td width="139"><p>Marker</p></td><td width="132"><p>SPD5_both 1&deg;</p></td><td width="41"><p>X4</p></td><td width="113"><p>SPD5_both 2&deg;</p></td><td width="38"><p>X5</p></td></tr><tr><td width="139"><p>GeneRuler 1kb</p></td><td width="132"><p>band</p></td><td width="41"><p>X4</p></td><td width="113"><p>band</p></td><td width="38"><p>X5</p></td></tr></tbody></table><p>&nbsp;</p><p><strong>Gel with digest</strong> from yesterday&rsquo;s digest of the backbone and the inserts for pMIC_SPD5 and pMIC_EncA</p><table width="100%"><tbody><tr><td width="13%"><p><strong>Marker</strong></p></td><td width="20%"><p><strong>pHT1spy_snoopT2T3 undigested</strong></p></td><td width="20%"><p><strong>pHT1spy_snoopT2T3 digest</strong></p></td><td width="4%"><p><strong>X3</strong></p></td><td width="9%"><p><strong>SPD5 digested</strong></p></td><td width="9%"><p><strong>SPD5 digested</strong></p></td><td width="10%"><p><strong>EncA digested</strong></p></td><td width="10%"><p><strong>EncA digested</strong></p></td></tr><tr><td width="13%"><p>GeneRuler 1kb</p></td><td width="20%"><p>band</p></td><td width="20%"><p>band</p></td><td width="4%"><p>X3</p></td><td width="9%"><p>band</p></td><td width="9%"><p>band</p></td><td width="10%"><p>band</p></td><td width="10%"><p>Band</p></td></tr><tr><td width="13%"><p><strong>Marker</strong></p></td><td width="20%"><p><strong>SPD5_snoop 2&deg;</strong></p></td><td width="20%"><p><strong>SPD5_spy 2&deg;</strong></p></td><td width="4%"><p>&nbsp;</p></td><td width="9%"><p>&nbsp;</p></td><td width="9%"><p>&nbsp;</p></td><td width="10%"><p>&nbsp;</p></td><td width="10%"><p>&nbsp;</p></td></tr><tr><td width="13%"><p>GeneRuler 1kb</p></td><td width="20%"><p>Band x4</p></td><td width="20%"><p>Band x 4</p></td><td width="4%"><p>&nbsp;</p></td><td width="9%"><p>&nbsp;</p></td><td width="9%"><p>&nbsp;</p></td><td width="10%"><p>&nbsp;</p></td><td width="10%"><p>&nbsp;</p></td></tr></tbody></table><p>&nbsp;</p><p>Clean up of digest from gel was performed for all bands using <strong>gel-extraction</strong></p><p>SPD5_snoop: 99 ng/&micro;l</p><p>SPD5_micro: 41.5 ng/&micro;l</p><p>pHT1spy_snoopT2T3: 45 ng/&micro;l</p>

      <br><b>21/08-22</b><br>
      <div class="line-2"> </div>
      <p>Clean up from gel of the SPD5_both from the first and the second PCR. This was done using gel-extraction.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SPD5_both 2&deg;: 136 ng/&micro;l</p><p>&nbsp;</p><p><strong>Ligation of pMIC_SPD5: </strong>Ratio was 1:2, 3 ligations were performed as described:</p><table><tbody><tr><td width="149"><p>&nbsp;</p></td><td width="96"><p>pMIC_SPD5</p></td><td width="103"><p>Religation</p></td></tr><tr><td width="149"><p>pHT1spy/snoopT2T3 digest (50ng)</p></td><td width="96"><p>1.1 &micro;l</p></td><td width="103"><p>1.1 &micro;l</p></td></tr><tr><td width="149"><p>Insert</p></td><td width="96"><p>2.36 &micro;l</p></td><td width="103"><p>&nbsp;</p></td></tr><tr><td width="149"><p>water</p></td><td width="96"><p>13.54 &micro;l</p></td><td width="103"><p>15.9 &micro;l</p></td></tr><tr><td width="149"><p>T4 buffer</p></td><td width="96"><p>2 &micro;l</p></td><td width="103"><p>2 &micro;l</p></td></tr><tr><td width="149"><p>T4 ligase</p></td><td width="96"><p>1 &micro;l</p></td><td width="103"><p>1 &micro;l</p></td></tr></tbody></table><p>All ligations were incubated 10 min at RT at heat inactivated for 10 min at 65&deg;C.</p><p>&nbsp;</p><p>&nbsp;</p><p><strong>Digest of SPD5_tags</strong></p><table><tbody><tr><td width="321"><p>DNA SPD5_snoop or SPD5_spy or SPD5_both</p></td><td width="113"><p>1000ng (10 &micro;l)</p></td></tr><tr><td width="321"><p>NdeI</p></td><td width="113"><p>1 &micro;l</p></td></tr><tr><td width="321"><p>XhoI</p></td><td width="113"><p>1 &micro;l</p></td></tr><tr><td width="321"><p>Cutsmart buffer</p></td><td width="113"><p>5 &micro;l</p></td></tr><tr><td width="321"><p>Water</p></td><td width="113"><p>33 &micro;l</p></td></tr></tbody></table><p>Incubate 1 h at 37&deg;C</p><p>&nbsp;</p><p>An <strong>agarose gel</strong> was run with the digests of the 3 digests and all had bands at the correct size. These were cut out and purified using <strong>gel extraction</strong>.</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SPD5_both: 43 ng/&micro;l</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SPD5_snoop: 60 ng/&micro;l</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SPD5_spy: 50 ng/&micro;l</p><p>&nbsp;</p><p><strong>Glycerol stocks</strong> were prepared of pBAD_SPD5_sfGFP and #2332 (pBAD_sfGFP) in BL21(DE3) for microscopy at a later timepoint. 1 ml of the overnight cultures were centrifuged and resuspended in 1:1 glycerol and LB.</p><p>&nbsp;</p>

      <br><b>22/08-22</b><br>
      <div class="line-2"> </div>
      <p><br /><br /></p><p><strong>Ligation of pMIC_SPD5</strong> with tags: 3 ligations were prepared with ratio 1:2 of vector:insert. For each ligation 50 ng of the backbone was used, corresponding to 22 fmol.</p><p>&nbsp;</p><p>pHT1(snoop/spy)T2T3 digest: 1,1 &micro;l</p><p>SPD5_snoop/spy: 44 fmol</p><p>T4 ligase: 1&micro;l</p><p>T4 ligase buffer: 2&micro;l</p><p>&nbsp;</p><p>Ligation time was 10 min at RT and followed by heat inactivation 10 min at 65&deg;C</p><p>&nbsp;</p><p><strong>Heat shock:</strong> Trafo was performed using 10 &micro;l of bought competent DH5-alpha and 2 &micro;l ligation mix. The total volume was plated og LB+Amp plates.</p><p>&nbsp;</p>

      <br><b>23/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Electoporation of the SPD5 with tags constructs.</strong> As the last trafo yesterday did not give many colonies I try the trafo again using electroporation to get more colonies.</p>
      <p>&nbsp;</p>
      <p>I <strong>send the pMIC_SPD5 c1 for sequencing</strong> using standard primers for the encapsulins and self-sent primers for SPD5</p>
      <p>Result: The primers for the SPD5 were wrong.</p>
      <p>&nbsp;</p>
      <p><strong>Miniprep</strong> of the overnight cultures from yesterday of pMIC_SPD5, eluted in 50 &micro;l using Sigma Aldrich kit.</p>
      <p>&nbsp;</p>

      <br><b>24/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Plates from yesterday&rsquo;s electroporation: </strong>there were 1 colony of the two SPD5_spy/snoop respectively, no colonies of the SPD5_snoop.</p><p>&nbsp;</p><p><strong>Colony PCR</strong> of the 2 colonies from SPD5_snoop/spy as well as the colonies from last trafo of the same ligation, total samples 9. Colony PCR was done using Green Dreamtaq Mastermix and primer 35+34.</p><p>&nbsp;</p><p><strong>Gel of colony PCR</strong></p><table><tbody><tr><td width="64"><p>Marker</p></td><td width="64"><p>C1</p></td><td width="64"><p>C2</p></td><td width="64"><p>C3</p></td><td width="64"><p>C4</p></td><td width="64"><p>C6</p></td><td width="64"><p>C7</p></td><td width="64"><p>C8</p></td><td width="64"><p>C9</p></td><td width="64"><p>C10</p></td></tr><tr><td width="64"><p>GeneRuler 1kb</p></td><td width="64"><p>-</p></td><td width="64"><p>-</p></td><td width="64"><p>-</p></td><td width="64"><p>-</p></td><td width="64"><p>-</p></td><td width="64"><p>-</p></td><td width="64"><p>-</p></td><td width="64"><p>-</p></td><td width="64"><p>-</p></td></tr></tbody></table><p>The gel showed no bands for any of the colonies.</p><p>&nbsp;</p><p><strong>Overnight ligation</strong>: As none of the colonies showed bands at the expected size, I decide to try to make an overnight ligation instead at 16&deg;C. 2 ligations were prepared with ratio 1:3 of vector:insert. For each ligation 50 ng of the backbone was used, corresponding to 22 fmol.</p><p>&nbsp;</p><p>pHT1(snoop/spy)T2T3 digest: 1,1 &micro;l</p><p>SPD5_snoop/spy: 66 fmol</p><p>T4 ligase: 1&micro;l</p><p>T4 ligase buffer: 2&micro;l</p><p>&nbsp;</p><p>Both ligations were incubated at 16&deg;C overnight.</p><p>&nbsp;</p>

      <br><b>25/08-22</b><br>
      <div class="line-2"> </div>
      <p>From the ligation from yesterday of pMIC_SPD5_Spy/snoop a <strong>heat-shock</strong> was performed with DH5-alpha and plated on LB + Amp plates.</p><p>&nbsp;</p><p><strong>Overnight cultures</strong> were prepared of pMIC_SPD5 c1 in 5 ml LB + Amp. They were incubated 25&deg;C for 24 h.</p><p>&nbsp;</p><p><strong>PCRs </strong>were prepared to amplify different parts of the SPD5 with and without tags. Primers and annealing temperatures were as described. This time I amplify the SPD5 in two parts, split at the SpeI restriction site. For fragments amplifying the tags 2 PCRs are needed:</p><p>SPD5_spy: 23+24, 60&deg;C</p><p>SPD5_snoop: 26 +28, 65&deg;C</p><p>SPD5_c-term: 28+35, 59&deg;C</p><p>SPD5_n-term 34+23, 59&deg;C</p><p>&nbsp;</p><p>All PCRs were run on an <strong>agarose gel </strong>and all samples took up 2 lanes. The bands were cut out and purified with gel-extraction</p><table><tbody><tr><td width="83"><p>Marker</p></td><td width="78"><p>SPD5_spy 1&deg;</p></td><td width="96"><p>SPD5_snoop 1&deg;</p></td><td width="71"><p>SPD5_n-term 1&deg;</p></td><td width="71"><p>SPD5_c-term 1&deg;</p></td></tr><tr><td width="83"><p>GeneRuler 1kb</p></td><td width="78"><p>23 ng/&micro;l</p></td><td width="96"><p>152 ng/&micro;l</p></td><td width="71"><p>139 ng/&micro;l</p></td><td width="71"><p>134 ng/&micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>For the <strong>second PCRs</strong> of the fragments SPD5_tags different primers were used to amplify it, with specified annealing temperatures:</p><p>SPD5_spy: 23+36, 64&deg;C</p><p>SPD5_snoop: 37 +28, 60&deg;C</p><p>&nbsp;</p><p>The PCRs were run on an <strong>agarose gel</strong>:</p><table><tbody><tr><td><p>Marker</p></td><td><p>SPD5_spy</p></td><td><p>X4</p></td><td><p>SPD5_snoop</p></td><td><p>X4</p></td></tr><tr><td><p>GeneRuler 1kb</p></td><td><p>Unspecific bands</p></td><td><p>&nbsp;</p></td><td><p>Unspecific bands</p></td><td><p>&nbsp;</p></td></tr></tbody></table><p>The gel was thrown out as there were multiple unspecific bands. The PCR would need optimization.</p><p>&nbsp;</p>

      <br><b>26/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight cultures</strong> of SPD5_snoop/spy were prepared</p>
      <p>&nbsp;</p>
      <p><strong>Miniprep </strong>of overnight cultures of pMIC_SPD5 c1 was prepared and they were subsequently sent for sequencing.</p>
      <p>&nbsp;</p>
      <p><strong>Transformation</strong> via heat-shock of SPD5_snoop/snpy into Dh5 alpha</p>
      <p>&nbsp;</p>
      <p><strong>Colony PCR</strong> of the SPD5_spy/snoop in DH5-alpha from yesterday.</p>
      <p>&nbsp;</p>

      <br><b>27/08-22</b><br>
      <div class="line-2"> </div>
      <p>From the overnight culture of SPD5_snoop/spy c2, 3 ml were spun down and stored at -20&deg;C.</p><p>&nbsp;</p><p><strong>Trafo</strong> of already done ligations, to screen more colonies: trafo of SPD5_snoop/spy and SPD5_both. 2 &micro;l were transformed into highly competent DH5-alpha. Afterwards, plated on LB+amp.&nbsp;</p><p>&nbsp;</p><p><strong>SPD5 #01</strong></p><p><strong>Aim:</strong> to check via fluorescent microscopy if there are fluorescent &ldquo;dots&rdquo; shown for the SPD5 like was shown in 2019. Different concentrations of arabinose are tested.</p><p>2 strains in BL21:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Strain 1: pBAD_sfGFP</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Strain 2: pBAD_SPD5_sfGFP</p><p>&nbsp;</p><table><tbody><tr><td width="128"><p>Samples</p></td><td width="128"><p>1/5</p></td><td width="128"><p>2/6</p></td><td width="128"><p>3/7</p></td><td width="128"><p>4/8</p></td></tr><tr><td width="128"><p>Arabinose %</p></td><td width="128"><p>0</p></td><td width="128"><p>0.0025%</p></td><td width="128"><p>0.005%</p></td><td width="128"><p>0.01%</p></td></tr></tbody></table><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p>

      <br><b>28/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight </strong>cultures from glycerol stock were prepared for the experiments SPD5 #01. Cultures are grown in LB.</p><p>&nbsp;</p><p><strong>Colony PCR of SPD5_both tags</strong> and SPD5_snoop/spy from yesterdays transformation. For this green dream-tag polymerase MM was used and the primers 34 and 35. Annealing temp: 55&deg;C and Extension time: 4min.</p><p>&nbsp;</p><p><strong>Colony PCR</strong> was run on an agarose gel with the ladder GeneRuler 1kb</p><p>&nbsp;</p><table><tbody><tr><td><p>Marker</p></td><td><p>C1</p></td><td><p>C2</p></td><td><p>C3</p></td><td><p>C5</p></td><td><p>C6</p></td><td><p>C7</p></td><td><p>C8</p></td><td><p>C9</p></td><td><p>C10</p></td></tr><tr><td><p>&nbsp;</p></td><td><p>&nbsp;</p></td><td><p>&nbsp;</p></td><td><p>&nbsp;</p></td><td><p>&nbsp;</p></td><td><p>&nbsp;</p></td><td><p>&nbsp;</p></td><td><p>&nbsp;</p></td><td><p>band</p></td><td><p>band</p></td></tr></tbody></table><p>&nbsp;</p><p>For 2 colonies a band at the correct size, 3.6 kb, was seen and those colonies came from the plate with SPD5_both tags.</p>

      <br><b>29/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Overnight culture</strong> of the SPD5_snoop C1 was spun down and the pellet was stored in -20&deg;C.</p>
      <p>&nbsp;</p>
      <p><strong>Overnight cultures</strong> of two colonies of SPD5_both were prepared and grown at 30&deg;C, C9 and C10.</p>

      <br><b>30/08-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Miniprep</strong> of the two colonies of SPD5_both C9 and C10 as well as the stored pellet of SPD5_snoop C1.</p><table><tbody><tr><td><p>SPD5_both c9</p></td><td><p>SPD5_both c10</p></td><td><p>SPD5_snoop c1</p></td><td><p>SPD5_spy c2</p></td></tr><tr><td><p>97 ng/&micro;l</p></td><td><p>97 ng/&micro;l</p></td><td><p>87 ng/&micro;l</p></td><td><p>71 ng/&micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p>Send SPD5_snoop c1 for<strong> sequencing</strong> using primers amplifying the ends.</p><p>&nbsp;</p><p><strong>Digests of multiple fragments:</strong></p><table border="black"><tbody><tr><td><p>Fragment/plasmid</p></td><td><p>DNA</p></td><td><p>Enzymes</p></td><td><p>Water</p></td><td><p>CutSmart</p></td></tr><tr><td><p>SPD5_both (36+37)</p></td><td><p>20 &micro;l</p></td><td><p>NdeI, XhoI</p></td><td><p>23 &micro;l</p></td><td><p>5 &micro;l</p></td></tr><tr><td><p>mVENUS backbone</p></td><td><p>21 &micro;l</p></td><td><p>BamHI, EcoRI</p></td><td><p>22 &micro;l</p></td><td><p>5 &micro;l</p></td></tr><tr><td><p>BMC backbone</p></td><td><p>7.7 &micro;l</p></td><td><p>NdeI, XhoI</p></td><td><p>33 &micro;l</p></td><td><p>5 &micro;l</p></td></tr></tbody></table><ul><li>Incubate all at 37&deg;C for 1 h</li></ul><p>&nbsp;</p><p>All Digests were loaded on a gel and subsequently cut out an purified using gel-extraction:</p><table><tbody><tr><td><p>Marker</p></td><td><p>mVENUS</p></td><td><p>SPD5_both</p></td><td><p>BMC</p></td></tr><tr><td><p>GeneRuler 1kb</p></td><td><p>6.8 ng/&micro;l</p></td><td><p>5.6 ng/&micro;l</p></td><td><p>5.9 ng/&micro;l</p></td></tr></tbody></table><p>&nbsp;</p><p><strong>Microscopy</strong> of the SPD5 #01 was performed.</p><p>However, the SPD5 showed no fluorescence what so ever, so there might be a problem with the glycerol stock.</p>

      <br><b>01/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Sequencing results:</strong> from the SPD5_both tag: the two colonies I sent for sequencing looks good. Two of the primers had no result, I will resend those for another sequencing.</p><p><strong>Small culture of SPD5_snoop</strong> c1 in LB + amp. After 6 hours 3 ml of the small culture was spun down and miniprep was prepared. The SPD5_snoop c1 was send for sequencing using 4x primers.</p><p>&nbsp;</p><p><strong>New 2&deg; PCR</strong> of SPD5_snoop using primer 34 + 37, using Q5 polymerase. Annealing temp: 59&deg;C and Extension: 4 min. This amplifies the whole SPD5, I choose this&nbsp; as the other method gave unspecific bands.</p><p>&nbsp;</p><p><strong>Digest of BMC backbone</strong></p><p>2000ng = 15&micro;l</p><p>NdeI 2&micro;l</p><p>XhoI 2&micro;l</p><p>Cutsmart 10 &micro;l</p><p>Water 71 &micro;l</p><p><strong>&nbsp;</strong></p><p><strong>Digest of SPD5_snoop fragment</strong></p><p>2000ng = 17.3 &micro;l</p><p>NdeI 2&micro;l</p><p>XhoI 2&micro;l</p><p>Cutsmart 10 &micro;l</p><p>Water 66.4 &micro;l</p><p>&nbsp;</p><p>All digests were incubated 1h at 37 &deg;C and then loaded on a gel. The digests were cut out and purified using gel-extraction</p><p>&nbsp;</p><p><strong>Ligation of SPD5_snoop</strong> 1:1 ratio</p><table><tbody><tr><td><p>BMC backbone (50 ng)</p></td><td><p>5 &micro;l</p></td></tr><tr><td><p>SPD5_snoop (50 ng)</p></td><td><p>3 &micro;l</p></td></tr><tr><td><p>T4 buffer</p></td><td><p>2 &micro;l</p></td></tr><tr><td><p>T4 ligase</p></td><td><p>1 &micro;l</p></td></tr><tr><td><p>Water</p></td><td><p>7 ml</p></td></tr></tbody></table><p>Ligation mix was kept at 4&deg;C overnight</p><p>&nbsp;</p><p><strong>&nbsp;</strong></p><p><strong>_______________________SPD5 #02_____________________</strong></p><p><strong>&nbsp;</strong></p><p><strong>Aim: </strong>to repeat the SPD5_sfGFP microscopy results, which were done in 2019 by the Freiburg iGEM team with our newly cloned construct. This time from freshly prepared strains, which I make sure contain the plasmids.</p><p>&nbsp;</p><p>Strains are prepared in BL21 and are transformed with either pBAD_sfGFP or pBAD_SPD5_sfGFP</p><p>&nbsp;</p><table><tbody><tr><td width="104"><p>Sample</p></td><td width="57"><p>1/4</p></td><td width="57"><p>2/5</p></td><td width="38"><p>3/6</p></td></tr><tr><td width="104"><p>Arabinose (%)</p></td><td width="57"><p>0</p></td><td width="57"><p>0.005</p></td><td width="38"><p>0.1</p></td></tr></tbody></table><p><strong>&nbsp;____________________________________________________</strong></p>

      <br><b>02/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Sequencing results</strong> of SPD5_both showed that both C9 and C10 looked good and had no mutations. I will continue with C9 for future experiments.</p><p>&nbsp;</p><p><strong>Transformations</strong> via heat-shock:</p><p>Ligation of pMIC_SPD5_snoop into DH5-alpha</p><p>pMIC_SPD5 + pBbA2c_mVenus_ SpyCatcher (mVENUS)</p><p>pMIC_SPD5_both + mVENUS</p><p>pMIC_SPD5 + mVENUS + mTurquoise</p><p>pMIC_SPD5_both + mVENUS + pBbA2k_mTurquoise2-SnoopCatcher (mTurquoise)</p><p>&nbsp;</p><p><strong>Colony PCR</strong> of new colonies of the trafo of pBAD_sfGFP or pBAD_SPD5_sfGFP. This was done using Green Dream-tag polymerase MM using primers 14 +16 to amplify sfGFP.</p><p>3 colonies of pBAD_sfGFP was screened and 5 colonies of pBAD_SPD5_sfGFP as well as a positive control using the pBAD_SPD5_sfGFP plasmid. Besides, the current not-working glycerol stock were screened.</p><p>&nbsp;</p><p><strong>Colony PCR</strong> of SPD5_snoop C1 re-trafo colonies. This was done using Green Dream-tag polymerase MM using primers&nbsp; 34 + 37 to amplify SPD5_snoop.</p><p>&nbsp;</p><p><strong>Agarose gel of colony PCRs:</strong></p><p>Gel 2: marker, pBAD_sfGFP (x3), pBAD_SPD5_sfGFP (x5), positive control, Glycerol stock (x2)</p><p>Gel 3: marker sfGFP (primer 38 + 39) (x8)</p><p>&nbsp;</p><p><strong>Miniprep</strong> of mVENUS in TOP10, 3 ml were used for the miniprep with the sigma Aldrich kit.</p><p><strong>Overnight culture</strong> of one the screened pBAD_sfGFP/pBAD_SPD5_sfGFP colonies was prepared for microscopy on Sunday and for glycerol stock.</p>

      <br><b>03/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>SPD5_#02:</strong> Preparation of 50 ml larger culture from the overnight culture (1:50) and grown to OD 0.6 at 30&deg;C and shortly at 37&deg;C. Induction was performed according to previous scheme from a 20% arabinose stock:</p><p>0,1%: 15 &micro;l</p><p>0,005%: 0.75&micro;l</p><p>Then samples were incubated at 18&deg;C overnight. 2 additional samples were prepared with 0,1% arabinose but incubated at 37&deg;C overnight.</p><p><strong>&nbsp;</strong></p><p><strong>Heat shock transformation</strong> was performed of; pMIC_SPD5_both into TOP10, pMIC_SPD5 + mTURQ, pMIC_SPD5_both + mTURQ.</p>

      <br><b>04/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Glycerol stock</strong> was prepared from the overnight cultures of pMIC_SPD5 + mVEN, pMIC_SPD5_both + mVEN, pMIC_SPD5 + mVEN + mTUR, pMIC_SPD5_both + mVEN +mTUR.</p>
      <p>&nbsp;</p>
      <p><strong>Overnight cultures</strong> were prepared of the trafos from yesterday of pMIC_SPD5_both into TOP10, pMIC_SPD5 + mTURQ, pMIC_SPD5_both + mTURQ.</p>
      <p>&nbsp;</p>
      <p><strong>Colony PCR of colonies</strong> from the overnight ligation of PMIC_SPD5_snoop, this was done using Dream-taq green MM and primers 34 + 37 at annealing 55&deg;C and extension 4 min.</p>
      <p>&nbsp;</p>

      <br><b>05/09-22</b><br>
      <div class="line-2"> </div>
        <p><strong>Glycerol stocks</strong> of yesterdays overnight cultures and pMIC_SPD5 + mTURQ, pMIC_SPD5_both + mTURQ</p><p>&nbsp;</p><p><strong>Miniprep</strong> of the colonies for pMIC_SPD5_both was prepared an afterwards send for sequencing.</p><p>&nbsp;</p><p><strong>Induction </strong>of the pMIC_SPD5 + mTURQ, pMIC_SPD5_both + mTURQ was prepared according to previous description</p><p>&nbsp;</p><p><strong>_______________________SPD5 #03_______________________________________________</strong></p><p><strong>&nbsp;</strong></p><p><strong>Aim: </strong>to repeat the SPD5_sfGFP microscopy results, which were done in 2019 by the Freiburg iGEM team with our newly cloned construct. Repeat of SPD5 #02 because the last time I thought I made a mistake by induction.</p><p>&nbsp;</p><p>Strains are prepared in BL21 and are transformed with either pBAD_sfGFP or pBAD_SPD5_sfGFP</p><p>&nbsp;</p><table><tbody><tr><td width="104"><p>Sample</p></td><td width="57"><p>1/4</p></td><td width="57"><p>2/5</p></td><td width="38"><p>3/6</p></td></tr><tr><td width="104"><p>Arabinose (%)</p></td><td width="57"><p>0</p></td><td width="57"><p>0.005</p></td><td width="38"><p>0.1</p></td></tr></tbody></table><p>&nbsp;</p><p>&nbsp;</p>

        <br><b>06/09-22</b><br>
        <div class="line-2"> </div>
        <p>&nbsp;</p>
        <p><strong>Overnight cultures</strong> from yesterday of pBAD_SPD5_sfGFP/pBAD_sfGFP and pBAD33+pET_sfGFP/pBAD33 + pET_sfGFP_TP were dilutes 1:50 in larger cultures and grown at 30&deg;C until they reach OD: 0.6</p>
        <p><strong>Microscopy</strong> of pMIC_SPD5 + fluorescent catcher</p>

      <br><b>07/09-22</b><br>
      <div class="line-2"> </div>
      Microscopy of pBAD_SPD5 #03

      <br><b>11/09-22</b><br>
      <div class="line-2"> </div>
      <p>Heat shock transformations of the following was performed:</p><table border="black"><tbody><tr><td><p><strong>Strain #</strong></p></td><td><p><strong>Plasmids</strong></p></td><td><p><strong>Bacteria</strong></p></td></tr><tr><td><p>01</p></td><td><p>pMIC_SPD5 + mVENUS</p></td><td><p>MG1655</p></td></tr><tr><td><p>02</p></td><td><p>pMIC_SPD5_both + mVENUS</p></td><td><p>MG1655</p></td></tr><tr><td><p>03</p></td><td><p>pMIC_SPD5 + mTURQUOISE</p></td><td><p>MG1655</p></td></tr><tr><td><p>04</p></td><td><p>pMIC_SPD5_both + mTURQUOISE</p></td><td><p>MG1655</p></td></tr><tr><td><p>05</p></td><td><p>pBAD_SPD5_sfGFP</p></td><td><p>MG1655</p></td></tr><tr><td><p>06</p></td><td><p>pBAD_sfGFP</p></td><td><p>MG1655</p></td></tr><tr><td><p>09</p></td><td><p>pMIC_SPD5 + pBAD_sfGFP (#2332)</p></td><td><p>BL21</p></td></tr><tr><td><p>10</p></td><td><p>pMIC_SPD5_both + pBAD_sfGFP (#2332)</p></td><td><p>BL21</p></td></tr><tr><td><p>11</p></td><td><p>pMIC_SPD5_both + pBAD_sfGFP (#2332)</p></td><td><p>MG1655</p></td></tr><tr><td><p>12</p></td><td><p>pMIC_SPD5 + pBAD_sfGFP (#2332)</p></td><td><p>MG1655</p></td></tr></tbody></table>

      <br><b>12/09-22</b><br>
      <div class="line-2"> </div>
      Overnight cultures of transformations strains were prepared

      <div class="line-2"> </div>
      <p><strong>SPD5 #04</strong></p><p>Aim: To see if there is a difference in the dots which are formed in BL21 and in MG1655.</p><p>Experimental setup: MG1655 is transformed with either pBAD_SPD5_sfGFP (Strain 2) or pBAD_sfGFP (Strain 1) and from an overnight culture a larger culture is grown to OD: 0.6 and induced according to the scheme.</p><p>&nbsp;</p><p>Strain 1: samples 1-4</p><p>Strain 2: samples 5-8</p><p>&nbsp;</p><table border="black"><tbody><tr><td width="107"><p>Sample</p></td><td width="107"><p>1 / 5</p></td><td width="107"><p>2 / 6</p></td><td width="107"><p>3 / 7</p></td><td width="107"><p>4 / 8</p></td></tr><tr><td width="107"><p>Arabinose (%)</p></td><td width="107"><p>0</p></td><td width="107"><p>0.005</p></td><td width="107"><p>0.01</p></td><td width="107"><p>0.1</p></td></tr></tbody></table>
      <div class="line-2"> </div>

      <br><b>13/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Glycerol stocks</strong> of all overnight cultures was prepared.</p>
      <p>Grew a larger culture (50 ml) of the 1:50 diluted overnight culture pBAD_SPD5_sfGFP or pBAD_sfGFP in MG1655 and induced at OD: 0.6 according to SPD5 #04 scheme. Then they were incubated at 18 &deg;C.</p>

      <br><b>14/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Microscopy</strong> of the prepared samples of pBAD_SPD5_sfGFP and pBAD_sfGFP, this time in MG1655.</p>
      <p> This time the drops could move!</p>

      <br><b>15/09-22</b><br>
      <div class="line-2"> </div>
      <p>Send pBAD_SPD5_sfGFP for <strong>sequencing</strong> the promoter region to look for mutations in the region causing it to be constitutively expressed.</p>

      <br><b>16/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Sequencing result</strong> of SPD5_sfGFP from yesterday showed for the PETUP primer that it could be a mix of two different plasmids, but the primer I send showed a cleaner result. However, there seems to be a deletion in front of the AraC, which could influence how well the AraC is translated.</p><p>&nbsp;</p><p><strong>Heat-shock transformation</strong> of</p><p>pMIC_EncA + pVEN:sfGFP_TP in ME5119&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (AmR + CmR)</p><p>1939 + pVEN_sfGFP_TP in ME5119&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (AmR + CmR)</p><p>pMIC_SPD5 + mVEN in C321∆exp&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (AmR + CmR)</p><p>pMIC_SPD5_both + mVEN in C321∆exp&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (AmR + CmR)</p><p>pMIC_SPD5 + mVEN in Genome Reduced&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (AmR + CmR)</p><p>pMIC_SPD5_both + mVEN in Genome Reduced&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; (AmR + CmR)</p><p>&nbsp;</p><p><strong>Microscopy</strong> of Encapsulin #04</p><p>Prepare <strong>overnight cultures</strong> for the microscopy on Sunday from the MG1655</p><p>pMIC_SPD5 + mVEN</p><p>pMIC_SPD5_both + mVEN</p><p>pMIC_SPD5 + mTURQ</p><p>pMIC_SPD5_both + mTURQ</p>

      <div class="line-2"> </div>
      <p><strong>pMIC_SPD5_spy/snoop #02</strong></p><p><strong>&nbsp;</strong></p><p><strong>Aim</strong>: to check whether the pMIC_SPD5 with and without tags, look like the SPD5_sfGFP in regards of being dynamics and if the catching seems to work.</p><p>&nbsp;</p><p><strong>Experimental setup: </strong>all strains in MG1655</p><p>&nbsp;</p><p>Strain A1: SPD5 + mVEN</p><p>Strain A2: SPD5_both + mVEN</p><p>Strain B1: SPD5 + mTURQ</p><p>Strain B2: SPD5_both + mTURQ</p><p>&nbsp;</p><table style="width: 724px;" border="b"><tbody><tr><td style="width: 48px;"><p>Sample</p></td><td style="width: 50px;"><p>1</p></td><td style="width: 50px;"><p>2</p></td><td style="width: 50px;"><p>3</p></td><td style="width: 50px;"><p>4</p></td><td style="width: 50px;"><p>5</p></td><td style="width: 50px;"><p>6</p></td><td style="width: 50px;"><p>7</p></td><td style="width: 50px;"><p>8</p></td><td style="width: 50px;"><p>9</p></td><td style="width: 48px;"><p>10</p></td><td style="width: 48px;"><p>11</p></td><td style="width: 48px;"><p>12</p></td></tr><tr><td style="width: 48px;"><p>IPTG (&micro;M)</p></td><td style="width: 50px;"><p>0</p></td><td style="width: 50px;"><p>0</p></td><td style="width: 50px;"><p>0</p></td><td style="width: 50px;"><p>10</p></td><td style="width: 50px;"><p>50</p></td><td style="width: 50px;"><p>100</p></td><td style="width: 50px;"><p>10</p></td><td style="width: 50px;"><p>50</p></td><td style="width: 50px;"><p>100</p></td><td style="width: 48px;"><p>10</p></td><td style="width: 48px;"><p>50</p></td><td style="width: 48px;"><p>100</p></td></tr><tr><td style="width: 48px;"><p>Doxy (ng/&micro;l)</p></td><td style="width: 50px;"><p>0</p></td><td style="width: 50px;"><p>10</p></td><td style="width: 50px;"><p>25</p></td><td style="width: 50px;"><p>0</p></td><td style="width: 50px;"><p>0</p></td><td style="width: 50px;"><p>0</p></td><td style="width: 50px;"><p>10</p></td><td style="width: 50px;"><p>10</p></td><td style="width: 50px;"><p>10</p></td><td style="width: 48px;"><p>25</p></td><td style="width: 48px;"><p>25</p></td><td style="width: 48px;"><p>25</p></td></tr></tbody></table><p>&nbsp;</p><p>Not all samples were microscoped, only the following based on initial microscopy.</p><p>&nbsp;</p><p>Strain A: 10, 4, 3, 1</p><p>Strain B: 8, 5, 2, 1</p>
      <div class="line-2"> </div>
      <p><strong>SPD5 #05</strong></p><p><strong>&nbsp;</strong></p><p><strong>Aim:</strong> To see if there is a difference in the dots which are formed in BL21 and in MG1655. <strong>Experimental setup:</strong> MG1655 is transformed with either pBAD_SPD5_sfGFP (Strain 2) or pBAD_sfGFP (Strain 1) and from an overnight culture a larger culture is grown to OD: 0.6 and induced according to the scheme. This time one concentration lower than previous was also performed.</p><p>&nbsp;</p><p>Strain 1: samples 1-3</p><p>Strain 2: samples 4-6</p><p>&nbsp;</p><table style="width: 456px;" border="b"><tbody><tr><td style="width: 107px;"><p>Sample</p></td><td style="width: 107px;"><p>1 / 4</p></td><td style="width: 107px;"><p>2 / 5</p></td><td style="width: 107px;"><p>3 / 6</p></td></tr><tr><td style="width: 107px;"><p>Arabinose (%)</p></td><td style="width: 107px;"><p>0</p></td><td style="width: 107px;"><p>0.001</p></td><td style="width: 107px;"><p>0.005</p></td></tr></tbody></table>
      <div class="line-2"> </div>

      <br><b>17/09-22</b><br>
      <div class="line-2"> </div>
      Induction of pMIC_SPD5_spy/snoop + Fluorescent protein #02 according to previously described
      
      <br><b>18/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Microscopy</strong> of pMIC_SPD5_spy/snoop #02</p>

      <br><b>19/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Induction</strong> of samples for FRAP pBAD_sfGFP and pBAD_SPD5_sfGFP accordingly</p>

      <br><b>20/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>FRAP + fluorescent microscopy</strong> of pBAD_sfGFP and pBAD_SPD5_sfGFP</p>

      <br><b>22/09-22</b><br>
      <div class="line-2"> </div>
      <p><strong>SDS</strong> of the pMIC_SPD5_spy/snoop #02 (12% SDS gel prestained protein ladder plus, thermos fischer)</p>
      <p>&nbsp;</p>
      <p>Transfer via semidry (Run for 1 h at constant voltage 10V)</p>
      <p>&nbsp;</p>
      <p>Blocking with 10% milk powder PBS-T 1X</p>
      <p>&nbsp;</p>
      <p>Primary antibody anti-GFP (Rabbit)</p>

      <br><b>23/09-22</b><br>
      <div class="line-2"> </div>
      Secondary antibody anti-Rabbit (HRP) + image<br>

      SDS of the pMIC_SPD5_spy/snoop #0 this time (8% SDS gel, Blue classic protein ladder, Jena)
      
      <br>Transfer via wetblot (0.001 A/membrane constant for 2 h, room temp)<br>

       <br><b>24/09-22</b><br>
      <div class="line-2"> </div><br>
      Blocking and Primary antibody anti-GFP (Rabbit)

      <br><b>24/09-22</b><br>
      <div class="line-2"> </div><br>
      Secondary antibody anti-Rabbit (HRP) + image
      <br>

      
      <!-- _______________________________ Microcompartments ______________________________ -->
      <h2 id='link_labbook_microcopartments'>Microcompartments</h2>
          <div class="line-1"> </div>

          <br><b>16/06-22</b><br>
          <div class="line-2"> </div>
          <p>Preparing 250 ml <strong>LB medium</strong> following the protocol #001 and adding 250&micro;l K and 250&micro;l Sm&nbsp;</p>
          <p>&nbsp;</p>
          <p>Preparing 250 ml <strong>LB agar</strong> following the protocol #004 and adding 250&micro;l K and 250&micro;l Sm </p>
          <p>&nbsp;</p>
          <p>Picking 2 colonies from the E.coli strain with the plasmid pHT1 and 2 from the one with the plasmid pHT1T2T3 --&gt; <strong>overnight culture</strong> in LB-medium + Amp</p>

          <br><b>17/06-22</b><br>
          <div class="line-2"> </div>
          <p><strong>Glycerol stocks</strong> of all overnight cultures using protocol #008&nbsp;</p>
          <p>&nbsp;</p>
          <p><strong>Miniprep</strong> of all overnight cultures using protocol #007&nbsp;</p>

          <br><b>22/06-22</b><br>
          <div class="line-2"> </div>
          <p><u>BMCs</u>:</p>
          <ol style="padding-left: 30px;">
          <li>SnpCat-mTurquoise2&nbsp; KanR</li>
          <li>SpyCat-mVenus2&nbsp; CmR</li>
          <li>pHT1SnpISpyTagT2T3&nbsp; AmpR</li>
          <li>pHT1SnpISpyTag&nbsp; AmpR</li>
          </ol>
          <p><strong>Transformation</strong> in BL21 of:</p>
          <p><u>BMCs</u>:&nbsp;&nbsp; a + b + c</p>
          <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a + b + d</p>

          <br><b>23/06-22</b><br>
          <div class="line-2"> </div>
          <p><strong>Transformation</strong> in BL21 of:</p>
          <p><u>BMCs</u>:&nbsp;&nbsp; a + b + c</p>
          <p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; a + b + d</p>

          <br><b>24/06-22</b><br>
          <div class="line-2"> </div>
          <p><strong>Overnight culture</strong> of :</p>
          <ul style="padding-left: 30px;">
          <li><u>BMCs</u>:&nbsp;&nbsp; a + b + c and a + b + d (in +K+Cm+Amp medium)</li>
          </ul>

          <br><b>25/06-22</b><br>
          <div class="line-2"> </div>
          <p><strong>Glycerol stocks</strong> of the overnight cultures of the previous day (2 each):</p>
          <ul style="padding-left: 30px;">
          <li><u>BMCs</u>:&nbsp;&nbsp; a + b + c and a + b + d (in +K+Cm+Amp medium)</li>
          </ul>

          <br><b>29/06-22</b><br>
          <div class="line-2"> </div>
          <p>BMCs: we want to prove that they can form and we also want to make them visible</p>
          <p>We made 4 <strong>transformations</strong> with BL21 competent cells from Nicole with following plasmids, following the protocol #003:</p>
          <ul style="padding-left: 30px;">
          <li>pHT1 + mVenus2-spyCat</li>
          <li>pHT1T2T3 + mVenus2-spyCat</li>
          <li>pHt1snpspyTag + mVenus2-spyCat</li>
          <li>pHT1snpspyTagT2T3 + mVenus2-spyCat</li>
          </ul>
          <p>We let the bacteria grow on CmR + AmpR plates at 37&deg;C.</p>

          <br><b>30/06-22</b><br>
          <div class="line-2"> </div>
          <p><strong>Overnight cultures</strong> of all transformations and let them grow at 30&deg;C.</p>
          <p>Prepare medium for the induction next day and also let it at 30&deg;C to have a constant temperature.</p>
          <p>--&gt; 5 ml LB + 5 &micro;l Cm + 5 &micro;l Amp pro tube</p>
          <p>--&gt;100 ml LB + 100 &micro;l Cm + 100 &micro;l Amp pro Erlenmayer glas</p>

          <br><b>01/07-22</b><br>
          <div class="line-2"> </div>
          <p>We <strong>induced the bacteria</strong> of all trafos. First we prepared the bacteria as following:</p><ul style="padding-left: 30px;"><li>Centrifuge 2 ml at 3000 g for 100 min in an eppi in order to get a 1:50 culture in a 100 ml LB</li><li>Throw supernatent away and resuspend bacteria in little amount of the LB that has been prepared on the previous day, then put all the bacteria into the 100 ml culture</li><li>Prepare a cuvette with just LB as blank and measure the OD600 of the bacteria until they have reached 0,6-0,8</li><li>Distribute the 100 ml culture into 9 tubes for the 9 induction conditions</li></ul><p><strong>Induction conditions:</strong> we used a 100 mM stock solution of IPTG for the BMCs and a 50 &micro;g/ml stock solution of doxycycline for Venus.</p><table><tbody><tr><td width="50"><p>&micro;M IPTG</p></td><td width="25"><p>0</p></td><td width="32"><p>0</p></td><td width="38"><p>0</p></td><td width="32"><p>50</p></td><td width="38"><p>100</p></td><td width="32"><p>50</p></td><td width="38"><p>100</p></td><td width="38"><p>50</p></td><td width="38"><p>100</p></td></tr><tr><td width="50"><p>ng/ml DOX.</p></td><td width="25"><p>0</p></td><td width="32"><p>50</p></td><td width="38"><p>200</p></td><td width="32"><p>0</p></td><td width="38"><p>0</p></td><td width="32"><p>50</p></td><td width="38"><p>50</p></td><td width="38"><p>200</p></td><td width="38"><p>200</p></td></tr></tbody></table><p>Samples:</p><ul style="padding-left: 30px;"><li>pHT1 + mVenus2-spyCat: samples n.1-9</li><li>pHT1T2T3 + mVenus2-spyCat: samples n.10-18</li><li>pHt1snpspyTag + mVenus2-spyCat: samples n.19-27</li><li>pHT1snpspyTagT2T3 + mVenus2-spyCat: samples n.28-36</li></ul><p>24h incubation at 18&deg;C</p>

          <br><b>02/07-22</b><br>
          <div class="line-2"> </div>
          <p>We prepared the cells for <strong>fluorescence microscopy</strong> as following:</p>
          <ul style="padding-left: 30px;">
          <li>Take 1 ml bacteria from induced cultures</li>
          <li>Centrifuge them at 8000 rpm for 1 min and throw supernatent away</li>
          <li>Resuspend in 10-20 &micro;l PBS (wash)</li>
          <li>Centrifuge again and resuspend again</li>
          <li>Prepare the agar pads for each sample --&gt; fluorescence microscopy</li>
          </ul>
          <p>We also<strong> prepared the cells for SDS</strong> as following:</p>
          <ul style="padding-left: 30px;">
          <li>Take 1 ml bacteria from induced cultures</li>
          <li>Centrifuge them at 3000g for 10 min and throw supernatent away</li>
          <li>Freeze the pellet for the next day</li>
          </ul>

          <br><b>03/07-22</b><br>
          <div class="line-2"> </div>
          <p>We <strong>prepared the cells for sonication</strong> by resuspending the pellet in 1 ml lysis buffer (20 mM Tris + 50 mM NaCl)</p><p>--&gt; Sonication of all the samples</p><p>Centrifuge all the samples at 3000 rpm for 10 min</p><p><strong>Prepare samples for SDS</strong>:</p><ul style="padding-left: 30px;"><li>Pipette 10 &micro;l supernatent of each sample together with 10 &micro;l loading buffer (Laemmli buffer)</li><li>Prepare gel by mixing and polymerizing separating gel and stacking gel</li></ul style="padding-left: 30px;"><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p><strong><u>Separating gel (12,5%)</u></strong></p><p>1,33 ml H2O</p><p>1,5 ml Tris/HCl 1,88 M pH 8,8</p><p>1,5 ml SDS 0,5% w/v (1 g per 100 ml)</p><p>3,13 ml Acrylamide (37, 5:1)</p><p>7,5 &micro;l TEMED</p><p>37,5 &micro;l APS (10% w/v)</p><p>&nbsp;</p><p>Invert the tube before pouring the gel in the chamber</p><p>Add 500 &micro;l isopropanol on top and let it polymerize</p><p>&nbsp;</p><p><strong><u>Stacking gel (5%)</u></strong></p><p>1,06 ml H2O</p><p>0,5 ml Tris/HCl pH 6,8</p><p>0,5 ml SDS 0,5% w/v</p><p>0,42 ml Acrylamide</p><p>2,5 &micro;l TEMED</p><p>12,5 &micro;l APS</p><p>&nbsp;Invert the tube</p><p>Pour the isopropanol away and give the stacking gel on top of the polymerized separating gel</p><p><strong>&nbsp;</strong></p><p>We loaded all the samples in the chambers as in the table below, poured the running buffer and let the SDS PAGE run at 120V with 0,08 Amper</p><p>&nbsp;</p><table><tbody><tr><td width="61"><p>marker</p></td><td width="61"><p>marker</p></td><td width="31"><p>1</p></td><td width="31"><p>3</p></td><td width="31"><p>5</p></td><td width="31"><p>7</p></td><td width="31"><p>x</p></td><td width="31"><p>9</p></td><td width="31"><p>x</p></td><td width="22"><p>x</p></td></tr><tr><td width="61"><p>marker</p></td><td width="61"><p>10</p></td><td width="31"><p>11</p></td><td width="31"><p>13</p></td><td width="31"><p>14</p></td><td width="31"><p>15</p></td><td width="31"><p>16</p></td><td width="31"><p>17</p></td><td width="31"><p>18</p></td><td width="22"><p>x</p></td></tr></tbody></table><p>&nbsp;</p><p>We stained the gels with Roti blue overnight</p>

          <br><b>04/07-22</b><br>
          <div class="line-2"> </div>
          <p>We ran a <strong>SDS PAGE</strong> again with the samples 1-9, then we transferred the proteins on nitrocellulose membrane with the semidry blotting technique</p>
          <p>--&gt; We ran the <strong>western blot</strong> at 10 V with 1 Amper, for 1 h at 4&deg;C</p>
          <p>We <strong>blocked the membrane</strong> with 10% milk in PBS-Tween20 solution for 1 h, then washed it 4 times for 5 min with PBS-Tween20 buffer (PBS 1x + Tween20 0,1%)</p>
          <p><strong>1st Ab incubation:</strong></p>
          <p>We prepared a solution of BSA 3%, 0,1% sodium azide, PBS-Tween20 + antiHis6xTag-Ab 1:1000</p>
          <p>Incubate overnight at 4&deg;C</p>

          <br><b>05/07-22</b><br>
          <div class="line-2"> </div>
          <p><strong>Wash the membrane</strong> 4 times for 5 min with PBS-Tween20</p>
          <p>&nbsp;</p>
          <p><strong>2nd Ab incubation:</strong></p>
          <p>We prepared a solution of 5% milk in PBS-Tween20 + 1:10000 HRP(anti-mouse) and incubated the membrane for 2 h at room temperature</p>
          <p>&nbsp;</p>
          <p>We washed again the membrane 4 times for 5 min with PBS-Tween20 and then detected the proteins with the imager</p>
          <p>--&gt; for the detection we mixed 500 &micro;l luminol enhancer with 500 &micro;l peroxide solution and incubated for 1 min before taking the picture with the imager</p>

          <br><b>06/07-22</b><br>
          <div class="line-2"> </div>
          <p>We ran <strong>SDS gels</strong> of the following samples:</p><ul style="padding-left: 30px;"><li>11, 13, 15, 16, 17,18 (pHT1T2T3 + mVenus)</li><li>19, 21, 22, 25, 26, 27, 28 (pHT1snpspyTag + mVenus)</li><li>29, 30, 31, 32, 33, 34, 35, 36 (pHT1snpspyTagT2T3 + mVenus)</li></ul><p>--&gt; We had to repeat the last row since the marker went into all the wells; for this we made a 8% separating gel and a 3% stacking gel</p><p>&nbsp;</p><p>We ran a <strong>Western blot</strong> for the first two rows following the same procedure as the days before, until the 1st ab incubation</p>

          <br><b>07/07-22</b><br>
          <div class="line-2"> </div>
          <p><span lang="DA">We ran the Western blot for the pHT1snpspyTagT2T3 + mVenus samples until the 1st ab incubation and finished the other ones for pHT1T2T3 + mVenus and for pHT1snpspyTag + mVenus)</span></p>

          <br><b>08/07-22</b><br>
          <div class="line-2"> </div>
          <p>We finished the Western blot procedure for pHT1snpspyTagT2T3 + mVenus samples</p>
          <p>The detection did not work, so we incubated the membranes again with the 1st ab</p>

          <br><b>09/07-22</b><br>
          <div class="line-2"> </div>
          2nd ab incubation of all 3 membranes and detection with the imager<br>

          <br><b>12/07-22</b><br>
          <div class="line-2"> </div>
          <p>Run <strong>SDS gels</strong> at 120 V, 0,4 Ampere</p>
          <p>The samples have been pipetted in the wells as following:</p>
          <ul style="padding-left: 30px;">
          <li>Gel 1: marker, 1, 2, 3, 4, 5, 6 7, 8, 9</li>
          <li>Gel 2: 36, marker, 21, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35</li>
          </ul>
          <p><strong>Western Blot</strong> process until 1st Ab incubation (overnight)</p>

          <br><b>13/07-22</b><br>
          <div class="line-2"> </div>
          <p>&nbsp;</p><p>Finish <strong>Western Blot</strong> + detection on imager</p><p>We made <strong>4 transformations</strong> with BL21, ME5000 and MG1655 competent cells&nbsp; with following plasmids, following the protocol #003:</p><ul style="padding-left: 30px;"><li>pHT1 + mVenus2-spyCat</li><li>pHT1T2T3 + mVenus2-spyCat</li><li>pHt1snpspyTag + mVenus2-spyCat</li><li>pHT1snpspyTagT2T3 + mVenus2-spyCat</li><li>pBAD33 (empty backbone)</li></ul><p>We let the first 4 transformed cultures grow on CmR + AmpR plates, pBAD33 on CmR plates, at 37&deg;C</p>

          <br><b>13/07-22</b><br>
          <div class="line-2"> </div>
          Only the transformations with pBAD33 worked, so we repeated the others

          <br><b>16/07-22</b><br>
          <div class="line-2"> </div>
          <p>Overnight cultures of:</p>
            <ul style="padding-left: 30px;">
            <li>pHT1</li>
            <li>pHT1T2T3</li>
            <li>pHt1snpspyTag</li>
            <li>pHT1snpspyTagT2T3</li>
            <li>mVenus2-spyCat</li>
            <li>mTurquoise-spyCat</li>
            </ul>

            <br><b>17/07-22</b><br>
            <div class="line-2"> </div>
            Miniprep of all overnight cultures

            <br><b>18/07-22</b><br>
            <div class="line-2"> </div>
            <p>Overnight cultures in BL21 of following transformations;</p>
              <ul style="padding-left: 30px;">
              <li>pHT1 + mVenus2-spyCat</li>
              <li>pHT1T2T3 + mVenus2-spyCat</li>
              <li>pHt1snpspyTag + mVenus2-spyCat</li>
              <li>pHT1snpspyTagT2T3 + mVenus2-spyCat</li>
              <li>pBAD33 (empty backbone)</li>
              </ul>

              <br><b>19/07-22</b><br>
              <div class="line-2"> </div>
              <p>Induction of all overnight cultures as following:</p><p>&nbsp;</p><table><tbody><tr><td width="63"><p>0</p></td><td width="67"><p>0</p></td><td width="71"><p>100</p></td><td width="71"><p>100</p></td><td width="90"><p>&micro;M IPTG</p></td></tr><tr><td width="63"><p>0</p></td><td width="67"><p>50</p></td><td width="71"><p>0</p></td><td width="71"><p>50</p></td><td width="90"><p>ng/ml DOXY</p></td></tr><tr><td width="63"><p>0</p></td><td width="67"><p>1</p></td><td width="71"><p>2</p></td><td width="71"><p>3</p></td><td width="90"><p>Condition</p></td></tr></tbody></table><p><strong>&nbsp;</strong></p><p>Samples:</p><ul style="padding-left: 30px;"><li>pHT1 + mVenus2-spyCat: 1-4</li><li>pHT1T2T3 + mVenus2-spyCat: 5-8</li><li>pHt1snpspyTag + mVenus2-spyCat: 9-12</li><li>pHT1snpspyTagT2T3 + mVenus2-spyCat: 13-16</li><li>pBAD33 (empty backbone): 17-20</li></ul><p>Same procedure as the 1st of July but we induced 5 ml cultures with 1:1000 inducers</p><p>&nbsp;</p><p>Growth curve assay in a 96-well plate with 180 &micro;l of each sample, as in the table below (indicated are the sample numbers and the conditions of the samples). We pipetted each sample 3 times in order to get the mean of the values and so get more precise results. We let the plate shake at medium intesity continuosly, at 37&deg;C, measuring the values every 10 min.</p><p>&nbsp;</p><table><tbody><tr><td width="30"><p>1-0</p></td><td width="37"><p>2-1</p></td><td width="37"><p>3-2</p></td><td width="37"><p>4-3</p></td><td width="37"><p>5-0</p></td><td width="37"><p>6-1</p></td><td width="37"><p>7-2</p></td><td width="37"><p>8-3</p></td><td width="37"><p>17-0</p></td><td width="37"><p>19-2</p></td></tr><tr><td width="30"><p>1-0</p></td><td width="37"><p>2-1</p></td><td width="37"><p>3-2</p></td><td width="37"><p>4-3</p></td><td width="37"><p>5-0</p></td><td width="37"><p>6-1</p></td><td width="37"><p>7-2</p></td><td width="37"><p>8-3</p></td><td width="37"><p>17-0</p></td><td width="37"><p>19-2</p></td></tr><tr><td width="30"><p>1-0</p></td><td width="37"><p>2-1</p></td><td width="37"><p>3-2</p></td><td width="37"><p>4-3</p></td><td width="37"><p>5-0</p></td><td width="37"><p>6-1</p></td><td width="37"><p>7-2</p></td><td width="37"><p>8-3</p></td><td width="37"><p>17-0</p></td><td width="37"><p>19-2</p></td></tr><tr><td width="30"><p>9-0</p></td><td width="37"><p>10-1</p></td><td width="37"><p>11-2</p></td><td width="37"><p>12-3</p></td><td width="37"><p>13-0</p></td><td width="37"><p>14-1</p></td><td width="37"><p>15-2</p></td><td width="37"><p>16-3</p></td><td width="37"><p>18-1</p></td><td width="37"><p>20-3</p></td></tr><tr><td width="30"><p>9-0</p></td><td width="37"><p>10-1</p></td><td width="37"><p>11-2</p></td><td width="37"><p>12-3</p></td><td width="37"><p>13-0</p></td><td width="37"><p>14-1</p></td><td width="37"><p>15-2</p></td><td width="37"><p>16-3</p></td><td width="37"><p>18-1</p></td><td width="37"><p>20-3</p></td></tr><tr><td width="30"><p>9-0</p></td><td width="37"><p>10-1</p></td><td width="37"><p>11-2</p></td><td width="37"><p>12-3</p></td><td width="37"><p>13-0</p></td><td width="37"><p>14-1</p></td><td width="37"><p>15-2</p></td><td width="37"><p>16-3</p></td><td width="37"><p>18-1</p></td><td width="37"><p>20-3</p></td></tr></tbody></table><p><strong>&nbsp;</strong></p>

              <br><b>20/07-22</b><br>
              <div class="line-2"> </div>
              <p><strong>Fluorescence microscopy</strong></p><p>Transformation of:</p><p>ME5000 with</p><ul style="padding-left: 30px;"><li>pHT1 + mVenus2-spyCat</li><li>pHT1T2T3 + mVenus2-spyCat</li><li>pHt1snpspyTag + mVenus2-spyCat</li><li>pHT1snpspyTagT2T3 + mVenus2-spyCat</li><li>pBAD33 (empty backbone)</li><li>mVenus2-spyCat</li></ul><br><p>NEBDH5a with</p><ul style="padding-left: 30px;"><li>pHT1</li><li>pHT1T2T3</li><li>pHt1snpspyTag</li><li>pHT1snpspyTagT2T3</li><li>mVenus2-spyCat</li><li>mTurquoise-spyCat</li></ul><br><p>BL21 with</p><ul style="padding-left: 30px;"><li>pHT1snpspyTagT2T3 + mVenus2-spyCat + mTurquoise-spyCat</li></ul>

              <br><b>21/07-22</b><br>
              <div class="line-2"> </div>
              <p><strong>Sonication</strong></p><p><strong>SDS</strong> of all induced samples 1-20 (separating gel: 8%, stacking gel: 3%)</p><p style="padding-left: 30px;">Gel A: marker, 1, 2, 3, 4, 5, 6, 7, 8</p><p style="padding-left: 30px;">Gel B: marker, 9, 10, 11, 12, 13, 14, 15, 16</p><br><p><strong>Western Blot</strong> --&gt; transfer did not work properly</p><ul style="padding-left: 30px;"><li>Overnight cultures of all the NEBDH5a transformations and pHT1snpspyTagT2T3 + mVenus2-spyCat + mTurquoise-spyCat</li></ul>

              <br><b>22/07-22</b><br>
              <div class="line-2"> </div>
              <p>We repeated the<strong> SDS</strong> and made the<strong> Western blot</strong> again until the 1st Ab incubation</p><p>(Western blot at 9-10V, 0,4 ampere, 4W, 1h)</p><p><strong>Roti-Blue staining</strong> of the gels overnight</p><p><strong>Miniprep</strong> of all the NEBDH5a overnight cultures</p><p><strong>Transformations</strong> of ME5000 with new DNA (minipreps):</p> <br> <ul style="padding-left: 30px;"><li>pHT1 + mVenus2-spyCat</li><li>pHT1T2T3 + mVenus2-spyCat</li><li>pHt1snpspyTag + mVenus2-spyCat</li><li>pHT1snpspyTagT2T3 + mVenus2-spyCat</li><li>1939(AmpR) + pBAD33(CmR) (empty backbones)</li><li>1939</li></ul>

              <br><b>23/07-22</b><br>
              <div class="line-2"> </div>
              <p>Finish <strong>Western blot</strong> + detection --&gt; not enough bands and bands at wrong places</p>

              <p><strong>Transformations</strong> of ME5000 with new DNA (minipreps):</p><br>

                <ul style="padding-left: 30px;">
                <li>pHT1 + mVenus2-spyCat</li>
                <li>pHT1T2T3 + mVenus2-spyCat</li>
                <li>pHt1snpspyTag + mVenus2-spyCat</li>
                <li>pHT1snpspyTagT2T3 + mVenus2-spyCat</li>
                <li>1939(AmpR) + pBAD33(CmR) (empty backbones)</li>
                <li>1939</li>
                </ul>


              <br><b>24/07-22</b><br>
              <div class="line-2"> </div>
              Overnight cultures of all transformations of BL21 and MG1655 (but 1939 alone)


  <!-- _______________________________ ncAA ______________________________ -->
  <br><br><h2 id='link_labbook_ncaa'>Noncanonical Amino Acids</h2>
    <div class="line-1"> </div><br><br>
      <br><b>16/05-22</b><br>
      <div class="line-2"> </div>
      The over night cultures grew well, but we discovered that Top10 2019 have a Sm resistance<br>
      <strong>Miniprep:</strong> according to protocoll #005 but at the end of the lysis we centrifuged for 10 min<br>
      We made 4 samples: 2 x pIG22_001 and 2 x pIG22_002<br><br>
      <strong>Test plate Sm+Cm</strong>: We suspect that something is wrong with our plates, so we put 200 µl of over-night-culture of <br>
      Top10 2019 on one of our LB-agar plates with Sm and CM and icubated it over night

      <br><br><b>17/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Test plate Sm+Cm</strong>: Nothing grew on the test plate<br>
    <br>  <strong>Transformation</strong><br>
      We transformed a strain of BL21 E.coli Nicole gave to us with the 2 plasmids that we <br>
      extracted yesterday. Trafo was done according to the protocol #003.<br>
      To a 50 µl aliquot of BL21 cell we added:<br>
      - 2.7 µl pIG22_001 (plasmid conc.: 36.9 mg/ml)<br>
      - 1.6 µl pIG22_002 (plasmid conc.: 93.9 mg/ml)<br>
      The transformed cell were plated on Cm + Sm LB-agar medium and incubated over night:<br>
      - 1 plate 50 µl cells<br>
      - 1 plate 200 µl cells<br>

      <br><b>18/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Update Trafo:</strong> Sm + CM plates with pIG22_001 und _002<br>
      10 am: Nothing grew on the test plate<br>
      3 pm: We have one single colonie<br>
      -> we incubate the plates one more night<br><br>
      <strong>Trafo Repeat</strong> We transformed a strain of BL21 E.coli Nicole gave to us with the 2 plasmids that we <br>
      extracted on monday. Trafo was done according to the protocol #003.<br>
      To a 50 µl aliquot of BL21 cell we added:<br>
      - 3 µl pIG22_001<br>
      - 5 µl pIG22_002<br>
      We used different volumes of plasmid, because we remeasured the plasmid concentration with nanodrop and it gave us very different results for _002<br>
      <br><strong>Plating</strong>: We incubated the following plates over night:<br>
      - LB+Cm+Sm: 600 µl Cells from trafo<br>
      - LB+Cm: 200 µl Cells from trafo<br>
      - LB+Sm: 200 µl Cells from trafo (Sm plate was made by putting Sm on LB plate w/o anything from Nicole)<br>
      - LB+Cm+Sm: ~400 µl unfreezed cells from Tuesday (17.05.22)<br><br>
      <strong>Testing our Cm plates</strong>: We took one plate from each batch and put cells contining pIG22_002 (CmR) on them<br>
      Note: Some weird white stuff grew with the cells (maybe dead cell?)<br><br>
      <strong>Over-night culture</strong>: For plasmid extraction tomorrow we started a over-night culture of cells containing <br>
      pIG22_001(SmR) and pIG22_002(CmR) with their respective antibiotic

      <br><br><b>19/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Update</strong> Trafo 18.05.22 what grew on the plates<br>
      - LB+Cm+Sm with cells from trafo: some colonies! Trafo worked!<br>
      - LB+Cm with cells from trafo: lots of colonies<br>
      - LB+Sm with cells from trafo: lots of colonies<br>
      - LB+Cm+Sm with unfreezed cells: nothing grew, freezig cells not in a proper glycerol stock kills them<br>
      -<br>
      <strong>Update Testing our Cm plates</strong>: Nothing grew on them -> is something still wrong with our plates? Or were the cells not <br>
      good (weird white stuff)<br><br>
      <strong>Update Single colonie from trafo 17.05.22</strong>: Colony kept in fridge in the lab. Maybe we want to sequence it later<br>
      <br><strong>Over-night culture</strong> of succeccful trafo (BL21: pIG22_001+_002)<br>
      6 ml LB + 6 µl Sm + 6 µl Cm: pick one colony and incubate over night<br><br>
      <strong>Plasmid extraction</strong><br>
      We got a new miniprep kit so we want to test it by purifying pIG22_001 and pIG22_002 <br>
      from over-night cultures. Plasmid extration done according to instruction in kit: GenElute & trade; Plasmid Miniprep-Kit <br>
      from Sigma. Plasmid concentration is still rather low (~70 mg/ml and 20 mg/ml) even though we used the new kit. For exact data o plasmid extration see 18-05-22_miniprep_ncAA.xls<br>
      Maybe we should use more cells for the miniprep next time?<br>

      <br><b>20/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Plating trafo cells</strong><br>
      We made 2 plates with cells from the successful trafo on 19.05.22<br>
      1. 100 µl of the overnight culture on LB+Cm+Sm<br>
      2. Pick one colony from plate (19.05.22) and stroke it out with a pipette on LB+Cm+Sm<br><br>
      <strong>Glycerol Stock of trafo cells:</strong> We made 2 cryotubes according to protocol #007 with 3 ml culture for each

      <br><br><b>21/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Inoculation for plasmid extraction</strong>: Inoculation of 6 ml of LB + Cm (25 µg/ml) for plasmid #1881 (Add number of our plasmid list)<br>
      Inocculated at 14:30,  21.05.2022<br>

      <br><b>22/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Plasmid extraction</strong>: Plasmid extraction of pIG22_003 (#1881) with the <br>
      GenElute & trade; Plasmid Miniprep-Kit from Sigma<br>
      Extraction was started at 16:00<br>

      <br><b>25/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Cell cultures from Nicole </strong><br>
      We got 2 over-night cultures in LB from Nicole with cells containing:<br>
      2332 (pIG22_005: sfGFP)<br>
      2340 (pIG22_006: sfGFP w/ 2xMS2)<br>
      After use the rest of the cultures was stored in the lab-fridge<br><br>
      <strong>Miniprep</strong>: We did a plasmid extraction of 2332 and 2340 according to protocol #008 and stored the plasmids in the lab-freezer<br>
      <br><strong>Plates</strong><br>
      We were not sure how much liquid over night culture we could put on a plate so we tested some versions:<br>
      #61623;LB+Cm with 2332 split in half (5 µl one half, 50 µl on the other)<br>
      #61623;LB+Cm with 2340 split in half (5 µl one half, 50 µl on the other)<br>
      #61623;LB+Cm with 2332 1 : 3 dilution with Lb -> 5 µl of that put on plate<br>
      #61623;LB+Cm with 2340 1 : 3 dilution with Lb -> 5 µl of that put on plate<br><br>
      <strong>Making Tryptone Broth (TB medium) </strong><br>
      Induction with arabinose does not work well in LB medium so we decided to grow our cultures in TB<br>
      We made 0.5 l TB according to protocoll #009<br><br>
      <strong>Over-night cultures</strong>: We grow the cultures we want to use for the SDS page in TB medium. In total 9 culture tubes are incubated over night at 37 °C. Each tube consists of 6 ml TB with 6 µl of the respective antibiotic<br>
      - 4x TB+Cm+Sm + BL21 w/ pIG22_001 + pIG22_002 (we picked cultures of the old plates)<br>
      - 2x TB+Cm + 2332 (we took 100 µL of the cultures from nicole)<br>
      - 2x TB+Cm + 2340 (we took 100 µL of the cultures from nicole)<br>
      - 1 Blank just TB+Cm<br>

      <br><b>26/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Update Plates</strong><br>
      50 µl of over-night culture are way too much for a plate. We grew bacteria lawn on those 5 µl was still not good but there were some single colonies<br>
      1 : 3 dilution worked ok but even more diluted would have been good<br><br>
      <strong>Update TB over-night culture</strong><br>
      All cultures grew well. I took them out of the incubator at 10:30 <br><br>
      <strong>OD measurements of TB cultures</strong><br>
      We measured the OD with the nanodrop using cuvettes:<br>
      - Blank was TB medium with CM<br>
      The cultures were measured in 1:9 dilution: 900 µl TB medium and 100 µl culture.
      The names in the table correspond to the samples as following:<br>
      -<br>
      2340-1: first culture with the plasmid pIG22_006<br>
      -<br>
      2340-2: second culture with the plasmid pIG22_006<br>
      -<br>
      2332-1: first culture with the plasmid pIG22_005<br>
      -<br>
      2332-2: second culture with the plasmid pIG22_005<br>
      -<br>
      001_002-1: first culture with plasmid pIG22_001 and pIG22_002<br>
      -<br>
      001_002-2: second culture with plasmid pIG22_001 and pIG22_002<br>
      -<br>
      001_002-3: third culture with plasmid pIG22_001 and pIG22_002<br>
      -<br>
      001_002-4: fourth culture with plasmid pIG22_001 and pIG22_002<br><br>
      <strong>Induction of the cultures</strong><br>
      Table shows which cultures was induced with what:<br>
      <br><table style="border-color: black; width: 822px; height: 193px;" border="black">
      <tbody>
      <tr style="height: 3px;">
      <td style="width: 76.3667px; height: 3px;">
      <p>Sample</p>
      </td>
      <td style="width: 71.8px; height: 3px;">
      <p>001_002-1</p>
      </td>
      <td style="width: 94.8833px; height: 3px;">
      <p>001_002-2</p>
      </td>
      <td style="width: 70.7833px; height: 3px;">
      <p>&nbsp;</p>
      <p>001_002-3</p>
      <p>&nbsp;</p>
      </td>
      <td style="width: 71.8px; height: 3px;">
      <p>&nbsp;</p>
      <p>001_002-4</p>
      <p>&nbsp;</p>
      </td>
      <td style="width: 69.3167px; height: 3px;">
      <p>2332-1</p>
      </td>
      <td style="width: 69.3167px; height: 3px;">
      <p>2332-2</p>
      </td>
      <td style="width: 69.3px; height: 3px;">
      <p>2340-1</p>
      </td>
      <td style="width: 69.3167px; height: 3px;">
      <p>2340-2</p>
      </td>
      <td style="width: 95.1167px; height: 3px;">
      <p>Incubation time</p>
      </td>
      </tr>
      <tr style="height: 36px;">
      <td style="width: 76.3667px; height: 36px;">
      <p>IPTG</p>
      </td>
      <td style="width: 71.8px; height: 36px;">
      <p>+</p>
      </td>
      <td style="width: 94.8833px; height: 36px;">
      <p>+</p>
      </td>
      <td style="width: 70.7833px; height: 36px;">
      <p>-</p>
      </td>
      <td style="width: 71.8px; height: 36px;">
      <p>-</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>&nbsp;</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>&nbsp;</p>
      </td>
      <td style="width: 69.3px; height: 36px;">
      <p>&nbsp;</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>&nbsp;</p>
      </td>
      <td style="width: 95.1167px; height: 36px;">
      <p>1 h</p>
      </td>
      </tr>
      <tr style="height: 36px;">
      <td style="width: 76.3667px; height: 36px;">
      <p>Ara</p>
      </td>
      <td style="width: 71.8px; height: 36px;">
      <p>+</p>
      </td>
      <td style="width: 94.8833px; height: 36px;">
      <p>+</p>
      </td>
      <td style="width: 70.7833px; height: 36px;">
      <p>+</p>
      </td>
      <td style="width: 71.8px; height: 36px;">
      <p>-</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>+</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>-</p>
      </td>
      <td style="width: 69.3px; height: 36px;">
      <p>+</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>-</p>
      </td>
      <td style="width: 95.1167px; height: 36px;">
      <p>3,5 h</p>
      </td>
      </tr>
      <tr style="height: 36px;">
      <td style="width: 76.3667px; height: 36px;">
      <p>ncAA</p>
      </td>
      <td style="width: 71.8px; height: 36px;">
      <p>+</p>
      </td>
      <td style="width: 94.8833px; height: 36px;">
      <p>-</p>
      </td>
      <td style="width: 70.7833px; height: 36px;">
      <p>-</p>
      </td>
      <td style="width: 71.8px; height: 36px;">
      <p>-</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>&nbsp;</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>&nbsp;</p>
      </td>
      <td style="width: 69.3px; height: 36px;">
      <p>&nbsp;</p>
      </td>
      <td style="width: 69.3167px; height: 36px;">
      <p>&nbsp;</p>
      </td>
      <td style="width: 95.1167px; height: 36px;">
      <p>3,5 h</p>
      </td>
      </tr>
      </tbody>
      </table>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <strong>Induction with IPTG</strong>: 13:00 we used 3 µl IPTG (0,2 mM) to induce the samples and incubated them on 37 °C<br>
      <strong>Induction with Arabinose</strong>: 14:20 we used 0.006 g Ara (0.1% w/v) and incubated them on 37 °C<br>
      <strong>Addition of the ncAA 3-amino-L-tyrosine</strong>: 14:30 we used ~2 mM 3amino tyrosine and incubated them on 37 °C<br>
      <br><strong>Harvest the cultures and OD measurement</strong>: 18:00 we took the cultures out of the incubator and measured the OD again in the same way as
      before induction. We took 1,5 ml of each sample and spun it down in Eppis at 16 000 rmp. The supernatent was
      discarded and the cell pellts stored at -20 °C<br>

      <br><b>29/05-22</b><br>
      <div class="line-2"> </div>
      We prepared the <strong>buffers</strong> for SDS pages.<br>
      First we made the Tris/HCl solutions:<br>
      - 0,5 l Tris/HCl 0,625 M at pH 6,8 <br>
      - 0,5 l Tris/HCl 1,88 M at pH 8,8<br>
      There were some problems with getting the Tris solution to the desired pH level. We used ~ 30 ml smoking HCl solution<br>
      We made 500 ml running buffer as a 10x stock following protocol #011<br><br>

      <br><b>30/05-22</b><br>
      <div class="line-2"> </div>
      <p>We tried to cast<strong> SDS gels</strong> and prepared all the necessary <strong>buffers</strong> according to protocol #011_SDS_buffer. The first attempts at casting the gel did not work. The gel did not polymerize properly</p>
      <p>&nbsp;</p>
      <p><strong>Preparing the samples for the SDS page</strong></p>
      <p>To prepare the samples that were taken last week (frozen cell pellets) for loading on the SDS page, the cell pellets were resuspended in L&auml;mmli buffer according to the following formular:</p>
      <p>(200 x OD of the sample) &micro;l of L&auml;mmli buffer per cell pellet</p>
      <p>The samples were stored at -20&deg; C</p>
      <p>&nbsp;</p>

      <br><b>31/05-22</b><br>
      <div class="line-2"> </div>
      <p>We achieved to cast a <strong>SDS gel</strong> but it still takes 1-2 hours to fully polymerize. We will keep testing the chemicals until we can upload the final protocol. For now, we use our successful SDS gel to load the samples from last week</p><p>&nbsp;</p><p><strong>SDS page with samples from last week</strong></p><p>20 &micro;l of of sample were loaded in each pocket and 10 &micro;l of protein ladder. The SDS gel run was started at 14:15 at 100V at 16:45 the voltage was increased to 160 V. The run finished at 15:30</p><table><tbody><tr><td width="68"><p>Pocket</p></td><td width="60"><p>1</p></td><td width="41"><p>2</p></td><td width="41"><p>3</p></td><td width="41"><p>4</p></td><td width="55"><p>5</p></td><td width="44"><p>6</p></td><td width="61"><p>7</p></td><td width="60"><p>8</p></td><td width="62"><p>9</p></td><td width="60"><p>10</p></td></tr><tr><td width="68"><p>Sample</p></td><td width="60"><p>Ladder</p></td><td width="41"><p>2332-1</p></td><td width="41"><p>2332-2</p></td><td width="41"><p>2340-1</p></td><td width="55"><p>Broken pocket</p></td><td width="44"><p>2340-2</p></td><td width="61"><p>001_002-1</p></td><td width="60"><p>001_002-2</p></td><td width="62"><p>001_002-3</p></td><td width="60"><p>001_002-4</p></td></tr></tbody></table><p><strong>SDS page staining</strong></p><p>The gel was stained over night with &ldquo;ROTIBlue 5x staining solution&rdquo; from Carl Roth according to the manufactures protocol</p>

      <br><b>01/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Destaining SDS gel</strong></p>
      <p>9:30 SDS gel was taken out of the staining solution and transferred into 100 ml the washing solution also prepared according to the manufacturers protocol. The gel was incubated for 5 min in washing solution and then taken out to be scanned</p>
      <p><strong>&nbsp;</strong></p>
      <p><strong>Update Gel</strong></p>
      <p>The gel had some leaking pockets and it looked over loaded. The experiment should be repeated</p>

      <br><b>02/06-22</b><br>
      <div class="line-2"> </div>
      <p>We kept testing and altering our protocol for casting SDS gels. The final protocol that worked best for us is: #012_SDS_gel</p>

      <br><b>03/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Repeat: SDS page with samples from last week</strong></p><p>With the final protocol for casting SDS gels #012_SDS_gel two more gels were made and loaded with the same samples as on Tuesday but loaded with less sample and ladder:</p><ul><li>Gel 1: 10 &micro;l sample 3&micro;l ladder</li><li>Gel 2: 5 &micro;l sample 3 &micro;l ladder</li></ul><p>The gel run, staining and destaining was done in the same way as on Tuesday.</p>

      <br><b>04/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Update Repeat: SDS page with samples from last week</strong></p>
      <p>Loading 10 &micro;l of sample resulted in the clearest bands. Staining can be done longer than in the protocol suggested. Old gels can be stored in water for multiple days and stay stable.</p>

      <br><b>06/06-22</b><br>
      <div class="line-2"> </div>
      Solving Problems regarding SDS gel (new Acrylamid/Bisacrylamid)

      <br><b>07/06-22</b><br>
      <div class="line-2"> </div>
      Solving Problems regarding SDS gel (new APS)

      <br><b>08/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Transformation </strong>of BL21(DE3) with plasmids pIG22_001 and pIG22_002 + plating according to protocol #003</p>

      <br><b>09/06-22</b><br>
      <div class="line-2"> </div>
      <p>In order to test if the plasmids with the synthetases work we want to test their expression at different concentrations of induction</p><p><strong>Update Transformation</strong></p><p>The trafo worked. At least 3 colonies from each trafo were picked for the over-night cultures</p><p><strong>Preparing the overnight cultures</strong></p><p>9 over- night cultures were prepared in total, each with 50 ml of medium and 50 &micro;l of the respective antibiotic. The overnight cultures contained the following:</p><ul><li>LB + Cm + colony 1 with pIG22_003</li><li>LB + Cm + colony 2 with pIG22_003</li><li>LB + Cm + colony 2 with pIG22_003</li></ul><p>(TB = tryptone broth to not interfere with Ara induction)</p><ul><li>TB + Cm + colony 1 with pIG22_003</li><li>TB + Cm + colony 2 with pIG22_003</li><li>TB + Cm + colony 3 with pIG22_003</li></ul><ul><li>LB + Sm + colony 1 with pIG22_001</li><li>LB + Sm + colony 2 with pIG22_001</li><li>LB + Sm + colony 3 with pIG22_001</li></ul><p><strong>OD measurements</strong></p><p>- All OD measurements can be found in the LabBook folder-</p><p>The cultures were incubated at 37 &deg;C at 180 rmp for 5.5 hours.</p><p>The OD was measured by diluting the samples 1:1 with the respective medium.</p><p>After 1 more hour of incubation the OD was measured again.</p><p>With an OD of roughly 2 culture &ldquo;LB + Sm + colony 1 with pIG22_001&rdquo; was ready for induction. The other cultures were further incubated</p><p><strong>Induction with IPTG</strong></p><p>First a liquid aliquot of IPTG was made: 5 ml milliQ water + 1.19g IPTG&nbsp; -&gt; 1M stock solution IPTG</p><p>The culture was split into 6 culture tubes with 5 ml each and the following induction concentration were tested for IPTG:</p><ul><li>Not induced</li><li>05 mM</li><li>1 mM</li><li>5 mM</li><li>1 mM</li><li>2 mM</li></ul><p>Half an hour later the cultures &ldquo;LB + Sm + colony 2 with pIG22_001&rdquo; and &ldquo;LB + Sm + colony 3 with pIG22_001&rdquo; also reached an OD of ~2 and were to be split and induced in at same concentrations as above.</p><p>All cultures in grown in TB take longer to reach the appropriate OD as they grow slower.</p><p><strong>Induction with Arabinose</strong></p><p>After 10 hours of incubation all other cultures reached an OD ready for induction. The remaining cultures all contained pIG22_003 and required induction with arabinose. The cultures were split into culture tubes with 5 ml each and the following induction concentration were tested for IPTG for all 6 cultures:</p><ul><li>Not induced</li><li>05 %</li><li>1 %</li><li>2 %</li></ul><p>To not have too many samples it was decided to only test 4 concentrations instead of 6 like before.</p><p><strong>Cell harvest</strong></p><p>After induction all samples were incubated for 3.5 hours before harvest.</p><p>To harvest the cells 1.5 ml of each culture were spun down in an eppi. The supernatant was discarded and the cell pellets stored at &ndash;20 &deg;C</p>

      <br><b>11/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Loading SDS pages</strong></p><p>To see if and how well the induction of the synthetases worked we loaded the samples on SDS pages. The samples were prepared my resuspending them in 200 x OD ml l&auml;mmli buffer.</p><p>For the protein ladder 3 &micro;l were used. The pockets were loaded with 10 &micro;l sample according to the following table:</p><p>&nbsp;</p><p>Sample 003-3 LB 0.1 % was lost</p>Staining was done overnight according to the Carl Roth ROTIBlue manufacturer protocol</p>

      <br><b>12/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>&nbsp;Gel washing and scanning</strong></p>
      <p>Like before the gel was washed accord to manufacturer protocol and scanned for further evaluation</p>
      <p><strong>&nbsp;</strong></p>

      <br><b>13/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>SDS gels were poured</strong>&nbsp;</p>
      <p><strong>Cultures</strong> of BL21 without plasmid were used as control to see if synthetase was detected on the other 3 gels.</p>

      <br><b>14/06-22</b><br>
      <div class="line-2"> </div>
      <p>SDS page with BL21 without plasmid</p><p>&nbsp;</p><p>Primer were diluted to a stock concentration of 100 &mu;M</p><p><strong>Site directed mutagenesis of GFP </strong>(protocol #027)</p><p>Annealing temperature was determined with the NEB Tm calculator</p><p>25 &mu;l per tube were used</p><p>Amber_4 Anneal 68 &deg;C and 63 &deg;C</p><p>Amber_8 Anneal 72 &deg;C and 67 &deg;C</p><p>Amber_57 Anneal 72 &deg;C and 67 &deg;C</p><p>Amber_74 Anneal 72 &deg;C and 67 &deg;C</p><p>Amber_151 Anneal 70 &deg;C and 65 &deg;C</p><p>Amber_216 Anneal 72 &deg;C and 67 &deg;C</p><p>Elongation was set to 185 sec</p><p>&nbsp;</p><p>After the PCR run, the DNA was stored in the fridge.</p>

      <br><b>15/06-22</b><br>
      <div class="line-2"> </div>
      <p>DPNI digest of the site directed mutagenesis products</p>
      <p><strong>Transformation of BL21 with site directed mutagenesis products </strong>according to protocol #003 with 50 &mu;l bacteria and 10 &mu;l PCR product.</p>

      <br><b>16/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>LB agar plates with Cm and Kan</strong></p><p>500 ml of LB agar with Cm and Kan were made and poured into plates.</p><p>&nbsp;</p><p>Over inoculation of the different bacteria with the Amber mutations in LB medium.</p><p>Amber_4 68 &deg;C</p><p>Amber_8 72 &deg;C</p><p>Amber_57 67 &deg;C</p><p>Amber_74 72 &deg;C</p><p>Amber_151 65 &deg;C</p><p>Amber_216 67 &deg;C</p><p>&nbsp;</p><p><strong>Restriction digest of pIG2022_001 (with NgoMIV) and pIG2022_003 (with EcoRI-HF)</strong></p><table><tbody><tr><td width="197"><p>Plasmid</p></td><td width="197"><p>pIG2022_001</p></td><td width="197"><p>pIG2022_003</p></td></tr><tr><td width="197"><p>Restriction enzyme</p></td><td width="197"><p>NgoMIV</p></td><td width="197"><p>EcoRI-HF</p></td></tr><tr><td width="197"><p>DNA conc. / ng/&micro;l</p></td><td width="197"><p>51.6</p></td><td width="197"><p>66.9</p></td></tr><tr><td width="197"><p>DNA amount / &micro;g</p></td><td width="197"><p>1.0</p></td><td width="197"><p>1.0</p></td></tr><tr><td width="197"><p>DNA volume / &micro;l</p></td><td width="197"><p>19.38</p></td><td width="197"><p>14.94</p></td></tr><tr><td width="197"><p>Buffer solution 10x / &micro;l</p></td><td width="197"><p>5</p></td><td width="197"><p>5</p></td></tr><tr><td width="197"><p>RE units</p></td><td width="197"><p>10</p></td><td width="197"><p>20</p></td></tr><tr><td width="197"><p>RE amount / &micro;l</p></td><td width="197"><p>1.0</p></td><td width="197"><p>1.0</p></td></tr><tr><td width="197"><p>Water / &micro;l</p></td><td width="197"><p>24.62</p></td><td width="197"><p>29.06</p></td></tr></tbody></table><p>&nbsp;</p><p>Both plasmids were digested at 37&deg;C for 15 minutes. EcoRI-HF was inactivated at 65&deg;C for 20 minutes. pIG2022_001 was loaded onto an 1.2% agarose gel but no band was visible after running the gel at 100 V for 1 hour and 20 minutes.</p><p>&nbsp;</p><p><strong>PCR of pIG2022_003 to amplify and prepare the pAzF backbone for Gibson assembly</strong></p><p>1 &micro;g/&micro;l DNA of pIG2022_003 was used for PCR</p><p>&nbsp;</p><p><strong>Plasmid extraction of Amber stop codon mutated plasmids</strong></p><p>Measuring OD600s of o/n culture before preparing for plasmid extraction and measuring DNA concentration via NanoDrop.</p><p><strong>&nbsp;</strong></p><table><tbody><tr><td width="74"><p>sample</p></td><td width="74"><p>WT</p></td><td width="74"><p>A4</p><p>68&deg;C</p></td><td width="74"><p>A8</p><p>72&deg;C</p></td><td width="74"><p>A57</p><p>67&deg;C</p></td><td width="74"><p>A74</p><p>72&deg;C</p></td><td width="74"><p>A151</p><p>65&deg;C</p></td><td width="74"><p>A206</p><p>67&deg;C</p></td></tr><tr><td width="74"><p>OD600</p></td><td width="74"><p>1.402</p></td><td width="74"><p>0.306</p></td><td width="74"><p>1.254</p></td><td width="74"><p>1.626</p></td><td width="74"><p>0.982</p></td><td width="74"><p>1.58</p></td><td width="74"><p>1.63</p></td></tr><tr><td width="74"><p>Conc. / ng/&micro;l</p></td><td width="74"><p>54.2</p></td><td width="74"><p>-</p></td><td width="74"><p>81.2</p></td><td width="74"><p>97.4</p></td><td width="74"><p>65.2</p></td><td width="74"><p>83.1</p></td><td width="74"><p>80.5</p></td></tr></tbody></table><p><strong>&nbsp;</strong></p><p>Letting A4 68&deg;C grow until Friday morning. Nicole inoculated a culture with 2338 (Kanamycin resistance)</p>

      <br><b>17/06-22</b><br>
      <div class="line-2"> </div>
      <table><tbody><tr><td width="607"><p><strong>Plasmid extraction of A4 68&deg;C and 2338 with Merck kit</strong></p><table><tbody><tr><td width="197"><p>sample</p></td><td width="197"><p>A4 68&deg;C</p></td><td width="197"><p>2338&nbsp; (KanR)</p></td></tr><tr><td width="197"><p>OD600</p></td><td width="197"><p>1.656</p></td><td width="197"><p>2.238</p></td></tr><tr><td width="197"><p>Conc. / ng/&micro;l</p></td><td width="197"><p>94</p></td><td width="197"><p>87.8</p></td></tr></tbody></table><p><strong>&nbsp;</strong></p><p><strong>Digestion of 1881, 1882 and 2338</strong></p><table><tbody><tr><td width="159"><p>Plasmid</p></td><td width="141"><p>1881</p></td><td width="154"><p>1882</p></td><td width="146"><p>2338</p></td></tr><tr><td width="159"><p>Restriction enzyme</p></td><td width="141"><p>EcoRI-HF</p></td><td width="154"><p>NgoMIV</p></td><td width="146"><p>XhoI</p></td></tr><tr><td width="159"><p>DNA conc. / ng/&micro;l</p></td><td width="141"><p>54.5</p></td><td width="154"><p>25</p></td><td width="146"><p>87.8</p></td></tr><tr><td width="159"><p>DNA amount / &micro;g</p></td><td width="141"><p>1.0</p></td><td width="154"><p>1.0</p></td><td width="146"><p>1.0</p></td></tr><tr><td width="159"><p>DNA vol. / &micro;l</p></td><td width="141"><p>18.34</p></td><td width="154"><p>40.0</p></td><td width="146"><p>11.39</p></td></tr><tr><td width="159"><p>Buffer solution (10x) / &micro;l</p></td><td width="141"><p>5 (1x)</p></td><td width="154"><p>5 (1x)</p></td><td width="146"><p>5 (1x)</p></td></tr><tr><td width="159"><p>RE units / units/ml</p></td><td width="141"><p>20000</p></td><td width="154"><p>10000</p></td><td width="146"><p>20000</p></td></tr><tr><td width="159"><p>Re vol. / &micro;l</p></td><td width="141"><p>1.0</p></td><td width="154"><p>1.0</p></td><td width="146"><p>1.0</p></td></tr><tr><td width="159"><p>Water / &micro;l</p></td><td width="141"><p>25,66</p></td><td width="154"><p>4.0</p></td><td width="146"><p>32.61</p></td></tr></tbody></table><p><strong>&nbsp;</strong></p><p>All plasmids were digested at 37&deg;C for 15 minutes and loaded onto an 1.2% agarose gel at 100 V for 1 hour. The PCR for pAzF with the primers for the Gibson assembly was also loaded. There were no bands visible for 1882.</p><p><strong>&nbsp;</strong></p><p><strong>Gel purification</strong></p><p>The DNA of the digested pAzF backbone (pIG2022_003), the pAzF backbone (pIG2022_003) after the PCR and the KanR backbone were purified from the gel with the NEB Monarch kit.</p><p>&nbsp;</p><table><tbody><tr><td width="140"><p>Sample</p></td><td width="99"><p>KanR bb</p></td><td width="120"><p>pAzF bb PCR</p></td><td width="120"><p>pAzf bb dig</p></td><td width="120"><p>CNF bb dig</p></td></tr><tr><td width="140"><p>Empty Eppi / g</p></td><td width="99"><p>0.9791</p></td><td width="120"><p>0.9764</p></td><td width="120"><p>0.9711</p></td><td width="120"><p>No clear bands</p></td></tr><tr><td width="140"><p>Eppi with gel / g</p></td><td width="99"><p>1.0985</p></td><td width="120"><p>1.0962</p></td><td width="120"><p>1.0564</p></td><td width="120"><p>No clear bands</p></td></tr><tr><td width="140"><p>Weight of gel / mg</p></td><td width="99"><p>119.4</p></td><td width="120"><p>119.8</p></td><td width="120"><p>85.3</p></td><td width="120"><p>No clear bands</p></td></tr><tr><td width="140"><p>Amount of dissolving buffer / &micro;l</p></td><td width="99"><p>477.6</p></td><td width="120"><p>479.2</p></td><td width="120"><p>341.2</p></td><td width="120"><p>-</p></td></tr></tbody></table></td></tr></tbody></table>

      <br><b>18/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Concentration of previous day gel extracts</strong></p><p>Measuring the DNA concentration of KanR bb, the pAzF PCR backbone and pAzF digested backbone.</p><table><tbody><tr><td width="148"><p>Sample</p></td><td width="148"><p>KanR bb</p></td><td width="148"><p>pAzF bb PCR</p></td><td width="148"><p>PAzF bb digest</p></td></tr><tr><td width="148"><p>Conc. / ng/&micro;g</p></td><td width="148"><p>2.8</p></td><td width="148"><p>8.0</p></td><td width="148"><p>6.8</p></td></tr></tbody></table><p><strong>&nbsp;</strong></p><p><strong>PCR for pAzF backbone and Kanamycin restistance inserts</strong></p><p>The PCR for each sample was set up with ng/&micro;l for each sample.</p>

      <br><b>19/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Agarose gel of PCR samples:</strong></p><p>1.2% Agarose gel was used with the NEB Quickload 1 kb plus ladder to separate the pcr product from the digested sample. The PCR for the pAzF backbone and the CNF insert did not work.</p><p>&nbsp;</p><p><strong>Gel extraction:</strong></p><p>The DNA from the agarose gel was extracted by the NEB Monarch kit.</p><table><tbody><tr><td width="197"><p>Sample</p></td><td width="197"><p>KanR_pAzF1</p></td><td width="197"><p>KanR_pAzF2</p></td></tr><tr><td width="197"><p>Empty Tube / g</p></td><td width="197"><p>1.113</p></td><td width="197"><p>1.13</p></td></tr><tr><td width="197"><p>Weight / g</p></td><td width="197"><p>0.9751</p></td><td width="197"><p>0.9753</p></td></tr><tr><td width="197"><p>Gel / mg</p></td><td width="197"><p>138</p></td><td width="197"><p>155</p></td></tr><tr><td width="197"><p>Dissolving buffer / &micro;l</p></td><td width="197"><p>552</p></td><td width="197"><p>620</p></td></tr><tr><td width="197"><p>DNA conc.</p></td><td width="197"><p>56.4</p></td><td width="197"><p>73.4</p></td></tr></tbody></table><p><strong>&nbsp;</strong></p><p><strong>Preparing for Gibson assembly</strong></p><p>Preparing protocols and samples for gibson assembly</p>






  <!-- _______________________________ Trehalose ______________________________ -->
  <h2 id="link_labbook_trehalose">Trehalose Pathway</h2>
      <div class="line-1"> </div><br><br>
      <br><b>23/05-22</b><br>
      <div class="line-2"> </div>
      <strong>Heat-Shock-Transformation</strong> as in Protocol #003, but we let the bacteria incubate for 5 minutes longer at 37&deg;C.<br>
       Also we put the antibiotica straight on the plate:<br>
       First a Volume of 300 µl of LB- medium and then 30 µl of Spectinomycin. <br>
      Then we spread the antibiotica with a glass spreader.<br>
      Put the two plates with 50 µl of bacteria and 100 µl of bacteria into the incubator at 37 °C at 18.15
      <br>
      <br><b>24/05-22</b><br>
      <div class="line-2"> </div>
      We put the plates out oft he incubator at 10:00 in the morning, the <br>
      bacteria seems to grow properly.<br>
      <strong>Overnight Culture</strong> with 6 ml of LB medium and 6 µl of Spectinomycin and one culture picked from the plates we did on Monday. Put in the incubator<br>
      at 19.30.
      <br>
      <br><b>25/05-22</b><br>
      <div class="line-2"> </div>
      <p>We put the <strong>overnight culture</strong> out of the incubator at 9.15 in the morning. The bacteria did grow.</p>
      <p>We did a <strong><u>glycerolstock </u></strong>of our transfomed bacteria ( with pCDF-Duet-1) as n protocol #007 using glycerol 50%. </p>
      <p>We did a <strong>miniprep</strong> as in protocol #008. At the end we had a plasmid concentration of 28,6 ng/&micro;l. Mistake we used optional washing buffer.</p>
      <br><b>26/05-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>overnight culture</strong> like in protocol #014 with 6ml Lb and 6 µl Spectinomycin. And incubated it from 20.21 until 9.00 the next day</p>
      <p>We the <strong>TAE-Buffer</strong> as in Protocol #013, but used to much NaOH and the TAE was to alcalic.</p>
      <p>&nbsp;</p>
      <b>27/05-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>Miniprep</strong> as in protocol #008. We used 5 ml. And had a DNA concentration of 39,5 ng/&micro;l.</p>
      <p>We prepared the agarose gels with the TAE Buffer we prepared. We diluted the 50x TAE Buffer to 1x and mixed 1,2% Agarose ( 0,598g) into it. We heated it for 10-15 seconds in the microwave untik the agarose is disolved and pured it into the form.</p>

      <br><b>29/05-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>TAE buffer</strong> as in protocol#013 but for a half liter.</p>
      <p>We did a <strong>agarose gel</strong> again the same as on Friday but with the new TAE-Buffer.</p>
      <p>We did a <strong>Restricition Enzyme Digest</strong>. We had fore samples: NC the control sample. Re1 with one restiricition enzyme.Re2 with the second restriction enzyme. And T with both restriction enzymes.</p>
      <p>We proceeded as in protocol #015.</p>
      <p>We did a <strong>agarose Gelelectrophresis</strong> as in protocol #016 with 10&micro;l samples with 2,1&micro;l loading dye and 0,5&micro;l Midori Green at constant 80Volt. The gel started at 16.30, finfteen minutes after we loaded the gel and endet at 18.23. Then we analysed the gel.</p>

      <br><b>30/05-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Mini-Prep</strong> following protocol #008 with 5mL of BL21 pCDFDuet-1 to purify the empty plasmid. Deviaton from protocol: Use of 50&micro;L water to elute DNA. Plasmid-conc. measured with Nanodrop: 75,4ng/&micro;L.</p>
      <p>&nbsp;</p>
      <p><strong>Overnight </strong>culture of BL21 pCDFDuet-1 following protocol#014, 50mL as well as 10mL. Start of incubation at 13:35.</p>
      <p>&nbsp;</p>
      <p><strong>Restriction digestion</strong> and <strong>agarose-gel-electrophoresis</strong> following protocols #015 and #016 of empty pCDFDuet-1 with BamHI and NheI-HF. Devitaton from protocol: Used 200ng DNA per sample. Pictures of gel were taken. Note: too many bands in long bp-range + RE1-bands look identical to undigested DNA. Interpretion: RE1 (BamHI) is defect.</p>

      <br><b>31/05-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Mini-Prep</strong> following protocol #008 with 2mL of BL21 pCDFDuet-1 to purify the empty plasmid. Plasmid-conc. measured with Nanodrop: 52,5ng/&micro;L.</p>
      <p>&nbsp;</p>
      <p>Setting an <strong>Ampicillin-stock solution</strong> of 100mg/mL following protocol #018.</p>
      <p>&nbsp;</p>

      <br><b>01/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>TraFo</strong> of otsA_blunted in BL21, otsB_blunted in BL21 and otsA_blunted+otsB_blunted in BL21 following HeatShock protocol #003. Deviations from protocol: Added 3&micro;L in total of DNA to BL21 each time (for otsA_blunted+otsB_blunted 1,5 &micro;L +1,5&micro;L); At step 4, first incubation took 20min instead of 10min because heat block took more time to heat on 42&deg;C. Result (on Thursday collected): 4 big colonies on otsA_blunted, no colonies on other plates.</p>
      <p>&nbsp;</p>
      <p><strong>Mini-Prep</strong> following protocol #008 with 2x 2mL of BL21 pCDFDuet-1 to purify the empty plasmid. Plasmid-conc. measured with Nanodrop: c<sub>1</sub> = 33ng/&micro;L, c<sub>2</sub> = 43ng/&micro;L.</p>

      <br><b>04/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>Restriction digestion</strong> and <strong>agarose-gel-electrophoresis</strong> following protocols #015 and #016 of otsA and otsB seperately, each in pJET1.2. Restriction enzymes used for otsA are NotI and NcoI, for otsB are XhoI and NdeI. Deviaton from protocol: Used 200ng DNA per sample. Note: smearing bands for undigested otsA, some samples with RE seem not to be cut. Interpretion: something wrent wrong with otsA cloning, will be repeated.</p>

      <br><b>06/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>PCR </strong>as on Protocol #019. We used a 2xQ5Polymerase Mastermix from NEB. For cloning otsB out of the bacterial genome of e.Coli we used 25 &micro;l of TaqPolymerase Mastermix, <a href="https://www.dict.cc/englisch-deutsch/respectively.html">https://www.dict.cc/englisch-deutsch/respectively.html</a> respectively 1 &micro;l of each Primer (olG22_001 and olG22_002) one colony of the strain BL21 with the pCDF-Duet-1 And 23 &micro;l of endonuclease free water.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>gelelectrophoresis</strong>. Therefore we cast a 50 ml gel as in protocol #016 with 50 ml TAE-Buffer and 2 % Agarose and 1 &micro;l Midori Green. It did show to clear bands.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>gel extrection</strong> with the NEB as in #021 using 400&micro;l of Dissolving Buffer for otsA.&nbsp; At the end we eluted otsA and otsB with 10&micro;l nuclease free water. Under the Nanodrop we had a concentration of 30.4 ng/&micro;l.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>subcloning with the ThermoFisher-CloningKit </strong>with otsB. We proceede as in protocoll #23 without the blunting reaction, because a Q5 was used in the PCR. As for the PCR product for otsB 2 &micro;l was used.</p>
      <p>&nbsp;</p>
      <p>We did <strong>heatshock-traffolds</strong> as in protocol #003 with otsB and TOP10. On ampecilin plate with 100&micro;l of incubated bacteria.</p>

      <br><b>07/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>PCR </strong>as on Protocol #019. We used a 2xQ5Polymerase Mastermix from NEB. For cloning otsB out of the bacterial genome of e.Coli we used 25 &micro;l of Q5Polymerase Mastermix, <a href="https://www.dict.cc/englisch-deutsch/respectively.html">https://www.dict.cc/englisch-deutsch/respectively.html</a> respectively 1 &micro;l of each Primer (olG22_001 and olG22_002) one colony of the strain BL21 with the pCDF-Duet-1 And 23 &micro;l of endonuclease free water.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>gelelectrophoresis</strong>. Therefore we cast a 50 ml gel as in protocol #016 with 50 ml TAE-Buffer and 1.5 % Agarose and 0.5 &micro;l Midori Green. We used 8 &micro;l of the 100bp ladder. It did show to clear bands.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>gel extrection</strong> with the NEB as in #021 using 400&micro;l of Dissolving Buffer for otsA.&nbsp; At the end we eluted otsA and otsB with 10&micro;l nuclease free water. Under the Nanodrop we had a concentration of 91.0 ng/&micro;l.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>subcloning with the ThermoFisher-CloningKit </strong>with otsB. We proceede as in protocoll #23 without the blunting reaction, because a Q5 was used in the PCR. As for the PCR product for otsB 0.8 &micro;l was used.</p>
      <p>&nbsp;</p>
      <p>We did <strong>heatshock-traffolds</strong> as in protocol #003 with otsB and TOP10. On ampecilin plate with 100&micro;l of incubated bacteria.</p>

      <br><b>09/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>overnight cultur</strong>e as in protocol #014 with 10ml LB medium, 10&micro;l ampecelin and a culture with pJet1.2 + otsA in TOP10.</p>

      <br><b>10/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>miniprep</strong> as in protocol #008. Making 2 samples with 4ml. Which were eluted in 50&micro;l ddH<sub>2</sub>O. We got for otsA 1 355,4 ng/&micro;l and for otsA 2 335,6&nbsp; ng/&micro;l.</p>
      <p>We did a <strong>Restriction Enzyme Digest </strong>like in the protocol #022 with otsA. I had four samples: N0 with no restriction enzymes, Re1 with Nco1, Re2 with Not1 and T with both Restriction enzymes. Of each restriciton enzyme 0.5&micro;l were used and then filled up with ddH<sub>2</sub>O until 20&micro;l. And were inubated for 1 hour.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>gelelectrophoresis</strong>. Therefore we cast a 50 ml gel as in protocol #016 with 50 ml TAE-Buffer and 1.2 % Agarose and 0.5 &micro;l Midori Green. We used 8 &micro;l of the 1kb ladder. It did not show the expected bands.</p>
      <p>&nbsp;</p>
      <p>We did a colony <strong>PCR </strong>as on Protocol #019. We used a 2xDreamPolymerase Mastermix. For cloning otsA out of the bacterial genome of e.Coli we used 25 &micro;l of DreamTaqPolymerase Mastermix, <a href="https://www.dict.cc/englisch-deutsch/respectively.html">https://www.dict.cc/englisch-deutsch/respectively.html</a> respectively 0.2 &micro;l of each Primer (olG22_001 and olG22_002),10 &micro;l of endonuclease free water and 16 different colonys of the strain TOP10 with the of the otsApJet1.2 from the&nbsp; 07.06. Each colony was picked and then put on another ampecilin plate, where every colony got a number.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>gelelectrophoresis</strong>. Therefore we casted two 100 ml gel as in protocol #016 with 100ml TAE-Buffer and 1.2 % Agarose and 1 &micro;l Midori Green. We used 8 &micro;l of the 1kb plus ladder. It did show that most of the colonys had pJet with otsA. Those colonys where used later.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>overnight cultur</strong>e as in protocol #014 with 10ml LB medium, 10&micro;l spectinomycin and a culture with pCDF-Duet in Bl21.</p>

      <br><b>11/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>0.2 M stock of Trehalose</strong>. With 2g of Trehalose dihydrat and 26 ml ddH<sub>2</sub>O.</p>
      <p>&nbsp;</p>
      <p>We diluted it to a <strong>2mM stock of trehalose</strong>, with 100&micro;l of the 0.2 M Stock and 9900&micro;l of ddH<sub>2</sub>O.</p>
      <p>&nbsp;</p>
      <p>We diluted it again into a <strong>0.2 mM stock of Trehalose</strong>. We used 100&micro;l of the 2mM Trehalose stock and 900&micro;l of ddH<sub>2</sub>O.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>test of the Trehalose Test Kit</strong> like in protocol #026. We only tested it with ddH<sub>2</sub>O.</p>


      <br><b>14/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did <strong>Sonication</strong> of 9ml overnight culture of e.Coli. We sonicated for 10 minutes with a 60% at 24-25% power</p>
      <p>&nbsp;</p>
      <p>We did a <strong>test of the Trehalose Test Kit</strong> like in protocol #026. We did the test with ddH<sub>2</sub>O and e.Coli Lysate.</p>

      <br><b>16/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>Restriction digest</strong> as in protocol #022. We digested the pJet1.2 with otsB from the 04.06. We made a 20&micro;l sample.</p>
      <p><strong>&nbsp;</strong></p>
      <p>We did a <strong>gelelectrophoresis</strong>. Therefore we cast a 50 ml gel as in protocol #016 with 50 ml TAE-Buffer and 1.2 % Agarose and 1 &micro;l Midori Green. It did show the bands we expected.</p>

      <br><b>17/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>&nbsp;</strong>We did a <strong>Restriction digest</strong> as in protocol #022. We digested otsB and pCDF-Duet-1. We didnt make any control digest. We made 20&micro;l samples.</p>
      <p>&nbsp;</p>
      <p><strong>&nbsp;</strong>We did a <strong>gelelectrophoresis</strong>. Therefore we cast a 50 ml gel as in protocol #016 with 50 ml TAE-Buffer and 1.2 % Agarose and 1 &micro;l Midori Green. It did show bands we expected for pJet1.2 + otsB. But we had a lot of different bands for pCDF-Duet-1.</p>
      <p>&nbsp;</p>
      <p>We did two <strong>overnight</strong> cultures as in protocol #014 with pCDF-Duet-1. With 10 ml LB-Medium and 10&micro;l Spectinomycin.</p>

      <br><b>18/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>miniprep</strong> as in protocol #008. We did 2 samples with 2 overnight cultures. We centrifuged for the first overnight culture 8 ml in two eppis down for the second we centrifuged 7 ml in two eppis down. After resuspetion we combined the two eppis of each overnight culture into one and proceeded as in the protocol but with each volume dobbled.</p><p>We got 174.9 ng/&micro;l for the first overnight culture and 228.2 ng/&micro;l for the second.</p><p><strong>&nbsp;</strong></p><p>&nbsp;</p><p>We did a <strong>Restriction digest</strong> as in protocol #022. We did two samples with otsB and two samples with pCDF-Duet-1. We used 2&micro;l rCutSmart Buffer, 1 &micro;l XhoI restrictionenzyme and 1&micro;l NdeI restrictionenzyme for every sample. For otsB 1 and 2 we used also 4ml of DNA and 12 &micro;l of ddH<sub>2</sub>O. For pCDF-Duet-1 sample 1 we also used 8&micro;l DNA and 8&micro;l ddH<sub>2</sub>O. For pCDF-Duet-1 sample 2 we also used 11&micro;l DNA and 5&micro;l ddH<sub>2</sub>O.</p><p>&nbsp;</p><p>We did a <strong>gelelectrophoresis</strong>. Therefore we cast a 50 ml gel as in protocol #016 with 50 ml TAE-Buffer and 1.2 % Agarose and 1 &micro;l Midori Green. It did show bands we expected for pJet1.2 + otsB. And we had the bands we expected for pCDF-Duet-1.</p><p>&nbsp;</p><p>We did a <strong>gel extrection</strong> with the NEB as in #021 using 400&micro;l of Dissolving Buffer for otsB 1 and 240 &micro;l for otsB 2 and 80 &micro;l for both pCDF-Duet-1 gels, wich we put together.&nbsp; At the end we eluted&nbsp; otsB and pCDF-Duet-1 with 10&micro;l nuclease free water. Under the Nanodrop we had a concentration of 18.6 ng/&micro;l fot otsB1, but the absorbation for this sample was very bad. For otsB2 we had 22.2 ng/&micro;l. For pCDF-Duet-1 we had a concentration 16.7 ng/&micro;l.</p><p>&nbsp;</p><p>We did a <strong>DNA Ligation</strong> using the NEB Protocol #028. We used 7 &micro;l pCDF-Duet and 3 &micro;l otsB. We incubated it for 10 min at roomtemperature.</p><p>&nbsp;</p><p>We did a <strong>heatshock-traffolds</strong> as in protocol #003 with otsB on pCDF-Duet-1. On spectiomycin plate with 100&micro;l of incubated bacteria. And one spectiomycinplate where we put the rest on.</p>

      <br><b>20/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>miniprep</strong> as in protocol #008. From the overnightcultures from BL21 with pCDF-Duet-1 and what we thought were BL21 with pCDF-Duet-1 + otsB. For just pCDF-Duet we got 60,3ng/&micro;l and for pCDF-Duet-1 + otsB 48ng/&micro;l.</p>
      <p><strong>NOTE: </strong>The plate with pCDF-Duet-1 + otsB was contaminated, because the competent cells were contaminated.</p>

      <br><b>21/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did two <strong>overnight</strong> cultures as in protocol #014 with pCDF-Duet-1, pJet1.2 + otsA and pJet1.2 + otsB. With 10 ml LB-Medium and 10&micro;l Spectinomycin for pCDF-Duet-1 and 10m Ampecilin for pJet1.2 + otsA and pJet1.2 + otsB.</p>

      <br><b>22/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>miniprep</strong> as in protocol #008. From the overnightcultures from BL21 with pCDF-Duet-1 and pJet1.2 + otsB. For the pCDF-Duet-1 we did two samples by spinning down 5ml each. For pJet1.2 + otsB we spinned down 6 ml. We got for pCDF-Duet-1 the first sample a concentration of 294.6ng/&micro;l, for the second sample we got 148.8 ng/&micro;l. For pJet1.2 + otsB we got a concentration of 687.1 ng/&micro;l.</p><p>&nbsp;</p><p>We did <strong>glycerol stocks</strong> as in protocol #007 from pJet+otsA, pJet+otsB, pJet+sTAF, pJet+TasF, pJET+XTB, pJET+sXTB, pJet+TAF and pJET+sXTB. We always spinned down 3 ml of the overnight cultures.</p><p>&nbsp;</p><p>We did a <strong>restriciton digest</strong> as in protocol #022. We digested pJet+otsB and the two pCDF-Duet-1 samples fromt todays mini-prep. For pJet1.2+otsB we used 4&micro;l DNA, for the first pCDF-Duet-1 sample we used 9&micro;l DNA and for the second we used 13&micro;l DNA.</p><p>&nbsp;</p><p>We did a <strong>agarose gelelectrophoresis</strong> as in protocol #016. We used a 50ml agarose gel with 1.2% agarose and 0.5&micro;l Midori green.</p><p>&nbsp;</p><p>We did a <strong>gel extrection </strong>as in protocol #021. For the pCDF-Duet-1 we had 384mg and used 1600&micro;l Disolving Buffer. For otsB we had 105 mg and used 400&micro;l Disolving Buffer.&nbsp; First we added Washbuffer by mistake.</p>

      <br><b>25/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>heat-shock traffold </strong>as in protocol #003 with DH5-&alpha; and no inserts on our Spectinomycin and Chloramphenicol plates. And our competent BL21 on Barbara Di Venturas Spectinomycin and Chloramphenicol plates.</p>

      <br><b>26/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>&nbsp;</strong>We did a sonication with three 1ml samples of a BL21 with pCDF-Duet-1 from a overnight cultures. We used 6 x 10% cycle and 24-25% power.</p>
      <p>&nbsp;</p>
      <p>We did a <strong>Trehalose-Kit</strong> test as in protocol as on protocol #026. We incubated without Trehalase for two hours and the for 18 hours. We had samples from 0.1 &micro;g to 10&micro;g.</p>

      <br><b>27/06-22</b><br>
      <div class="line-2"> </div>
      <p>&nbsp;We did a <strong>sonication</strong> with three 1ml samples of a BL21 with pCDF-Duet-1 from a overnight cultures. We used 6 x 10% cycle and 24-25% power.</p><p>&nbsp;</p><p>&nbsp;We did a <strong>Trehalose-Kit</strong> test as in protocol as on protocol #026. We incubated from the beginning without Trehalase. The assay was 18 ours long. And we used the range from 1&micro;g to 12 &micro;g Trehalose.</p>

      <br><b>29/06-22</b><br>
      <div class="line-2"> </div>
      <p><strong>&nbsp;</strong>We mixed <strong>NPS-Buffer</strong> with 6.6g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 90ml ddH<sub>2</sub>O, 13.6g KH<sub>2</sub>PO<sub>4</sub> and 14.2g Na<sub>2</sub>HPO<sub>4.</sub></p>
      <p>&nbsp;</p>
      <p>We did a overnight culture with 0.5ml NPS Buffer, 0.6ml 2M Glucose, 10&micro;l MgSO<sub>4, </sub>10&micro;l Spectinomycin, 9.4ml and one Colony of BL21.</p>

      <br><b>30/06-22</b><br>
      <div class="line-2"> </div>
      <p>We did a <strong>miniprep</strong> as in protocol #008. From the overnight culture with pJet1.2+otsB we did 2 samples with each 4ml. For the pCDF-Duet we used 5 ml. At the end we had concentrations for pJet1.2+otsB sample one: 604.2ng/&micro;l, for pJet1.2+0tsB sample two: 241.5 ng/&micro;l and pCDF-Duet-1 218ng/&micro;l.</p><p>&nbsp;</p><p>We did a <strong>restriction digest</strong> as in in protocol #022. We digested the two pJet1.2+otsB samples. For pJet1.2+otsB sample one we used 3.3&micro;l DNA, for the second sample we used 8&micro;l DNA and for the pCDF-Duet-1 we used 20&micro;l of another miniprep from the indigo group with a concentration of 96.5 ng/&micro;l. The pCDF-Duet-1 sample was only incubated 45 min.</p><p>&nbsp;</p><p>We did a <strong>agarose gelelectrophoresis</strong> as in protocol #016. We used a 50ml agarose gel with 1.2% agarose and 0.5&micro;l Midori green.</p><p>&nbsp;</p><p>We did a <strong>gel extrection </strong>as in protocol #021. For the pCDF-Duet-1 we had 128mg and used 256&micro;l Disolving Buffer. For otsB we had 180 mg and used 360&micro;l Disolving Buffer. At the end we got for pCDF-Duet-1 19.2ng/&micro;l of concentration and 22.8ng/&micro;l for otsB.</p><p>&nbsp;</p><p><strong>&nbsp;</strong>We did a <strong>heat-shock traffold </strong>as in protocol #003 with NEB5-&alpha; and the ligation from this day and the ligation from the 03.06.22.</p>





  <!-- _______________________________ Indigo ______________________________ -->
  <h2 id="link_labbook_indigo">Indigo Pathway</h2>
        <div class="line-1"> </div><br><br>

        <br><b>06/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Overnightculture </strong>of BL21 pCDFDuet-1 for <strong>indigo-quantification-assay</strong> following protocol #023. 3mL sample used. Start of incubation: 20.31.</p>

        <br><b>07/06-22</b><br>
        <div class="line-2"> </div>
        <p>Preparing of BL21 pCDFDuet-1 (from 06.06.22) for <strong>indigo-quantification-assay</strong> following protocol #023. Derivation: Took 107min for bacteria to grow to OD<sub>600</sub> of 0.833.</p>

        <br><b>08/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>PCR</strong> of all gene-fragments (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre, XiaI-TnaB, spyXiaI-TnaB, XiaIspy-TnaB) for indigo-pathway following protocol #019. Use of Q5 High Fidelity 2x MasterMix, as primers T7 forward and T7 terminal, total volume of samples each 25&micro;L. PCR-annealing-temp. 56&deg;C, elongation-time 3min.</p>
        <p>&nbsp;</p>
        <p><strong>Agarose-gel-electrophoresis</strong> with gene-fragments from PCR (08.06.22) following protocol #016. 1.5% agarose-conc. for gel used. Results saved as image. Interpretion: Chose too short DNA-ladder, have to repeat. Bands look like expected, only in second sample from left seems to be a second shorter fragment assembled.</p>
        <p>&nbsp;</p>
        <p><strong>Gel extraction </strong>from agarose-gel-electrophoresis (08.06.22) for all gene-fragments following protocol #021. Heated to 50&deg;C in heating step, used 6&micro;L Elution-buffer to elute DNA.</p>

        <br><b>09/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>PCR</strong> of all gene-fragments (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre, XiaI-TnaB, spyXiaI-TnaB, XiaIspy-TnaB) for indigo-pathway following protocol #019. Use of Q5 High Fidelity 2x MasterMix, as primers T7 forward and T7 terminal, total volume of samples each 25&micro;L. PCR-annealing-temp. 56&deg;C, elongation-time 3min.</p>
        <p><strong>Agarose-gel-electrophoresis</strong> with gene-fragments from PCR (09.06.22) following protocol #016. 1.5% agarose-conc. for gel used. Results saved as image. Interpretion: Results as expected, but second fragment in second sample from left there as on 08.06.22.</p>
        <p><strong>Gel extraction </strong>from agarose-gel-electrophoresis from PCR (09.06.22) for all gene-fragments following protocol #021. Heated to 50&deg;C in heating step, used 10&micro;L Elution-buffer to elute DNA. of DNA in external excel-table.</p>
        <p><strong>RE-digest </strong>and<strong> agarose-gel-electrophoresis</strong> of empty pCDFDuet-1 and gene-fragments for MCS1 (Tna-Fre, snpTnaA-Fre, TnaAsnp-Fre) following protocols #015 and #016. Used as REs NcoI and BamHI. Deviation from protocols: Used full DNA-volume of gene-fragments (5&micro;L) for RE-digest, because little volume in total of them. Only one control for empty plasmid because too little volume left of it. Results saved as image. Interpetion: Results as expected.</p>
        <p><strong>Gel extraction </strong>from agarose-gel-electrophoresis from RE-digest (09.06.22) for all DNA-fragments following protocol #021. Heated to 50&deg;C in heating step, used 10&micro;L Elution-buffer to elute DNA. of DNA in external excel-table.</p>

        <br><b>10/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>DNA-ligation </strong>of all by PCR extracted gene-fragments (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre, XiaI-TnaB, spyXiaI-TnaB, XiaIspy-TnaB) from 09.06.22 in pJET1.2 following protocol #024.</p>
        <p>Added <strong>Ampicilin to 6 Agarplates</strong> for following Trafo</p>
        <p><strong>Trafo of 9 plasmids</strong> (was done according to masters protocols):</p>
        <p>TnaA-Fre, sTnaA-Fre, TnaAs-Fre (in pCDF-Duet1) were transformed to Bl21 and left to grow on three Spec Agar plates</p>
        <p>XiaI-TnaB, sXiaI-TnaB, XiaIs-TnaB, TnaA-Fre, sTnaA-Fre, TnaAs-Fre (in pJet 1.2) were transormed to Top10 and left to grow on six Amp Agar plates.</p>

        <br><b>11/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Overnight-cultures</strong> of pJet/TAF, pJet/sTAF, pjet/TAsF, pJet/XTB, pJet/sXTB, pJet/XsTB (Amp R.) &nbsp;and Pcdf/TAF, pcdf/sTAF, pcdf/TAsF (Spec R.). These cultures were made using 4 ml of LB and 4&micro;l of antibiotics</p>
        <p><strong>Overnight-cultures </strong>of colonies (from colony pcr) 8 and 10 (Amp R.) both were done using 10ml LB and 10&micro;l antibiotics</p>


        <br><b>12/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Mini-Prep </strong>following protocol #008 with 3mL of different samples to purify the DNA. Deviation from protocol: Of overnightcultures for samples 1-8 2.7mL were used because too little volume for 3mL. Of samples 7,8 and 9 not all of the cleared lysate could be harvested as the solid pellet was blocking some liquid. Samples are:</p><br>
          <ol>
            <li>TnaA-Fre in pJET1.2, Top10</li>
            <li>SnpTnaA-Fre in pJET1.2, Top10</li>
            <li>TnaAsnp-Fre in pJET1.2, Top10</li>
            <li>XiaI-TnaB in pJET1.2, Top10</li>
            <li>SpyXiaI-TnaB in pJET1.2, Top10</li>
            <li>XiaIspy-TnaB in pJET1.2, Top10</li>
            <li>TnaA-Fre in pCDFDuet-1, BL21</li>
            <li>SnpTnaA-Fre in pCDFDuet-1, BL21</li>
            <li>TnaAsnp-Fre in pCDFDuet-1, BL21</li>
          </ol>

        <br><b>13/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Mini Prep</strong> for Nikita, one for colony 8 and three for colony 10</p>
        <p>Culture 8, pcdf/sTAF and pcdf/TAsF were sent for <strong>sequencing</strong></p>
        <p><strong>Overnight culture</strong> of pcdf Duet, Spec R. in 10ml LB and 10&micro;l antibiotics and pcdf TAF, Spec R. in 4ml of LB and 4&micro;l of antibiotics</p>
        <p><strong>Restriction digest</strong> of pcdf TAF, sTAF and TAsF</p>

        <br><b>14/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>RE-digest </strong>and<strong> agarose-gel-electrophoresis</strong> of gene-fragments for MCS1 (Tna-Fre, snpTnaA-Fre, TnaAsnp-Fre) in pCDFDuet-1 + gene-fragments for MCS2 (XiaI-TnaB, spyXiaI-TnaB, XiaIspy-TnaB)) following protocols #015 and #016. Used as REs BglII and NdeI. Results saved as image</p>

        <br><b>15/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony-PCR</strong> and <strong>agarose-gel-electrophoresis</strong> of all gene-fragments in pJET1.2 (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre, XiaI-TnaB, spyXiaI-TnaB, XiaIspy-TnaB) for indigo-pathway following protocols #019 and #016. Use of 5&micro;L Q5 High Fidelity 2x MasterMix, as primers each 1.25&micro;L of 10&micro;M pJET1.2 for and 10&micro;M pJET1.2 rev, total volume of samples each 25&micro;L. PCR-annealing-temp. 70&deg;C, elongation-time 2min, repeats 30x.</p>
        <p><strong>LB-Plates</strong>, one set of plates with spec. Res. And one set of plates with amp. Following proocol #004</p>

        <br><b>16/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony-PCR</strong> of all inserts in pJET1.2 (TnaA-Fre (A), sTnaA-Fre (B), TnaAs-Fre (C), XiaI-TnaB (D), sXiaI-TnaB (E), XiaIs-TnaB (F)). 10&micro;l of Dream-taq master mix, 10&micro;l MQ water, 0.2&micro;L of primers (pJET1.2 for, pJET1.2 rev), and one picked culture, picked culture is first placed onto a fresh LB plate (amp.) and left to grow. PCR-products are put on an agarose gel to check for positive clones.</p>
        <p>&nbsp;</p>
        <p><strong>Overnight culture </strong>of Samples A3, B2, E6 and F3 Following protocol #014</p>

        <br><b>17/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Minipreps</strong> of samples from colony pcr.</p>
        <p><strong>Colony pcr</strong> of C and D (same procedure as the day before)</p>


        <br><b>18/06-22</b><br>
        <div class="line-2"> </div>
        <p>We did a <strong>miniprep</strong> as in protocol #008. We did the miniprep for the samples pCDF-Duet-1 + TasF and pCDF-Duet-1 + TAF. We proceeded the same as above. We got concentration of 215 ng/&micro;l for pCDF-Duet-1 + TasF and 285 ng/ml.</p>
        <p><strong>Colony-PCR</strong> of the previous day was put on an agarose gel</p><p><strong>Overnight culture</strong> of C3 and D4</p><p><strong>RE-digest</strong> of pcdf-duet with TnaA-Fre and sTnaA-Fre, samples were put on an <strong>agarose gel</strong>, negative results</p><p><strong>PCR</strong> of all inserts for amplification, 5&micro;l Q5 Master mix, 0,25&micro;l of primers (T7 Primerfor, T7 Terminal), 2&micro;l of insert DNA, 2,5&micro;l of MQ water annealing temp. Was set to 59&deg;C and elongation time is 3 minutes</p>

        <br><b>19/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>RE-digest</strong> of pcdf-Duet1 and inserts. Products were put on an agarose gel, negative results, very thin, almost transparent bands, pcdf-duet bands did not fit the lenght they should have, digest was discarded</p>
        <p><strong>Overnight culture</strong> of pcdf-duet1 (twice) and one of pcdf-duet one with OtsB inserted</p>

        <br><b>21/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony-PCR </strong>and<strong> agarose-gel-electrophoresis</strong> of all gene-fragments for MCS1 in pCDFDuet-1 (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre) for indigo-pathway following protocols #019 and #016. Use of 10&micro;L Dream-taq-Master mix, 10&micro;L MQ water, 0.2&micro;L primers (DuetDown1, ACYCDuet up1). Anealing-temp 52&deg;C, Elongation time 3min. 1.5% agarose-gel used. Results saved as image.</p>
        <p><strong>PCR</strong> of all gene-fragments (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre, XiaI-TnaB, spyXiaI-TnaB, XiaIspy-TnaB) for indigo-pathway following protocol #019. Use of 12.5&micro;L Q5 High Fidelity 2x MasterMix, as primers each 1.25&micro;L of 10&micro;M T7 forward and 10&micro;M T7 terminal, total volume of samples each 25&micro;L. PCR-annealing-temp. 56&deg;C, elongation-time 3min, repeats 30x. Deviation: Incubation of samples in PCR-cycler at 10&deg;C overnight.</p>

        <br><b>22/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony-PCR </strong>and<strong> agarose-gel-electrophoresis</strong> of all gene-fragments for MCS1 in pCDFDuet-1 (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre) for indigo-pathway following protocols #019 and #016. Use of 10&micro;L Dream-taq-Master mix, 10&micro;L MQ water, 0.2&micro;L primers (DuetDown1, ACYCDuet up1). Anealing-temp 52&deg;C, Elongation time 3min. 1.5% agarose-gel used. Results saved as image.</p>
        <p><strong>PCR</strong> of all gene-fragments (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre, XiaI-TnaB, spyXiaI-TnaB, XiaIspy-TnaB) for indigo-pathway following protocol #019. Use of 12.5&micro;L Q5 High Fidelity 2x MasterMix, as primers each 1.25&micro;L of 10&micro;M T7 forward and 10&micro;M T7 terminal, total volume of samples each 25&micro;L. PCR-annealing-temp. 56&deg;C, elongation-time 3min, repeats 30x. Deviation: Incubation of samples in PCR-cycler at 10&deg;C overnight.</p>

        <br><b>24/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony-PCR </strong>and<strong> agarose-gel-electrophoresis</strong> of all gene-fragments for MCS1 in pCDFDuet-1 (TnaA-Fre, snpTnaA-Fre, TnaAsnp-Fre) for indigo-pathway following protocols #019 and #016. Use of 10&micro;L Dream-taq-Master mix, 10&micro;L MQ water, 0.2&micro;L primers (DuetDown1, ACYCDuet up1). Anealing-temp 52&deg;C, Elongation time 3min. 1.5% agarose-gel used. Results: no bands visible except DNA-ladder.</p>

        <br><b>25/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Indigo calibration curve </strong>measurement following protocol #023. Results saved as excel-sheet.</p>

        <br><b>28/06-22</b><br>
        <div class="line-2"> </div>
        <p><strong>MiniPrep</strong> of XiaIspy-TnaB (F1) and spyXiaI-TnaB (E3) in pJET1.2 from 16.06.2022 following protocol #008, using 3 mL of each overnight-culture. Deviation: Add 50 &micro;L of MQ H<sub>2</sub>O to elute. Measured DNA-Conc.: 211.3 ng/&micro;L for XiaIspy-TnaB (F1); 392.2 ng/&micro;L for spyXiaI-TnaB (E3).</p>
        <p><strong>Glycerol-stock</strong> of XiaIspy-TnaB (F1) and spyXiaI-TnaB (E3) in pJET1.2 in Top10 following protocol #007, using 3 mL for each overnight-culture.</p>

        <br><b>06/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>RE Digest </strong>of pCDF-Duet and all inserts for MCS1 Folloing protocol #015 sTAF negative</p>
        <p>&nbsp;</p>
        <p><strong>Gel electrophoresis </strong>folloing protocol #016 with 1,2% Agarose gel -&gt; negative results</p>
        <p>&nbsp;</p>
        <p><strong>TraFo </strong>of pET-Duet following protocol #003 20 and 50&micro;l were spread onto plates</p>
        <p>&nbsp;</p>
        <p><strong>Over night cultures</strong> of pcdf, TAF, sTAF, TAsF following protocol #014</p>

        <br><b>08/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>RE Disgest </strong>of pcdf and all MCS1 inserts folloing protocol #015 sTAF negative</p><p><strong>Gel extraction </strong>of the samples,using the Jena Bioscience kit, extraordinarily low concentrations</p><p><strong>Over night cultures </strong>of pcdf duet and the inserts for MCS1 following protocol #014</p><p><strong>Tested chloramphenicol amplification </strong>0,72&micro;l of chloramphenicol were aded to an 4ml overnight culture of pcdf. (no growth could be seen the next day)</p>

        <br><b>09/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Mini Prep </strong>of over night cultures from 08.07.22 using the zymo research kit</p>
        <p><strong>Colony PCR </strong>of ligation products from 08.07.22 (wrong primer as used, negative results) using 10&micro;l of buffer, 10&micro;l of water, 0,2&micro;l of primer and bacteria picked from a plate.</p>
        <p><strong>RE Digest </strong>of pCDF Duet, pET Duet and all of the MCS1 inserts following protocol #015</p>
        <p><strong>Gel elctrophoresis </strong>was performed following protocol #016 with 1,2% Agarose gel, sTAF negative results</p>
        <p><strong>Gel Extraction </strong>using the zymo research kit</p>
        <p><strong>Liagtion </strong>of the extracted bands following protocol #028 (pcdf-TAF, pcdf-TAsF, pet-TAF, pet-TAsF</p>
        <p><strong>Trafo </strong>according to protocol #003, samples were spinned down and resuspended with 100&micro;l of lb before they were spread out</p>

        <br><b>10/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony PCR </strong>of the transfomed ligation products from 09.07.22 using 10&micro;l of water, 10&micro;l of buffer, 0,2&micro;l of primer and bacteria picked from the plates. Positive results for pcdf-TAF, pcdf-TAsF and pet-TAF</p>
        <p><strong>Control ligation </strong>of pET and pCDF without inserts following protocol #028, followed by a <strong>TraFo</strong> according to protocol #003, samples were spinned down and resuspended ith 100&micro;l of lb before they ere spread out.</p>
        <p>sTAF was <strong>Ligated </strong>into pJET using the thermo scientific cloneJET kit following protocol #024 (to make a new glycerol stock within the next days)</p>

        <br><b>11/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Glycerol-Stock</strong> of NEB5alpha with pCDFDuet-1 and TAF/TAsF (from overnight-culture 10.07.22) using 3 mL of each sample, following protocol #007.</p><p><strong>Mini-Prep</strong> of NEB5alpha with pCDFDuet-1 and TAF/TAsF (from overnight-culture 10.07.22) using 3 mL of each sample (2x each), following protocol #008. Measured DNA-Conc.: TAF s1:194.8 ng/&micro;L; TAF s2:285.2 ng/&micro;L; TAsF s1:41.7 ng/&micro;L; TAsF s2:84.2 ng/&micro;L. Sent to sequencing.</p><p><strong>Overnight-culture </strong>of NEB5alpha with pCDFDuet-1 and TAF/TAsF following protocol #014. Use of 12 mL LB + 12 &micro;L Spec each. Bacteria came from cPCR-plate (10.07.22)</p>

        <br><b>12/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Mini-Prep</strong> of NEB5alpha with pCDFDuet-1 and TAF/TAsF (from overnight-culture 10.07.22) using 3 mL of each sample (2x each), following protocol #008. Deviation: Column didn&rsquo;t fit, although from same kit, liquid was over filter, needed to lower wash-volume. Measured DNA-Conc.: TAF s1:88.5 ng/&micro;L; TAF s2:26.5 ng/&micro;L; TAsF s1:278.5 ng/&micro;L; TAsF s2:198.9 ng/&micro;L.</p>
        <p><strong>RE-digest</strong> and <strong>agarose-gel-electrophoresis</strong> following protocols #015 and #016. Samples: pCDFDuet-1 with TAF, pCDFDuet-1 with TAsF, XTB, sXTB, XsTB. Deviation: Samples were incubated 40 min on ice before loaded on the gel (gel chamber was not available in meantime). Results saved as image. Results as expected, problem: cutted pJET and sXTB/XsTB same length, need to cut pJET in two parts to distinguish next time.</p>
        <p><strong>Gel-extraction</strong> of samples from agarose-gel-electrophoresis (from 12.07.22) following protocol # . Measured DNA-conc.: pCDFDuet-1 TAF: 16.1 ng/&micro;L; pCDFDuet-1 TAsF: 1 ng/&micro;L; XTB: 56 ng/&micro;L; sXTB (with pJET): 2 ng/&micro;L; XsTB (with pJET): 132.6 ng/&micro;L.</p>
        <p><strong>Ligation </strong>of pCDF-TAF with XiaI-TnaB, pCDF-TAsF with spyXiaI-TnaB/XiaIspy-TnaB according to protocol #028.</p>
        <p><strong>TraFo </strong>of the liagtion products according to protocol #003</p>
        <p><strong>Overnight cultures </strong>of XTB, sXTB, XsTB, pCDF-TAF and pCDF-TAsF</p>
        <p><strong>&nbsp;</strong></p>

        <br><b>13/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony PCR </strong>previous days transformants 10&micro;l of Water, 10&micro;l of dream Taq, 0,2 &micro;l of primers and picked colonies were used, afterwards samples were run through a 1,2% agarose gelaccording to protocol #016.</p>
        <p><strong>Heat-Shock-TraFo</strong> of BL21 with empty pCDFDuet-1 following protocol #003. Use of 1 &micro;L plasmid.</p>
        <p><strong>Overnight-cultures</strong> of Top10 pJET1.2 sTAF following protocol #014.</p>

        <br><b>14/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Mini-prep</strong> of Top10 pJET1.2 with sTAF of overnight-culture (13.07.22) following protocol #008. Measured DNA-conc.: pJET1.2 sTAF s1:86.8 ng/&micro;L; pJET1.2 sTAF s2:177.2 ng/&micro;L.</p>
        <p><strong>Glycerol stock </strong>of sTAF according to protocol #007</p>
        <p><strong>Colony PCR </strong>of a plate containing pCDF TAsF-sXTB. 10&micro;l of Water, 10&micro;l of dream Taq, 0,2 &micro;l of primers and picked colonies were used, afterwards samples were run through a 1,2% agarose gelaccording to protocol #016</p>
        <p><strong>Restriction enzyme digest </strong>of sTAF, pCDF, TAF, TAsF, XTB, sXTB, XsTB was performed according to protocol #015. A third RE (BsaI) was added to the sXTB and XsTB samples as these sequeces have the same length as pJET. BsaI cut pJET into smaler pieces making them distinuishable from the desired inserts</p>
        <p><strong>Gel electrophoresis </strong>of the digested samples according to protocol #016.</p>
        <p><strong>Overnight-cultures</strong> of 2x BL21 pCDFDuet-1, Top10 pJET1.2 sTAF, NEB5alpha pCDFDuet-1 TAF/TAsF, Top10 pJET1.2 XTB/sXTB/XsTB following protocol #014.</p>
        <p><strong>&nbsp;</strong></p>

        <br><b>15/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Preperation </strong>of BL21 pCDFDuet-1 for<strong> calibration curves</strong> with Indigo/Indirubin following protocol #023 to step 7. Indigo 36samples, OD of 200mL:0.807; Indirubin 36samples, OD of 200mL: 0.799.</p><p><strong>Mini-Prep </strong>of previous days overnight cultures. Deviations: Water for elution was heated in a microwave and left on collums for incubation for 10 minutes</p><p><strong>Restriction digest of </strong>sTAF, pCDF-Duet, XTB, sXTB, XsTB, pCDF-TAF, pCDF-TAsF. STAF and pCDF-Duet were performed according to protocol #015</p><p>XTB, sXTB, XsTB, pCDF-TAF and pCDF-TAsF were digested in two steps. Each sample was treated with one Restriction enzyme during the first step, and then treated with the remaining enzyme(s) in the second step. PCR cleanup was performed inbetween steps according to the kits manual. Additionally, alkaline phosphatase was added to pCDF-TAF and pCDF-TAsF after half an hour of 37&deg;C incubation (in both steps). XTB, sXTB and XsTB were run through an 1,2% agarose gel after the second step together with sTAF and pCDF-Duet. Another PCR cleanup of pCDF-TAF and pCDF-TAsF was performed after the second step and was then put aside for a later ligation.</p><p><strong>Gel extraction </strong>of XTB, sXTB and XsTB were performed according to the kits manual (sTAF and pCDF had negative results in the agarose gel)</p><p><strong>Ligation </strong>of pCDF-TAF + XTB, pCDF-TAsF + sXTB and pCDF-TAsF + XsTB was performed according to protocol #028</p><p><strong>TraFo </strong>of ligation products according to protocol #003. Samples were spinned down and eluated with 100&micro;l of lb before being spread.</p><p><strong>&nbsp;</strong></p>

        <br><b>16/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>RE-digest </strong>with<strong> PCR-cleanups/agarose-gel-electrophoresis</strong> following protocols #015, #016 and #. Deviation: Seperate RE-digest with each RE + alkaline phosphatase for backbones after 1/2h RE-digest.</p>
        <p>-<strong>Gel extraction </strong>of pCDF-Duet, sTAF, sXTB, XsTB according to the kits manual</p>
        <p>-<strong>Ligation </strong>of pCDF-Duet + sTAF, pCDF-TAsF + sXTB/XsTB according to protocol #028</p>
        <p>-<strong>Trafo </strong>of Ligation products into DH5alpha according to protocol #003. Samples were spinned down and resuspended with 100&micro;l of LB before being spread out</p>

        <br><b>17/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony PCR</strong> and agarose-gel-electrophoresis of TraFo-plates (from 16.07.22) follwing protocols #019 and #016, use of 6 colonies each. Result: Gel run too long, not useful, have to repeat.</p>

        <br><b>18/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Spec. Plates </strong>were made according to protocol #004</p>
        <p><strong>RE-digest</strong> of sTAF and empty pCDFDuet-1 following protocol #015.</p>
        <p><strong>ColonyPCR</strong> of TraFo-plates (from 15.07.22) following protocol #019, use of 3 colonies each.</p>
        <p><strong>Agarose-gel-electrophoresis</strong> of RE-digest samples and ColonyPCR samples (18.07.22) following protocol #016. Result saved as image, no inserts --&gt; no successful integration of inserts.</p>
        <p><strong>Gel extraction </strong>of sTAF and pCDF-Duet according to the kits manual.</p>
        <p><strong>Liagation </strong>of sTAF + pCDF-Duet according to protocol #028</p>
        <p><strong>Trafo </strong>of Liagtion products according to protocol #003, 10&micro;l of ligation samples were used</p>

        <br><b>19/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>ColonyPCR</strong> of NEB5alpha pCDFDuet-1 with sTAF from TraFo-plate (from 18.07.22) following protocol #019, use of 9 colonies.</p>
        <p><strong>Agarose-gel-electrophoresis</strong> of samples from ColonyPCR (19.07.22) following protocol #016. Samples run too far, not useful, have to repeat.</p>
        <p><strong>PCR </strong>of MCS 2 inserts with a new primer. The primer has an overhang on its 5&rsquo; end that corresponds to the XhoI recognition site. That way, we added a new restriction site to our inserts. According to protocol #019. Deviations: Elongation time: 3 minutes, Annealing temp.: 56&deg;C</p>
        <p><strong>Gel electrophoresis </strong>according to protocol #016</p>
        <p><strong>Gel extraction </strong>of XTB, sXTB and XsTB according to the kits manual.</p>
        <p>L<strong>igation </strong>of the inserts into pJET by using the cloneJET kit by thermo scientific, according to the kits manual</p>
        <p><strong>TraFo </strong>according to protocol #003</p>

        <br><b>20/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>PCR</strong> of XTB, sXTB, XsTB with new primer for XhoI-RE-sequence, following protocol #019 (same procedure as on 19.07.22).</p><p><strong>PCR-Cleanup</strong> of PCR-samples (20.07.22) following protocol #. Use of 25 &micro;L MQ H2O to elute. Measured DNA-conc.: XTB:44.5 ng/&micro;L; sXTB:74.9 ng/&micro;L; XsTB:45.6 ng/&micro;L.</p><p><strong>Colony PCR </strong>of TraFo products of the previous day. 10&micro;l of Dream Taq, 10&micro;l of Water and 0,2&micro;l of primers (pJET1.2 for/rev) and picked colonies were used. Samples were then loaded onto a Gel and showed positive results for sXTB and XsTB, over night cultures of these samples were made.</p><p><strong>RE-digest </strong>and <strong>agarose-gel-electrophoresis</strong> of XTB, sXTB, XsTB (from PCR 20.07.22), pCDFDuet-1 TAF/TAsF, pCDFDuet-1 and sTAF following protocols #015 and #016. Results saved as image. Looking as expected, only sTAF wrong REs were used, uncut this way.</p><p><strong>Gel extraction </strong>of XTB, sXTB and XsTB according to the kits manual</p><p><strong>Ligation </strong>of pCDF-TAF + XTB, pCDF-TAsF + sXTB/XsTB according to protocol #028. Ligation of pET-Duet + OtsA for Nikita, also according to protocol #028</p><p><strong>TraFo </strong>of Ligation products according to protocol #003</p>

        <br><b>21/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Glycerol Stock</strong> of sXTB and XsTB with the added re site according to protocol #007</p>
        <p><strong>Mini Prep </strong>of pCDF-Duet, sXTB and XsTB according to the kits manual</p>
        <p><strong>Colony PCR </strong>of TraFo products of the previous day (pCDF-TAF + XTB, pCDF-TAsF + sXTB/XsTB), 10&micro;l of Dream Taq, 10&micro;l of Water and 0,2&micro;l of primer (DuetUp2 / T7 Term) and picked colonies have been used. Samples were then loaded onto an agarose gel and showed negative results.</p>
        <p><strong>RE-digest </strong>of sTAF, pCDFDuet-1, sXTB, XsTB and pCDFDuet-1 TAsF following protocol #016.</p>
        <p>- garose-gel-electrophoresis of sTAF, XsTB and sXTB (from RE-digest 21.07.22) following protocol #016. Results saved as image. STAF fine, XsTB and sXTB weird, because no pJET-band and band too small for insert with catcher.</p>
        <p><strong>Gel-extraction</strong> of sTAF (from gel 21.07.22) following protocol #.</p>
        <p><strong>Liagtion </strong>of pCDF-Duet and sTAF according to protocol #028</p>
        <p><strong>&nbsp;</strong><strong>TraFo </strong>of ligation products according to protocol #003</p>
        <p><strong>Over night cultures </strong>of pCDF-TAsF, sXTB and XsTB accodring to protocol #014</p>

        <br><b>22/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Mini Prep </strong>of sXTB, XsTB and pCDF-TAsF according to the kits manual</p><p><strong>Colony PCR </strong>of TraFo products of the previous day (pCDF-Duet and sTAF). 10&micro;l of Water, 10&micro;l of Dream Taq, 0,2&micro;l of primers (ACYCDuetUp1 / DuetDown1) and picked colonies were used. Samples were then loaded onto an agarose gel. Sample 6 showed positive results. An over night culture of this sample was made later that day.</p><p><strong>Amplification of MCS 2 inserts, adding the new restriction site to the inserts,</strong> according to protocol #019. Deviations Annealing temp. 62&deg;C, Elongation time 3 min.</p><p><strong>Restriction Enzyme digest</strong> of sXTB, XsTB, pCDF-TAsF (multiple samples, no RE, 1. RE, 2.RE, both REs) according to protocol #015.</p><p><strong>Gel electrophoresis </strong>of Digested samples according to protocol #016. Positive results for pCDF-TAsF and sXTB.</p><p><strong>Gel extraction </strong>of sXTB and pCDF-TAsF according to the kits manual.</p><p><strong>Liagtion </strong>of pCDF-TAsF and XsTB according to protocol #028. Deviations multiple samples were made with differents backbone insert ratios (1/3, 1/5, 1/7). One Part of the samples was directly transformed, the rest was incubated over night at 16&deg;C.</p><p><strong>Liagtion </strong>of MCS 2 inserts with the new RE site into pJET using the cloneJET kit by thermo scientific, according to the kits manual</p><p><strong>TraFo </strong>of all Liagtion products accodring to protocol #003</p>

        <br><b>23/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Colony PCR </strong>of transformants of the previous day. Scaning for MCS2 inserts. 10&micro;l of Water, 10&micro;l of Water, 0,2&micro;l of primers (DuetUp2/T7Terminal) and picked colonies were used. Samples were loaded onto an agarose gel. Negative results for the ligation products, no bands could be seen, probably a mistake was made while pipetting the primers. Experiment will be repeated the next day. Negative results for the pJET Clones with MCS 2 inserts. Screening other clones the next day.</p>
        <p>-Another colony PCR was performed for NIkita, under the same conditions and following the sam protocol. Samples were NEB5alpha pET AB 1/3, DH5alpha pET AB 1/3, Top10 AB 1/3, NEB5alpha pET AB 1/5.</p>
        <p>-<strong>Over night cultures</strong> of TAsF, colony 4 of pJET sXTB and pJET sXTB from a glycerol stock were made, accodring to protocol #014</p>

        <br><b>24/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>&nbsp;</strong><strong>Mini Prep </strong>of pCDF-TAsF and sXTBn according to the kits manual</p>
        <p><strong>Overnight cultures </strong>of pCDF-Duet DH5alpha and pET-Duet AB Top 10 (from a colony pCR of the previous day. (for Nikita)</p>
        <p><strong>RE Digest </strong>of pCDF-TAsF, pCDF-sTAF and sXTBn following protocol #015. Alkaline phosphatase was added after half an hour of incubation on the heat block. -<strong>Gel electrophoresis </strong>of the digested samples according to protocol #016.</p>
        <p><strong>Gel extraction </strong>of pCDF-TAsF, pCDF-sTAF and sXTBn according to the kits manual. DNA was eluated into wash buffer by accident. The column was then transfered to a fresh eppi and DNA was eluated again. Eluation has appeared to be succesful after checking the DNA concentration on a Nanodrop.</p>
        <p><strong>Ligation </strong>of pCDF-TAsF + sXTBn, pCDF-sTAF + sXTBn and a control ligation without insert. According to protocol #028</p>
        <p><strong>Trafo </strong>of the ligation products according to protocol #003</p>
        <p><strong>&nbsp;</strong></p>

        <br><b>25/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>ColonyPCR </strong>and following<strong> agarose-gel-electrophoresis </strong>of NEB5alpha with pCDFDuet-1-TAsF-sXTB, with pCDFDuet-1-sTAF-sXTB, Top 10 with pJET1.2-XTB and Top10 with pJET1.2-XsTB, following protocol #019. Result: No bands except DNA-ladder, propably ColonyPCR went wrong (maybe too little primer-volumes in samples, because pipette-tips were bad), have to repeat.</p>

        <br><b>26/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>cPCR </strong>of pCDF-Duet C/N, pCDF-Duet N/N, XTB and XsTB. 10&micro;l of Dream taq master mix, 10&micro;l of water, 0,2&micro;l of primers (Duet Up2, T7 Terminal) and picked colonies were used.</p>
        <p><strong>Restriction enzyme digest </strong>of pCDF-sTAF, pCDF-TAsF and sXTBn according to protocol #015, alkaline phosphatase has been aded to the pCDF samples after half an hour of incubation. Another digest for Nikita was performed with pCDF-Duet and pET-Duet.</p>
        <p><strong>&nbsp;</strong><strong>DNA-ligation </strong>of pCDFDuet-1-sTAF with sXTB, as well as pCDFDuet-1-TAsF with sXTB (from RE-digest 26.07.22), following protocol #028.</p>
        <p><strong>HeatShock-TraFo</strong> following protocol #003, with pCDFDuet-1-sTAF-sXTB, pCDFDuet-1-TAsF-sXTB (from ligation 26.07.22, and repeated from ligation 24.07.22). Use of 10 &micro;L NEB5alpha with 1 &micro;L of ligation-mixture each.</p>
        <p><strong>Overnight-cultures </strong>of pCDFDuet-1 sTAF (6 mL sample), Bl21 (4 mL) and Bl21 with pCDFDuet-1 (4 mL), following protocol #014.</p>

        <br><b>27/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>MiniPrep </strong>of NEB5alpha pCDFDuet1 sTAF , following protocol #008. Use of 4 mL of overnight-culture (from 26.07.2022). Eluted in 25 yL MQ H2O. Measured DNA-conc.: 70.5 ng/yL.</p>
        <p><strong>Colony-PCR </strong>of pCDFDuet1 sTAF-sXTB / TasF sXTB and Nikitas sample (pCDFDuet1 otsA+otsB), following protocol #019. Use of 6 colonies each. Result: No insert. Probably too much religation, test with different AP-conc and more colonyPCR to avoid bad luck. Result saved as image, looking as expected.</p>
        <p><strong>Restriction enzyme digest </strong>of pCDF-sTAF, pCDF-TAsF, sXTBn according to protocol #015, alkaline phosphatase has been added after half an hour of incubation. Samples were then loaded onto a gel.</p>
        <p><strong>Gel extraction </strong>of the digested samples according to the kits manual</p>
        <p><strong>Ligation </strong>of pCDF-sTAF + sXTBn and pCDF-TAsF+sXTBn according to protocol #028</p>
        <p><strong>Trafo </strong>of the ligation protocols according to protocol #003</p>

        <br><b>28/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>RE-digest and agarose-gel-rlectrophoresis </strong>of pCDFDuet1 with TasF, and sXTB. Result saved as image, bands as expected.</p>
        <p><strong>Gel-extraction </strong>of pCDFDuet1 TasF and sXTB, according to manufacturers protocol. Eluted in 30 yL MQ H2O. Measured DNA-conc.: pCDFDuet1 TasF1: 2 ng/yL; pCDFDuet1 TasF2: 30.5 ng/yL; sXTB: 33.7 ng/yL.</p>
        <p><strong>DNA-ligation </strong>of pCDFDuet1 TasF with sXTB following protocol #028.</p>
        <p><strong>cPCR </strong>of C/N, N/N, XTBn and XsTBn. 10&micro;l of Dream taq master mix, 10&micro;l of water, 0,2&micro;l of Primers (pJET1.2 for, pJET1.2 rev) and picked colonies were used.</p>
        <p><strong>Over night cultures </strong>of sXTBn, XsTBn, pCDF-TAsF, pCDF-sTAF and pCDF-AB (for Nikita) according to protocol #014</p>

        <br><b>29/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Mini preps </strong>of the previous days overnight cultures according to the kits manual</p>
        <p><strong>Glycerol stock </strong>of pCDF-AB (for Nikita) according to protocol #007</p>
        <p><strong>cPCR </strong>of XTB, XsTB and pCDF-C/N. 10&micro;l of dream taq master mix, 10&micro;l of water, 0,2&micro;l of primers (pJET1.2 for, pJET1.2 rev for XTBn and XsTBn / Duet Up2 and T7 terminal for pCDF-C/N)</p>
        <p><strong>Agarose-gel-electrophoration </strong>of ColoyPCR-samples (from 29.07.22) following protocol #016. Results saved as image, no insert in &rdquo;C&rdquo;, maybe in &rdquo;A&rdquo; and &rdquo;B&rdquo;.</p>
        <p><strong>Overnight-culture </strong>of proposed pJET1.2 XTB/XsTB in Top10 from colonyPCR (29.07.22), following protocol #014. Colonies used: A7 and B3.</p>
        <p><strong>Restriction enzyme digest and agarose-gel-electrophoresis </strong>of pCDF-sTAF, pCDF-TAsF, sXTBn, XsTBn and pCDF-AB (for Nikita) according to protocol #015, alkaline phosphatase was added to pCDF- containing samples after half an hour of incubation. Deviations: pCDF-sTAF and pCDF-TAsF were digested in two steps. The samples were treated with only one enzyme during each step. Samples were incubated for one hour per step and alkaline phosphatase was added after half an hour of incubation during both steps</p>
        <p><strong>Gel-extraction </strong>of pCDFDuet1 sTAF, pCDFDuet1 TasF and XsTB (from agarose-gel 29.07.22), following manufactorers protocol.</p>
        <p>Measured DNA-conc.: pCDF sTAF: 15.3 ng/yL; pCDF TasF: 5.1 ng/yL; XsTB: 22.2 ng/yL.</p>
        <p><strong>DNA-ligation </strong>of pCDFDuet1 sTAF with XsTB, pCDFDuet1 TasF with XsTB, and N/O control, following protocol #028.</p>
        <p><strong>HeatShockTraFo </strong>of NEB5alpha with pCDF sTAF-XsTB, pCDF TasF-XsTB and pCDF sTAF-N/O (from 29.07.22). Use of 16.5 yL NEB5alpha with 1.65 yL ligation mix.</p>

        <br><b>30/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>ColonyPCR </strong>of NEB5alpha with pCDFDuet1 TQsF-sXTB (from plate of 28.07.22), following protocol #019.</p>
        <p><strong>Glycerol stock </strong>of XTBn according to protocol #007</p>
        <p><strong>Mini prep </strong>of XTBn according to the kits manual</p>
        <p><strong>RE Digest </strong>XTBn and sXTBn according to protocol #015, samples were then loaded onto a gel to compare their sizes. XTBn turned out to be shorter than it should be, so the glycerol stock that was made this day was discarded.</p>
        <p><strong>Over night cultures </strong>of BL21 (without transformed plasmid) and BL21 pCDF-duet according to protocol #014</p>


        <br><b>31/07-22</b><br>
        <div class="line-2"> </div>
        <p><strong>RE digest </strong>of pCDF-TAsF and sXTBn according to protocol #015. Deviations: different pCDF-TAsF samples were prepared, one without REs, one with only NdeI, one with only XhoI and&nbsp; one with both enzymes at the same time. Samples were loaded onto a 1,2% agarose gel and showed expected results. Additionally, two samples of pCDF-TAsF were digested to check, wheter both enzymes work in pCDF-TAsF at the same time (one digested samples and one control sample without addition of restriction enzymes). Samples were digested in two steps, with one enzyme in the first step and the other one in the second step. Each step consists of a full digest according to protocol #015. PCR cleanup was performed inbeteween steps and alkaline phosphatase was added. Samples were then loaded onto a 5% agarose gel. No band could be seen, indicating that the two enzymes apparently do not work at the same time. However, digest earlier that day showed expected results, the double digested band even appeared to run lower than the single digested bands. Assumption: alkaline phosphatase interferes with the REs activitiy</p>
        <p><strong>cPCR </strong>of pCDF-AB and XTBn according to protocol #019</p>
        <p><strong>Gel extraction </strong>of pCDF-TAsF and sXTB following manufacturers protocol.</p>
        <p><strong>Liagtion </strong>of pCDF-TAsF with sXTB following protocol #028.</p>
        <p><strong>Over night cultures</strong> of sample A6(from colony pcr of pCDF-TAsF), BL21 (without transformed plasmid) and BL21 pCDF-Duet according to protocol #014</p>


        <br><b>01/08-22</b><br>
        <div class="line-2"> </div>
        <p><strong>TraFo </strong>of pCDF-TAsF controll and pCDF-TAsF-SXTB according to protocol #003</p>
        <p><strong>RE Digest </strong>of pCDF-sTAF and pCDF-sTAF control without REs according to #015</p>
        <p><strong>Samples </strong>were then loaded onto a 5% gel to see whether the fragment between the RE sites is cut out. Unclear results.&nbsp;</p>

        <br><b>02/08-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Mini prep</strong> of pCDF-TAsF, pCDF-sTAF, sXTBn and XsTB according to the kits manual</p>
        <p>-samples were then loaded onto a gel and <strong>extracted </strong>from the gel according to the kits manual.</p>
        <p><strong>Liagtion </strong>of pCDF-sTAF + sXTBn and pCDF-sTAF + XsTBn according to protocol #028. PCDF-sTAF + XsTBn was ligated without the addition of Ligase.</p>
        <p><strong>ColonyPCR </strong>of pCDF-TAsF-sXTB following protocol #019.</p>
        <p><strong>Agarose-gel-electrophoresis</strong> of ColonyPCR-samples (01.08.2022), following protocol #016.</p>
        <p><strong>PCR</strong> of XTBn with new primer for XhoI-sequence integration, following protocol #019.</p>
        <p><strong>Ligation </strong>of XTBn with pJET1.2 following protocol #024.</p>

        <br><b>03/08-22</b><br>
        <div class="line-2"> </div>
        <p><strong>ColonyPCR </strong>of Top10 with pCDFDuet1-TAsF-sXTB (from ligation 02.08.2022) and from NEB5alpha with pCDFDuet1 sTAF-sXTB, following protocol #019.</p>
        <p><strong>RE Digest </strong>of pCDF-sTAF + sXTBn (no results here), pCDF-Duet, sXTBn and pCDF-TAsF according to protocol #015.</p>
        <p><strong>Gel-extraction</strong> of RE-digest-samples (03.08.2022), according to manufactorers protocol. Measured DNA-conc: pCDF-TAsF 9.3 ng/&micro;L; pCDF empty 14.3 ng/&micro;L; sXTB 11.5 ng/&micro;L.</p>
        <p><strong>&nbsp;</strong><strong>Ligation </strong>of pCDF-TAsF and pCDF empty with sXTB seperately, following protocol #028. Deviations: liagtion was performed over night in a heatblock set on 16&deg;C</p>
        <p>&nbsp;</p>

        <br><b>04/08-22</b><br>
        <div class="line-2"> </div>
        <p><strong>RE Digest </strong>of sXTBn, pCDF-TAsF, pCDF-sTAF, XsTBn and pCDF-duet according to protocol #015.</p>
        <p><strong>Gel extraction </strong>of digested samples according to the kits manual.</p>
        <p><strong>Liagtion of</strong> pCDF-sTAF + sXTBn, pCDF-TAsF + XsTBn and pCDF-Duet + sXTBn according to protocol #028. Deviations: liagtion was performed over night in a heatblock set on 16&deg;C</p>
        <p><strong>Heat-shock-TraFo</strong> of pCDF-TAsF-sXTB /pCDF-sXTB in NEB5alpha seperately, following protocol #003.</p>

        <br><b>05/08-22</b><br>
        <div class="line-2"> </div>
        <p><strong>ColonyPCR </strong>of NEB5alpha with pCDF-sXTB and NEB5alpha with pCDF-TAsF-sXTB, following protocol #019. Result saved as image. Colonies with insert: A12, B5, B10, B11 and B12.</p>
        <p><strong>RE digest </strong>of pCDF-duet, pCDF-sTAF, pCDF-TAsF, sXTBn and XsTBn according to protocol #015</p>
        <p><strong>Agarose-gel-electrophoresis</strong> of RE-digest-samples (05.08.22), following protocol #016.</p>
        <p><strong>Gel extraction </strong>of the digested samples according to the kits manual</p>
        <p><strong>Ligation </strong>of pCDF-sTAF + sXTBn and pCDF-sTAF + XsTBn according to protocol #028. Deviation :Samples were ligated over night.</p>
        <p>Overnight-cultures of ColonyPCR samples with insert (05.08.2022), following protocol #014.</p>

        <br><b>06/08-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Qualitative test-induction </strong>of NEB5alpha with pCDF-TAsF-sXTB (B5 from overnight-culture 05.08.22), following protocol # . Result: no color change, no indigo production. Need to use BL21 for production, NEB5alpha doesn&rsquo;t work for that.</p>
        <p><strong>Stock-solution of L-Trytophan</strong>, by diluting 0.2554 g L-Trp in 25 mL MQ H<sub>2</sub>O (stock solution with c = 49.95 mM). Transfered in 1 mL eppis and stored at &ndash;20 &deg;C.</p>
        <p><strong>Glycerol stocks </strong>of NEB5alpha with pCDF-TAsF-sXTB samples (B5,B10,B11,B12 from overnight-culture 05.08.22), following protocol #014.</p>
        <p><strong>HeatShock-TraFo </strong>of pCDF-sTAF-sXTB and pCDF-sTAF-XsTB seperately in NEB5alpha, following protocol #003. Result: Bacteria grew to fast, bacterial lawn.</p>
        <p><strong>Mini-Prep </strong>of NEB5alpha pCDF-sTAF-sXTB samples (B5,B10,B11,B12 from overnight-culture 05.08.22), following manufacturers protocol. Measured DNA-conc.: B5 159.8 ng/&micro;L; B10 80.5 ng/&micro;L; B11 254.1 ng/&micro;L(curve looked bad); B12 58.1 ng/&micro;L.</p>

        <br><b>07/08-22</b><br>
        <div class="line-2"> </div>
        <p><strong>Cultures for indiction </strong>were prepared of the samples B5, B10, B11 and B12 (pCDF-TAsF-sXTB). 4ml of LB, 4&micro;l of Spectinomycin and bacteria taken from a glycerol stock were used.</p>
        <p><strong>Trafo </strong>of pCDF-sTAF-sXTB and pCDF-sTAF-XsTB according to protocol #003</p>

      </div>
  </div>
</main>

  <!-- _______________________________ footer ______________________________ -->
  <!-- yellow line starting the footer -->
  <div class="footer_line"></div>

  <footer>
    <div class="container">

      <!-- footer infos (about us, follow us, contact) -->
      <div class="footer_infos grid grid_3">
        <div>
          <h2>About Us</h2>
          <p class="justified">We are <b>"chAMBER"</b> from Freiburg, Germany! <br>
            11 enthusiastic young scientists creating a toolbox for compartmentalization in bacteria for future synthetic biology applications and beyond.</p>
        </div>
        <div class="footer_infoitem2">
          <h2>Follow Us</h2>
          <ul>
            <li><a class="link" href="https://www.instagram.com/igem.freiburg/" target="_blank"><i class="fa fa-brands fa-instagram"></i>Instagram</a></li>
            <li><a class="link" href="https://twitter.com/FreiburgiGEM" target="_blank"><i class="fa fa-brands fa-twitter"></i>Twitter</a></li>
          </ul>
        </div>
        <div class="footer_infoitem3">
          <h2>Contact Us</h2>
          <ul>
            <li><i class="fa fa-solid fa-house-chimney"></i>University of Freiburg, GER</li>
            <li><i class="fa fa-solid fa-envelope"></i>igem2022@cibss.uni-freiburg.de</li>
          </ul>
        </div>
      </div>
    </div>

    <!-- sponsor-logo infinte carusel -->
    <div class="f_carousel_container">
      <section class="slider logo_carousel">
        <div class="slide"><a href="https://www.snapgene.com/" target="_blank"><img class="f_smaller_logo" src="https://static.igem.wiki/teams/4229/wiki/pictures/sponsor-logos/logo-snapgene.png" alt="link to Snapgene, one of our main sponsors"></a></div>
        <div class="slide"><a href="https://uni-freiburg.de/" target="_blank"><img class="f_larger_logo" src="https://static.igem.wiki/teams/4229/wiki/pictures/sponsor-logos/logo-uni-freiburg.png" alt="link to the university of Freiburg, one of our main sponsors"></a></div>
        <div class="slide"><a href="https://www.cibss.uni-freiburg.de/" target="_blank"><img class="f_larger_logo" src="https://static.igem.wiki/teams/4229/wiki/pictures/sponsor-logos/logo-cibss.png" alt="link to CIBSS, one of our main sponsors"></a></div>
        <div class="slide"><a href="https://www.geneious.com/" target="_blank"><img class="f_larger_logo" src="https://static.igem.wiki/teams/4229/wiki/pictures/sponsor-logos/logo-geneious.png" alt="link to geneious, one of our main sponsors"></a></div>
        <div class="slide"><a href="https://eu.idtdna.com/pages" target="_blank"><img class="f_smaller_logo" src="https://static.igem.wiki/teams/4229/wiki/pictures/sponsor-logos/logo-idt.png" alt="link to IDT, one of our main sponsors"></a></div>
      </section>
    </div>

    <!-- CreativeCommons licences + link to gitlab repository (worngly linked atm!!!) -->
    <section class="footer_licensing myt-2">
      <a class="link" href="https://gitlab.igem.org/2022/freiburg" target="_blank">gitlab.igem.org/2022/freiburg</a> | Creative Commons Attribution 4.0 License
    </section>

  </footer>


  <!-- _____________________________ javascript links _________________________ -->
  <script src="static/jquery-3-6-0.min.js"></script>
  <script src="static/bootstrap.min.js"></script>
  <script src="static/slick.min.js"></script>
  <script src="static/three.min.js"></script>
  <script src="static/gltf-loader.js"></script>
  <script src="static/3d-models.js"></script>
  <script src="static/main.js"></script>


</body>
</html>
